{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "#import the qiime2 python module to allow for inline visualizations\n",
    "\n",
    "import qiime2 as q2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSystem versions\u001b[0m\n",
      "Python version: 3.6.10\n",
      "QIIME 2 release: 2020.6\n",
      "QIIME 2 version: 2020.6.0\n",
      "q2cli version: 2020.6.0\n",
      "\u001b[32m\n",
      "Installed plugins\u001b[0m\n",
      "alignment: 2020.6.0\n",
      "composition: 2020.6.0\n",
      "cutadapt: 2020.6.0\n",
      "dada2: 2020.6.0\n",
      "deblur: 2020.6.0\n",
      "demux: 2020.6.0\n",
      "diversity: 2020.6.0\n",
      "diversity-lib: 2020.6.0\n",
      "emperor: 2020.6.0\n",
      "feature-classifier: 2020.6.0\n",
      "feature-table: 2020.6.0\n",
      "fragment-insertion: 2020.6.0\n",
      "gneiss: 2020.6.0\n",
      "longitudinal: 2020.6.0\n",
      "metadata: 2020.6.0\n",
      "phylogeny: 2020.6.0\n",
      "quality-control: 2020.6.0\n",
      "quality-filter: 2020.6.0\n",
      "sample-classifier: 2020.6.0\n",
      "taxa: 2020.6.0\n",
      "types: 2020.6.0\n",
      "vsearch: 2020.6.0\n",
      "\u001b[32m\n",
      "Application config directory\u001b[0m\n",
      "/Users/stephaniematthews/miniconda3/envs/qiime2-2020.6/var/q2cli\u001b[0m\n",
      "\u001b[32m\n",
      "Getting help\u001b[0m\n",
      "To get help with QIIME 2, visit https://qiime2.org\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#which version of qiime2 and python are we using?\n",
    "\n",
    "!qiime info"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "First, import the paired forward and reverse sequences with a manifest file. Manifest prepared by listing all files in bash and formatting in Excel. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported zhan_manifest_r3.csv as PairedEndFastqManifestPhred33 to 18S-paired-end-demux_r3.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#Import the fastq files into a QIIME2 artifact \n",
    "\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path zhan_manifest_r3.csv \\\n",
    "  --output-path 18S-paired-end-demux_r3.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported zhan_manifest_run2_v1.csv as PairedEndFastqManifestPhred33 to 18S-paired-end-demux_r2.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#Import the fastq files into a QIIME2 artifact \n",
    "\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path zhan_manifest_run2_v1.csv \\\n",
    "  --output-path 18S-paired-end-demux_r2.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported zhan_manifest_run1_grid.csv as PairedEndFastqManifestPhred33 to 18S-paired-end-demux_r1.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#Import the fastq files into a QIIME2 artifact \n",
    "\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path zhan_manifest_run1_grid.csv \\\n",
    "  --output-path 18S-paired-end-demux_r1.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Trim reverse primers\n",
    "See: https://docs.qiime2.org/2020.6/plugins/available/cutadapt/trim-paired/\n",
    "default is to match IUPAC wildcards ( --p-match-adapter-wildcards T )\n",
    "- these should be the F/R marker primers\n",
    "- cannot handle inosine bases; using N instead\n",
    "- f primer/ r primer\n",
    "- or could just trim off length of primer ? but this provides a test of primer accuracy  \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S124_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S124_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S124_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S124_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S124_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S124_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S124_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S124_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        67,912 reads  @     60.6 µs/read;   0.99 M reads/minute\n",
      "Finished in 4.12 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,912\n",
      "  Read 1 with adapter:                  67,897 (100.0%)\n",
      "  Read 2 with adapter:                  67,576 (99.5%)\n",
      "Pairs written (passing filters):        67,912 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,065,443 bp\n",
      "  Read 1:    20,033,305 bp\n",
      "  Read 2:    20,032,138 bp\n",
      "Total written (filtered):     37,426,257 bp (93.4%)\n",
      "  Read 1:    18,676,745 bp\n",
      "  Read 2:    18,749,512 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 67897 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t68\t0.0\t1\t0 0 68\n",
      "19\t1429\t0.0\t1\t0 1381 48\n",
      "20\t66252\t0.0\t2\t64654 1581 17\n",
      "21\t111\t0.0\t2\t0 105 6\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 67576 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1447\t0.0\t1\t0 1447\n",
      "19\t66000\t0.0\t1\t64304 1696\n",
      "20\t129\t0.0\t1\t0 129\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S125_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S125_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S125_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S125_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S125_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S125_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S125_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S125_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,744 reads  @     62.6 µs/read;   0.96 M reads/minute\n",
      "Finished in 3.12 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,744\n",
      "  Read 1 with adapter:                  49,730 (100.0%)\n",
      "  Read 2 with adapter:                  49,443 (99.4%)\n",
      "Pairs written (passing filters):        49,744 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,347,422 bp\n",
      "  Read 1:    14,673,323 bp\n",
      "  Read 2:    14,674,099 bp\n",
      "Total written (filtered):     27,415,283 bp (93.4%)\n",
      "  Read 1:    13,679,784 bp\n",
      "  Read 2:    13,735,499 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49730 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t47\t0.0\t1\t0 0 47\n",
      "19\t1083\t0.0\t1\t0 1057 26\n",
      "20\t48509\t0.0\t2\t47260 1233 16\n",
      "21\t66\t0.0\t2\t0 63 3\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49443 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t915\t0.0\t1\t0 915\n",
      "19\t48430\t0.0\t1\t46983 1447\n",
      "20\t98\t0.0\t1\t0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S126_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S126_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S126_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S126_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S126_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S126_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S126_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S126_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,118 reads  @     65.1 µs/read;   0.92 M reads/minute\n",
      "Finished in 3.53 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,118\n",
      "  Read 1 with adapter:                  54,106 (100.0%)\n",
      "  Read 2 with adapter:                  53,606 (99.1%)\n",
      "Pairs written (passing filters):        54,118 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,927,906 bp\n",
      "  Read 1:    15,964,212 bp\n",
      "  Read 2:    15,963,694 bp\n",
      "Total written (filtered):     29,829,258 bp (93.4%)\n",
      "  Read 1:    14,883,157 bp\n",
      "  Read 2:    14,946,101 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54106 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t43\t0.0\t1\t0 0 43\n",
      "19\t1098\t0.0\t1\t0 1061 37\n",
      "20\t52870\t0.0\t2\t51547 1297 26\n",
      "21\t71\t0.0\t2\t0 70 1\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53606 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1020\t0.0\t1\t0 1020\n",
      "19\t52487\t0.0\t1\t50768 1719\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S127_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S127_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S127_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S127_7_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S127_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S127_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S127_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S127_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        60,020 reads  @     65.2 µs/read;   0.92 M reads/minute\n",
      "Finished in 3.92 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,020\n",
      "  Read 1 with adapter:                  60,005 (100.0%)\n",
      "  Read 2 with adapter:                  59,765 (99.6%)\n",
      "Pairs written (passing filters):        60,020 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,410,803 bp\n",
      "  Read 1:    17,704,936 bp\n",
      "  Read 2:    17,705,867 bp\n",
      "Total written (filtered):     33,077,353 bp (93.4%)\n",
      "  Read 1:    16,506,054 bp\n",
      "  Read 2:    16,571,299 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60005 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t54\t0.0\t1\t0 0 54\n",
      "19\t1246\t0.0\t1\t0 1204 42\n",
      "20\t58590\t0.0\t2\t57053 1522 15\n",
      "21\t94\t0.0\t2\t0 94\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59765 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1095\t0.0\t1\t0 1095\n",
      "19\t58542\t0.0\t1\t56981 1561\n",
      "20\t128\t0.0\t1\t0 128\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S128_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S128_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S128_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S128_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S128_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S128_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S128_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S128_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,242 reads  @     64.3 µs/read;   0.93 M reads/minute\n",
      "Finished in 3.30 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,242\n",
      "  Read 1 with adapter:                  51,232 (100.0%)\n",
      "  Read 2 with adapter:                  50,999 (99.5%)\n",
      "Pairs written (passing filters):        51,242 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,231,296 bp\n",
      "  Read 1:    15,116,131 bp\n",
      "  Read 2:    15,115,165 bp\n",
      "Total written (filtered):     28,239,587 bp (93.4%)\n",
      "  Read 1:    14,092,615 bp\n",
      "  Read 2:    14,146,972 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51232 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t49\t0.0\t1\t0 0 49\n",
      "19\t1145\t0.0\t1\t0 1118 27\n",
      "20\t49934\t0.0\t2\t48735 1176 23\n",
      "21\t89\t0.0\t2\t0 85 4\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50999 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t881\t0.0\t1\t0 881\n",
      "19\t50025\t0.0\t1\t48672 1353\n",
      "20\t93\t0.0\t1\t0 93\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S129_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S129_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S129_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S129_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S129_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S129_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S129_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S129_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,537 reads  @     63.5 µs/read;   0.94 M reads/minute\n",
      "Finished in 3.09 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,537\n",
      "  Read 1 with adapter:                  48,523 (100.0%)\n",
      "  Read 2 with adapter:                  48,218 (99.3%)\n",
      "Pairs written (passing filters):        48,537 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,623,448 bp\n",
      "  Read 1:    14,317,401 bp\n",
      "  Read 2:    14,306,047 bp\n",
      "Total written (filtered):     26,738,717 bp (93.4%)\n",
      "  Read 1:    13,347,908 bp\n",
      "  Read 2:    13,390,809 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48523 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t58\t0.0\t1\t0 0 58\n",
      "19\t978\t0.0\t1\t0 950 28\n",
      "20\t47387\t0.0\t2\t46139 1234 14\n",
      "21\t73\t0.0\t2\t0 73\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48218 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t989\t0.0\t1\t0 989\n",
      "19\t47144\t0.0\t1\t45859 1285\n",
      "20\t85\t0.0\t1\t0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S130_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S130_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S130_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S130_13_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S130_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S130_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S130_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S130_13_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        60,486 reads  @     66.5 µs/read;   0.90 M reads/minute\n",
      "Finished in 4.03 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,486\n",
      "  Read 1 with adapter:                  60,472 (100.0%)\n",
      "  Read 2 with adapter:                  60,055 (99.3%)\n",
      "Pairs written (passing filters):        60,486 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,664,535 bp\n",
      "  Read 1:    17,843,224 bp\n",
      "  Read 2:    17,821,311 bp\n",
      "Total written (filtered):     33,316,132 bp (93.4%)\n",
      "  Read 1:    16,634,985 bp\n",
      "  Read 2:    16,681,147 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60472 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t44\t0.0\t1\t0 0 44\n",
      "19\t1246\t0.0\t1\t0 1200 46\n",
      "20\t59076\t0.0\t2\t57483 1564 29\n",
      "21\t79\t0.0\t2\t0 77 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 60055 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t973\t0.0\t1\t0 973\n",
      "19\t58990\t0.0\t1\t57546 1444\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S131_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S131_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S131_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S131_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S131_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S131_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S131_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S131_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        69,839 reads  @     69.5 µs/read;   0.86 M reads/minute\n",
      "Finished in 4.86 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,839\n",
      "  Read 1 with adapter:                  69,830 (100.0%)\n",
      "  Read 2 with adapter:                  67,848 (97.1%)\n",
      "Pairs written (passing filters):        69,839 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,092,988 bp\n",
      "  Read 1:    20,601,721 bp\n",
      "  Read 2:    20,491,267 bp\n",
      "Total written (filtered):     38,409,198 bp (93.5%)\n",
      "  Read 1:    19,206,506 bp\n",
      "  Read 2:    19,202,692 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 69830 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t50\t0.0\t1\t0 0 50\n",
      "19\t1444\t0.0\t1\t0 1385 59\n",
      "20\t68208\t0.0\t2\t66447 1737 24\n",
      "21\t97\t0.0\t2\t0 96 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 67848 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t647\t0.0\t1\t0 647\n",
      "19\t67091\t0.0\t1\t65644 1447\n",
      "20\t110\t0.0\t1\t0 110\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S132_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S132_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S132_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S132_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S132_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S132_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S132_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S132_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        68,047 reads  @     63.1 µs/read;   0.95 M reads/minute\n",
      "Finished in 4.30 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,047\n",
      "  Read 1 with adapter:                  68,033 (100.0%)\n",
      "  Read 2 with adapter:                  67,727 (99.5%)\n",
      "Pairs written (passing filters):        68,047 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,144,011 bp\n",
      "  Read 1:    20,073,426 bp\n",
      "  Read 2:    20,070,585 bp\n",
      "Total written (filtered):     37,498,770 bp (93.4%)\n",
      "  Read 1:    18,713,734 bp\n",
      "  Read 2:    18,785,036 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 68033 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t43\t0.0\t1\t0 0 43\n",
      "19\t1064\t0.0\t1\t0 1041 23\n",
      "20\t66783\t0.0\t2\t65001 1755 27\n",
      "21\t104\t0.0\t2\t0 103 1\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 67727 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1409\t0.0\t1\t0 1409\n",
      "19\t66173\t0.0\t1\t64532 1641\n",
      "20\t145\t0.0\t1\t0 145\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S133_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S133_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S133_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S133_19_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S133_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S133_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S133_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S133_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04        69,695 reads  @     59.6 µs/read;   1.01 M reads/minute\n",
      "Finished in 4.16 s (60 us/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,695\n",
      "  Read 1 with adapter:                  69,690 (100.0%)\n",
      "  Read 2 with adapter:                  69,316 (99.5%)\n",
      "Pairs written (passing filters):        69,695 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,119,345 bp\n",
      "  Read 1:    20,559,999 bp\n",
      "  Read 2:    20,559,346 bp\n",
      "Total written (filtered):     38,410,807 bp (93.4%)\n",
      "  Read 1:    19,167,293 bp\n",
      "  Read 2:    19,243,514 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 69690 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t52\t0.0\t1\t0 0 52\n",
      "19\t1159\t0.0\t1\t0 1118 41\n",
      "20\t68349\t0.0\t2\t66615 1712 22\n",
      "21\t91\t0.0\t2\t0 88 3\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 69316 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1282\t0.0\t1\t0 1282\n",
      "19\t67924\t0.0\t1\t66152 1772\n",
      "20\t110\t0.0\t1\t0 110\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S134_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S134_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S134_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S134_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S134_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S134_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S134_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S134_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        80,095 reads  @     60.2 µs/read;   1.00 M reads/minute\n",
      "Finished in 4.83 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,095\n",
      "  Read 1 with adapter:                  80,079 (100.0%)\n",
      "  Read 2 with adapter:                  79,737 (99.6%)\n",
      "Pairs written (passing filters):        80,095 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,255,407 bp\n",
      "  Read 1:    23,627,804 bp\n",
      "  Read 2:    23,627,603 bp\n",
      "Total written (filtered):     44,141,514 bp (93.4%)\n",
      "  Read 1:    22,027,438 bp\n",
      "  Read 2:    22,114,076 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 80079 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t52\t0.0\t1\t0 0 52\n",
      "19\t1314\t0.0\t1\t0 1288 26\n",
      "20\t78553\t0.0\t2\t76498 2026 29\n",
      "21\t116\t0.0\t2\t0 115 1\n",
      "22\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 79737 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1641\t0.0\t1\t0 1641\n",
      "19\t77931\t0.0\t1\t76056 1875\n",
      "20\t165\t0.0\t1\t0 165\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S135_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S135_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S135_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S135_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S135_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S135_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S135_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S135_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        81,314 reads  @     63.3 µs/read;   0.95 M reads/minute\n",
      "Finished in 5.15 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,314\n",
      "  Read 1 with adapter:                  81,300 (100.0%)\n",
      "  Read 2 with adapter:                  81,008 (99.6%)\n",
      "Pairs written (passing filters):        81,314 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,974,606 bp\n",
      "  Read 1:    23,987,034 bp\n",
      "  Read 2:    23,987,572 bp\n",
      "Total written (filtered):     44,812,090 bp (93.4%)\n",
      "  Read 1:    22,362,356 bp\n",
      "  Read 2:    22,449,734 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 81300 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t55\t0.0\t1\t0 0 55\n",
      "19\t1418\t0.0\t1\t0 1377 41\n",
      "20\t79671\t0.0\t2\t77587 2053 31\n",
      "21\t106\t0.0\t2\t0 104 2\n",
      "22\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 81008 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1482\t0.0\t1\t0 1482\n",
      "19\t79358\t0.0\t1\t77512 1846\n",
      "20\t168\t0.0\t1\t0 168\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S136_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S136_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S136_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S136_25_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S136_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S136_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S136_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S136_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,544 reads  @     67.7 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.16 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,544\n",
      "  Read 1 with adapter:                  46,532 (100.0%)\n",
      "  Read 2 with adapter:                  46,368 (99.6%)\n",
      "Pairs written (passing filters):        46,543 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,459,035 bp\n",
      "  Read 1:    13,729,821 bp\n",
      "  Read 2:    13,729,214 bp\n",
      "Total written (filtered):     25,648,572 bp (93.4%)\n",
      "  Read 1:    12,799,612 bp\n",
      "  Read 2:    12,848,960 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46532 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t765\t0.0\t1\t0 738 27\n",
      "20\t45646\t0.0\t2\t44561 1079 6\n",
      "21\t59\t0.0\t2\t0 57 2\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46368 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t821\t0.0\t1\t0 821\n",
      "19\t45464\t0.0\t1\t44391 1073\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S137_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S137_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S137_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S137_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S137_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S137_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S137_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S137_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        74,550 reads  @     63.6 µs/read;   0.94 M reads/minute\n",
      "Finished in 4.74 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,550\n",
      "  Read 1 with adapter:                  74,528 (100.0%)\n",
      "  Read 2 with adapter:                  74,148 (99.5%)\n",
      "Pairs written (passing filters):        74,549 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,966,112 bp\n",
      "  Read 1:    21,991,857 bp\n",
      "  Read 2:    21,974,255 bp\n",
      "Total written (filtered):     41,069,114 bp (93.4%)\n",
      "  Read 1:    20,502,233 bp\n",
      "  Read 2:    20,566,881 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 74528 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t60\t0.0\t1\t0 0 60\n",
      "19\t1298\t0.0\t1\t0 1263 35\n",
      "20\t73005\t0.0\t2\t71027 1950 28\n",
      "21\t123\t0.0\t2\t0 120 3\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 74148 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1567\t0.0\t1\t0 1567\n",
      "19\t72452\t0.0\t1\t70694 1758\n",
      "20\t129\t0.0\t1\t0 129\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S138_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S138_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S138_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S138_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S138_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S138_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S138_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S138_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:09       146,234 reads  @     63.6 µs/read;   0.94 M reads/minute\n",
      "Finished in 9.30 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            146,234\n",
      "  Read 1 with adapter:                 146,199 (100.0%)\n",
      "  Read 2 with adapter:                 145,342 (99.4%)\n",
      "Pairs written (passing filters):       146,233 (100.0%)\n",
      "\n",
      "Total basepairs processed:    86,226,364 bp\n",
      "  Read 1:    43,138,552 bp\n",
      "  Read 2:    43,087,812 bp\n",
      "Total written (filtered):     80,545,108 bp (93.4%)\n",
      "  Read 1:    40,216,550 bp\n",
      "  Read 2:    40,328,558 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 146199 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t94\t0.0\t1\t0 0 94\n",
      "19\t2388\t0.0\t1\t0 2306 82\n",
      "20\t143461\t0.0\t2\t139614 3787 60\n",
      "21\t189\t0.0\t2\t0 176 13\n",
      "22\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 145342 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t2493\t0.0\t1\t0 2493\n",
      "19\t142600\t0.0\t1\t139439 3161\n",
      "20\t249\t0.0\t1\t0 249\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S139_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S139_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S139_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S139_31_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S139_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S139_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S139_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S139_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        53,802 reads  @     67.0 µs/read;   0.90 M reads/minute\n",
      "Finished in 3.61 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,802\n",
      "  Read 1 with adapter:                  53,786 (100.0%)\n",
      "  Read 2 with adapter:                  52,314 (97.2%)\n",
      "Pairs written (passing filters):        53,802 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,658,865 bp\n",
      "  Read 1:    15,870,808 bp\n",
      "  Read 2:    15,788,057 bp\n",
      "Total written (filtered):     29,590,393 bp (93.5%)\n",
      "  Read 1:    14,795,912 bp\n",
      "  Read 2:    14,794,481 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53786 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t44\t0.0\t1\t0 0 44\n",
      "19\t890\t0.0\t1\t0 871 19\n",
      "20\t52731\t0.0\t2\t51329 1375 27\n",
      "21\t88\t0.0\t2\t0 82 6\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52314 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t465\t0.0\t1\t0 465\n",
      "19\t51774\t0.0\t1\t50705 1069\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S140_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S140_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S140_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S140_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S140_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S140_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S140_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S140_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        64,232 reads  @     60.8 µs/read;   0.99 M reads/minute\n",
      "Finished in 3.91 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,232\n",
      "  Read 1 with adapter:                  64,216 (100.0%)\n",
      "  Read 2 with adapter:                  63,918 (99.5%)\n",
      "Pairs written (passing filters):        64,232 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,895,298 bp\n",
      "  Read 1:    18,947,569 bp\n",
      "  Read 2:    18,947,729 bp\n",
      "Total written (filtered):     35,398,308 bp (93.4%)\n",
      "  Read 1:    17,663,725 bp\n",
      "  Read 2:    17,734,583 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64216 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t600\t0.0\t1\t0 584 16\n",
      "20\t63457\t0.0\t2\t61876 1561 20\n",
      "21\t90\t0.0\t2\t0 88 2\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63918 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1434\t0.0\t1\t0 1434\n",
      "19\t62346\t0.0\t1\t60605 1741\n",
      "20\t138\t0.0\t1\t0 138\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S141_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S141_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S141_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S141_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S141_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S141_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S141_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S141_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        58,139 reads  @     65.5 µs/read;   0.92 M reads/minute\n",
      "Finished in 3.81 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,139\n",
      "  Read 1 with adapter:                  58,133 (100.0%)\n",
      "  Read 2 with adapter:                  57,793 (99.4%)\n",
      "Pairs written (passing filters):        58,139 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,301,543 bp\n",
      "  Read 1:    17,150,746 bp\n",
      "  Read 2:    17,150,797 bp\n",
      "Total written (filtered):     32,042,320 bp (93.4%)\n",
      "  Read 1:    15,988,599 bp\n",
      "  Read 2:    16,053,721 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58133 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t599\t0.0\t1\t0 581 18\n",
      "20\t57416\t0.0\t2\t55950 1445 21\n",
      "21\t78\t0.0\t2\t0 74 4\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57793 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1093\t0.0\t1\t0 1093\n",
      "19\t56598\t0.0\t1\t54804 1794\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S142_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S142_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S142_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S142_37_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S142_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S142_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S142_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S142_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        55,305 reads  @     61.9 µs/read;   0.97 M reads/minute\n",
      "Finished in 3.43 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,305\n",
      "  Read 1 with adapter:                  55,291 (100.0%)\n",
      "  Read 2 with adapter:                  55,017 (99.5%)\n",
      "Pairs written (passing filters):        55,305 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,628,817 bp\n",
      "  Read 1:    16,314,843 bp\n",
      "  Read 2:    16,313,974 bp\n",
      "Total written (filtered):     30,479,015 bp (93.4%)\n",
      "  Read 1:    15,209,448 bp\n",
      "  Read 2:    15,269,567 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55291 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t535\t0.0\t1\t0 527 8\n",
      "20\t54619\t0.0\t2\t53254 1341 24\n",
      "21\t88\t0.0\t2\t0 87 1\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1027\t0.0\t1\t0 1027\n",
      "19\t53879\t0.0\t1\t52407 1472\n",
      "20\t111\t0.0\t1\t0 111\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S143_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S143_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S143_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S143_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S143_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S143_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S143_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S143_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,684 reads  @     62.3 µs/read;   0.96 M reads/minute\n",
      "Finished in 3.29 s (62 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,684\n",
      "  Read 1 with adapter:                  52,668 (100.0%)\n",
      "  Read 2 with adapter:                  52,404 (99.5%)\n",
      "Pairs written (passing filters):        52,684 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,083,180 bp\n",
      "  Read 1:    15,541,505 bp\n",
      "  Read 2:    15,541,675 bp\n",
      "Total written (filtered):     29,035,475 bp (93.4%)\n",
      "  Read 1:    14,488,589 bp\n",
      "  Read 2:    14,546,886 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52668 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t523\t0.0\t1\t0 508 15\n",
      "20\t52044\t0.0\t2\t50759 1267 18\n",
      "21\t63\t0.0\t2\t0 62 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52404 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t993\t0.0\t1\t0 993\n",
      "19\t51305\t0.0\t1\t49851 1454\n",
      "20\t106\t0.0\t1\t0 106\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S144_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S144_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S144_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S144_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S144_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S144_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S144_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S144_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06       103,391 reads  @     64.5 µs/read;   0.93 M reads/minute\n",
      "Finished in 6.68 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            103,391\n",
      "  Read 1 with adapter:                 103,372 (100.0%)\n",
      "  Read 2 with adapter:                 102,829 (99.5%)\n",
      "Pairs written (passing filters):       103,391 (100.0%)\n",
      "\n",
      "Total basepairs processed:    60,996,827 bp\n",
      "  Read 1:    30,500,168 bp\n",
      "  Read 2:    30,496,659 bp\n",
      "Total written (filtered):     56,977,920 bp (93.4%)\n",
      "  Read 1:    28,433,451 bp\n",
      "  Read 2:    28,544,469 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 103372 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t36\t0.0\t1\t0 0 36\n",
      "19\t905\t0.0\t1\t0 876 29\n",
      "20\t102233\t0.0\t2\t99783 2420 30\n",
      "21\t142\t0.0\t2\t0 140 2\n",
      "22\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 102829 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1727\t0.0\t1\t0 1727\n",
      "19\t100936\t0.0\t1\t98001 2935\n",
      "20\t166\t0.0\t1\t0 166\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S145_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S145_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S145_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S145_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S145_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S145_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S145_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S145_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        58,289 reads  @     64.7 µs/read;   0.93 M reads/minute\n",
      "Finished in 3.78 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,289\n",
      "  Read 1 with adapter:                  58,269 (100.0%)\n",
      "  Read 2 with adapter:                  57,874 (99.3%)\n",
      "Pairs written (passing filters):        58,289 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,377,277 bp\n",
      "  Read 1:    17,194,485 bp\n",
      "  Read 2:    17,182,792 bp\n",
      "Total written (filtered):     32,113,851 bp (93.4%)\n",
      "  Read 1:    16,029,541 bp\n",
      "  Read 2:    16,084,310 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58269 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t526\t0.0\t1\t0 512 14\n",
      "20\t57608\t0.0\t2\t56215 1373 20\n",
      "21\t84\t0.0\t2\t0 79 5\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57874 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1220\t0.0\t1\t0 1220\n",
      "19\t56558\t0.0\t1\t54945 1613\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S146_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S146_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S146_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S146_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S146_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S146_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S146_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S146_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        55,325 reads  @     69.8 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.87 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,325\n",
      "  Read 1 with adapter:                  55,316 (100.0%)\n",
      "  Read 2 with adapter:                  54,910 (99.2%)\n",
      "Pairs written (passing filters):        55,325 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,617,822 bp\n",
      "  Read 1:    16,320,614 bp\n",
      "  Read 2:    16,297,208 bp\n",
      "Total written (filtered):     30,469,470 bp (93.4%)\n",
      "  Read 1:    15,214,777 bp\n",
      "  Read 2:    15,254,693 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55316 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t560\t0.0\t1\t0 538 22\n",
      "20\t54630\t0.0\t2\t53268 1349 13\n",
      "21\t75\t0.0\t2\t0 72 3\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54910 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t865\t0.0\t1\t0 865\n",
      "19\t53955\t0.0\t1\t52614 1341\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S147_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S147_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S147_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S147_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S147_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S147_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S147_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S147_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        64,196 reads  @     67.9 µs/read;   0.88 M reads/minute\n",
      "Finished in 4.36 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,196\n",
      "  Read 1 with adapter:                  64,176 (100.0%)\n",
      "  Read 2 with adapter:                  61,755 (96.2%)\n",
      "Pairs written (passing filters):        64,196 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,737,260 bp\n",
      "  Read 1:    18,937,674 bp\n",
      "  Read 2:    18,799,586 bp\n",
      "Total written (filtered):     35,281,359 bp (93.5%)\n",
      "  Read 1:    17,654,614 bp\n",
      "  Read 2:    17,626,745 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64176 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t573\t0.0\t1\t0 550 23\n",
      "20\t63458\t0.0\t2\t61786 1646 26\n",
      "21\t85\t0.0\t2\t0 82 3\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61755 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t587\t0.0\t1\t0 587\n",
      "19\t61085\t0.0\t1\t59565 1520\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S148_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S148_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S148_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S148_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S148_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S148_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S148_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S148_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        70,857 reads  @     64.0 µs/read;   0.94 M reads/minute\n",
      "Finished in 4.54 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             70,857\n",
      "  Read 1 with adapter:                  70,835 (100.0%)\n",
      "  Read 2 with adapter:                  70,532 (99.5%)\n",
      "Pairs written (passing filters):        70,856 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,802,816 bp\n",
      "  Read 1:    20,901,800 bp\n",
      "  Read 2:    20,901,016 bp\n",
      "Total written (filtered):     39,048,450 bp (93.4%)\n",
      "  Read 1:    19,486,222 bp\n",
      "  Read 2:    19,562,228 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 70835 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t65\t0.0\t1\t0 0 65\n",
      "19\t1495\t0.0\t1\t0 1459 36\n",
      "20\t69104\t0.0\t2\t67653 1428 23\n",
      "21\t114\t0.0\t2\t0 112 2\n",
      "22\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 70532 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1462\t0.0\t1\t0 1462\n",
      "19\t68928\t0.0\t1\t67193 1735\n",
      "20\t142\t0.0\t1\t0 142\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S149_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S149_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S149_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S149_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S149_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S149_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S149_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S149_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,411 reads  @     63.6 µs/read;   0.94 M reads/minute\n",
      "Finished in 3.15 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,411\n",
      "  Read 1 with adapter:                  49,399 (100.0%)\n",
      "  Read 2 with adapter:                  49,190 (99.6%)\n",
      "Pairs written (passing filters):        49,411 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,152,007 bp\n",
      "  Read 1:    14,576,157 bp\n",
      "  Read 2:    14,575,850 bp\n",
      "Total written (filtered):     27,231,215 bp (93.4%)\n",
      "  Read 1:    13,589,222 bp\n",
      "  Read 2:    13,641,993 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49399 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t57\t0.0\t1\t0 0 57\n",
      "19\t1060\t0.0\t1\t0 1042 18\n",
      "20\t48179\t0.0\t2\t47233 936 10\n",
      "21\t77\t0.0\t2\t0 76 1\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49190 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t827\t0.0\t1\t0 827\n",
      "19\t48289\t0.0\t1\t47080 1209\n",
      "20\t74\t0.0\t1\t0 74\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S150_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S150_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S150_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S150_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S150_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S150_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S150_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S150_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        58,244 reads  @     69.7 µs/read;   0.86 M reads/minute\n",
      "Finished in 4.07 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,244\n",
      "  Read 1 with adapter:                  58,223 (100.0%)\n",
      "  Read 2 with adapter:                  58,022 (99.6%)\n",
      "Pairs written (passing filters):        58,244 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,363,637 bp\n",
      "  Read 1:    17,181,959 bp\n",
      "  Read 2:    17,181,678 bp\n",
      "Total written (filtered):     32,099,031 bp (93.4%)\n",
      "  Read 1:    16,018,698 bp\n",
      "  Read 2:    16,080,333 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58223 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t55\t0.0\t1\t0 0 55\n",
      "19\t1253\t0.0\t1\t0 1220 33\n",
      "20\t56788\t0.0\t2\t55700 1073 15\n",
      "21\t90\t0.0\t2\t0 87 3\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 58022 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1182\t0.0\t1\t0 1182\n",
      "19\t56731\t0.0\t1\t55426 1305\n",
      "20\t109\t0.0\t1\t0 109\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S151_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S151_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S151_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S151_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S151_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S151_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S151_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S151_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04        76,114 reads  @     65.0 µs/read;   0.92 M reads/minute\n",
      "Finished in 4.95 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             76,114\n",
      "  Read 1 with adapter:                  76,094 (100.0%)\n",
      "  Read 2 with adapter:                  75,813 (99.6%)\n",
      "Pairs written (passing filters):        76,114 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,906,674 bp\n",
      "  Read 1:    22,453,352 bp\n",
      "  Read 2:    22,453,322 bp\n",
      "Total written (filtered):     41,947,106 bp (93.4%)\n",
      "  Read 1:    20,932,967 bp\n",
      "  Read 2:    21,014,139 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 76094 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t57\t0.0\t1\t0 0 57\n",
      "19\t1609\t0.0\t1\t0 1574 35\n",
      "20\t74250\t0.0\t2\t72765 1462 23\n",
      "21\t128\t0.0\t2\t0 127 1\n",
      "22\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 75813 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1446\t0.0\t1\t0 1446\n",
      "19\t74185\t0.0\t1\t72400 1785\n",
      "20\t182\t0.0\t1\t0 182\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S152_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S152_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S152_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S152_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S152_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S152_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S152_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S152_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        65,705 reads  @     64.6 µs/read;   0.93 M reads/minute\n",
      "Finished in 4.25 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,705\n",
      "  Read 1 with adapter:                  65,688 (100.0%)\n",
      "  Read 2 with adapter:                  65,458 (99.6%)\n",
      "Pairs written (passing filters):        65,705 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,764,910 bp\n",
      "  Read 1:    19,382,696 bp\n",
      "  Read 2:    19,382,214 bp\n",
      "Total written (filtered):     36,209,786 bp (93.4%)\n",
      "  Read 1:    18,070,259 bp\n",
      "  Read 2:    18,139,527 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 65688 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t64\t0.0\t1\t0 0 64\n",
      "19\t1375\t0.0\t1\t0 1341 34\n",
      "20\t64106\t0.0\t2\t62825 1272 9\n",
      "21\t106\t0.0\t2\t0 105 1\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 65458 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1111\t0.0\t1\t0 1111\n",
      "19\t64251\t0.0\t1\t62711 1540\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S153_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S153_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S153_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S153_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S153_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S153_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S153_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S153_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        82,326 reads  @     58.8 µs/read;   1.02 M reads/minute\n",
      "Finished in 4.84 s (59 us/read; 1.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             82,326\n",
      "  Read 1 with adapter:                  82,304 (100.0%)\n",
      "  Read 2 with adapter:                  81,930 (99.5%)\n",
      "Pairs written (passing filters):        82,326 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,558,155 bp\n",
      "  Read 1:    24,286,138 bp\n",
      "  Read 2:    24,272,017 bp\n",
      "Total written (filtered):     45,358,658 bp (93.4%)\n",
      "  Read 1:    22,641,695 bp\n",
      "  Read 2:    22,716,963 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 82304 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t72\t0.0\t1\t0 0 72\n",
      "19\t1748\t0.0\t1\t0 1698 50\n",
      "20\t80300\t0.0\t2\t78783 1501 16\n",
      "21\t113\t0.0\t2\t0 109 4\n",
      "22\t71\t0.0\t2\t0 0 71\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 81930 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1774\t0.0\t1\t0 1774\n",
      "19\t79998\t0.0\t1\t78152 1846\n",
      "20\t158\t0.0\t1\t0 158\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S154_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S154_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S154_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S154_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S154_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S154_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S154_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S154_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        63,655 reads  @     60.2 µs/read;   1.00 M reads/minute\n",
      "Finished in 3.84 s (60 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,655\n",
      "  Read 1 with adapter:                  63,635 (100.0%)\n",
      "  Read 2 with adapter:                  63,213 (99.3%)\n",
      "Pairs written (passing filters):        63,655 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,538,390 bp\n",
      "  Read 1:    18,778,195 bp\n",
      "  Read 2:    18,760,195 bp\n",
      "Total written (filtered):     35,066,896 bp (93.4%)\n",
      "  Read 1:    17,506,812 bp\n",
      "  Read 2:    17,560,084 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63635 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t71\t0.0\t1\t0 0 71\n",
      "19\t1318\t0.0\t1\t0 1281 37\n",
      "20\t62128\t0.0\t2\t60902 1210 16\n",
      "21\t93\t0.0\t2\t0 89 4\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63213 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1038\t0.0\t1\t0 1038\n",
      "19\t62073\t0.0\t1\t60647 1426\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S155_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S155_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S155_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S155_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S155_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S155_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S155_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S155_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        62,580 reads  @     60.9 µs/read;   0.99 M reads/minute\n",
      "Finished in 3.81 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,580\n",
      "  Read 1 with adapter:                  62,565 (100.0%)\n",
      "  Read 2 with adapter:                  60,926 (97.4%)\n",
      "Pairs written (passing filters):        62,580 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,831,752 bp\n",
      "  Read 1:    18,460,673 bp\n",
      "  Read 2:    18,371,079 bp\n",
      "Total written (filtered):     34,424,665 bp (93.5%)\n",
      "  Read 1:    17,210,693 bp\n",
      "  Read 2:    17,213,972 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 62565 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t62\t0.0\t1\t0 0 62\n",
      "19\t1368\t0.0\t1\t0 1325 43\n",
      "20\t61001\t0.0\t2\t59766 1225 10\n",
      "21\t96\t0.0\t2\t0 94 2\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 60926 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t571\t0.0\t1\t0 571\n",
      "19\t60271\t0.0\t1\t58982 1289\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S156_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S156_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S156_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S156_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S156_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S156_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S156_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S156_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        65,196 reads  @     62.4 µs/read;   0.96 M reads/minute\n",
      "Finished in 4.07 s (62 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,196\n",
      "  Read 1 with adapter:                  65,175 (100.0%)\n",
      "  Read 2 with adapter:                  64,848 (99.5%)\n",
      "Pairs written (passing filters):        65,196 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,462,301 bp\n",
      "  Read 1:    19,231,322 bp\n",
      "  Read 2:    19,230,979 bp\n",
      "Total written (filtered):     35,928,864 bp (93.4%)\n",
      "  Read 1:    17,928,755 bp\n",
      "  Read 2:    18,000,109 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 65175 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t32\t0.0\t1\t0 0 32\n",
      "19\t1044\t0.0\t1\t0 1014 30\n",
      "20\t63960\t0.0\t2\t62389 1551 20\n",
      "21\t103\t0.0\t2\t0 98 5\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 64848 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1376\t0.0\t1\t0 1376\n",
      "19\t63338\t0.0\t1\t61660 1678\n",
      "20\t134\t0.0\t1\t0 134\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S157_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S157_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S157_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S157_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S157_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S157_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S157_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S157_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:03        60,461 reads  @     61.1 µs/read;   0.98 M reads/minute\n",
      "Finished in 3.70 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,461\n",
      "  Read 1 with adapter:                  60,448 (100.0%)\n",
      "  Read 2 with adapter:                  60,167 (99.5%)\n",
      "Pairs written (passing filters):        60,461 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,670,867 bp\n",
      "  Read 1:    17,835,431 bp\n",
      "  Read 2:    17,835,436 bp\n",
      "Total written (filtered):     33,320,623 bp (93.4%)\n",
      "  Read 1:    16,627,379 bp\n",
      "  Read 2:    16,693,244 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60448 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t41\t0.0\t1\t0 0 41\n",
      "19\t984\t0.0\t1\t0 953 31\n",
      "20\t59308\t0.0\t2\t57919 1374 15\n",
      "21\t72\t0.0\t2\t0 68 4\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 60167 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1077\t0.0\t1\t0 1077\n",
      "19\t58994\t0.0\t1\t57445 1549\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S158_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S158_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S158_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S158_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S158_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S158_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S158_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S158_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        62,249 reads  @     59.9 µs/read;   1.00 M reads/minute\n",
      "Finished in 3.74 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,249\n",
      "  Read 1 with adapter:                  62,240 (100.0%)\n",
      "  Read 2 with adapter:                  61,970 (99.6%)\n",
      "Pairs written (passing filters):        62,249 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,726,439 bp\n",
      "  Read 1:    18,363,153 bp\n",
      "  Read 2:    18,363,286 bp\n",
      "Total written (filtered):     34,306,242 bp (93.4%)\n",
      "  Read 1:    17,119,271 bp\n",
      "  Read 2:    17,186,971 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 62240 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t40\t0.0\t1\t0 0 40\n",
      "19\t994\t0.0\t1\t0 967 27\n",
      "20\t61075\t0.0\t2\t59580 1474 21\n",
      "21\t106\t0.0\t2\t0 102 4\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61970 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1248\t0.0\t1\t0 1248\n",
      "19\t60589\t0.0\t1\t59073 1516\n",
      "20\t133\t0.0\t1\t0 133\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S159_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S159_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S159_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S159_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S159_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S159_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S159_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S159_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        62,090 reads  @     60.5 µs/read;   0.99 M reads/minute\n",
      "Finished in 3.77 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,090\n",
      "  Read 1 with adapter:                  62,079 (100.0%)\n",
      "  Read 2 with adapter:                  61,796 (99.5%)\n",
      "Pairs written (passing filters):        62,090 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,632,071 bp\n",
      "  Read 1:    18,315,889 bp\n",
      "  Read 2:    18,316,182 bp\n",
      "Total written (filtered):     34,218,354 bp (93.4%)\n",
      "  Read 1:    17,075,233 bp\n",
      "  Read 2:    17,143,121 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 62079 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t37\t0.0\t1\t0 0 37\n",
      "19\t1024\t0.0\t1\t0 993 31\n",
      "20\t60877\t0.0\t2\t59385 1477 15\n",
      "21\t108\t0.0\t2\t0 106 2\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61796 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1199\t0.0\t1\t0 1199\n",
      "19\t60461\t0.0\t1\t58939 1522\n",
      "20\t136\t0.0\t1\t0 136\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S160_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S160_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S160_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S160_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S160_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S160_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S160_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S160_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,683 reads  @     59.7 µs/read;   1.01 M reads/minute\n",
      "Finished in 3.27 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,683\n",
      "  Read 1 with adapter:                  54,678 (100.0%)\n",
      "  Read 2 with adapter:                  54,451 (99.6%)\n",
      "Pairs written (passing filters):        54,683 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,261,717 bp\n",
      "  Read 1:    16,131,130 bp\n",
      "  Read 2:    16,130,587 bp\n",
      "Total written (filtered):     30,135,361 bp (93.4%)\n",
      "  Read 1:    15,038,454 bp\n",
      "  Read 2:    15,096,907 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54678 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t948\t0.0\t1\t0 917 31\n",
      "20\t53601\t0.0\t2\t52303 1281 17\n",
      "21\t70\t0.0\t2\t0 69 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54451 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t985\t0.0\t1\t0 985\n",
      "19\t53370\t0.0\t1\t52052 1318\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S161_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S161_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S161_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S161_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S161_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S161_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S161_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S161_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        47,169 reads  @     58.2 µs/read;   1.03 M reads/minute\n",
      "Finished in 2.75 s (58 us/read; 1.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,169\n",
      "  Read 1 with adapter:                  47,161 (100.0%)\n",
      "  Read 2 with adapter:                  46,896 (99.4%)\n",
      "Pairs written (passing filters):        47,169 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,821,323 bp\n",
      "  Read 1:    13,914,559 bp\n",
      "  Read 2:    13,906,764 bp\n",
      "Total written (filtered):     25,988,708 bp (93.4%)\n",
      "  Read 1:    12,972,041 bp\n",
      "  Read 2:    13,016,667 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47161 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t34\t0.0\t1\t0 0 34\n",
      "19\t762\t0.0\t1\t0 744 18\n",
      "20\t46265\t0.0\t2\t45158 1099 8\n",
      "21\t72\t0.0\t2\t0 70 2\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46896 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1011\t0.0\t1\t0 1011\n",
      "19\t45801\t0.0\t1\t44633 1168\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S162_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S162_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S162_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S162_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S162_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S162_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S162_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S162_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        46,075 reads  @     58.5 µs/read;   1.03 M reads/minute\n",
      "Finished in 2.70 s (59 us/read; 1.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,075\n",
      "  Read 1 with adapter:                  46,067 (100.0%)\n",
      "  Read 2 with adapter:                  45,795 (99.4%)\n",
      "Pairs written (passing filters):        46,074 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,170,567 bp\n",
      "  Read 1:    13,592,104 bp\n",
      "  Read 2:    13,578,463 bp\n",
      "Total written (filtered):     25,380,221 bp (93.4%)\n",
      "  Read 1:    12,671,171 bp\n",
      "  Read 2:    12,709,050 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46067 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t738\t0.0\t1\t0 722 16\n",
      "20\t45236\t0.0\t2\t44144 1073 19\n",
      "21\t62\t0.0\t2\t0 61 1\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45795 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t768\t0.0\t1\t0 768\n",
      "19\t44951\t0.0\t1\t43957 994\n",
      "20\t76\t0.0\t1\t0 76\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S163_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S163_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S163_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S163_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S163_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S163_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S163_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S163_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         1,034 reads  @    160.3 µs/read;   0.37 M reads/minute\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.17 s (163 us/read; 0.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              1,034\n",
      "  Read 1 with adapter:                   1,026 (99.2%)\n",
      "  Read 2 with adapter:                   1,004 (97.1%)\n",
      "Pairs written (passing filters):         1,034 (100.0%)\n",
      "\n",
      "Total basepairs processed:       608,726 bp\n",
      "  Read 1:       304,014 bp\n",
      "  Read 2:       304,712 bp\n",
      "Total written (filtered):        569,157 bp (93.5%)\n",
      "  Read 1:       283,513 bp\n",
      "  Read 2:       285,644 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 1026 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t21\t0.0\t1\t0 21\n",
      "20\t1003\t0.0\t2\t978 25\n",
      "21\t2\t0.0\t2\t0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 1004 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 12\n",
      "19\t988\t0.0\t1\t957 31\n",
      "20\t4\t0.0\t1\t0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S165_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S165_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S165_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S165_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S165_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S165_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S165_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S165_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        15,435 reads  @    121.9 µs/read;   0.49 M reads/minute\n",
      "Finished in 1.89 s (122 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             15,435\n",
      "  Read 1 with adapter:                  15,391 (99.7%)\n",
      "  Read 2 with adapter:                  15,316 (99.2%)\n",
      "Pairs written (passing filters):        15,435 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,102,935 bp\n",
      "  Read 1:     4,552,104 bp\n",
      "  Read 2:     4,550,831 bp\n",
      "Total written (filtered):      8,504,575 bp (93.4%)\n",
      "  Read 1:     4,244,395 bp\n",
      "  Read 2:     4,260,180 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 15391 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t155\t0.0\t1\t0 151 4\n",
      "20\t15179\t0.0\t2\t14696 478 5\n",
      "21\t50\t0.0\t2\t0 48 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 15316 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t393\t0.0\t1\t0 393\n",
      "19\t14883\t0.0\t1\t14412 471\n",
      "20\t40\t0.0\t1\t0 40\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S167_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S167_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S167_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S167_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S167_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S167_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S167_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S167_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,252 reads  @     62.8 µs/read;   0.96 M reads/minute\n",
      "Finished in 2.47 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,252\n",
      "  Read 1 with adapter:                  39,239 (100.0%)\n",
      "  Read 2 with adapter:                  39,025 (99.4%)\n",
      "Pairs written (passing filters):        39,252 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,156,843 bp\n",
      "  Read 1:    11,578,285 bp\n",
      "  Read 2:    11,578,558 bp\n",
      "Total written (filtered):     21,631,988 bp (93.4%)\n",
      "  Read 1:    10,794,129 bp\n",
      "  Read 2:    10,837,859 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39239 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t723\t0.0\t1\t0 704 19\n",
      "20\t38400\t0.0\t2\t37458 930 12\n",
      "21\t69\t0.0\t2\t0 69\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39025 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t846\t0.0\t1\t0 846\n",
      "19\t38109\t0.0\t1\t36979 1130\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S168_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S168_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S168_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S168_85_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S168_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S168_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S168_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S168_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,421 reads  @     64.5 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.48 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,421\n",
      "  Read 1 with adapter:                  38,413 (100.0%)\n",
      "  Read 2 with adapter:                  38,210 (99.5%)\n",
      "Pairs written (passing filters):        38,421 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,667,347 bp\n",
      "  Read 1:    11,334,034 bp\n",
      "  Read 2:    11,333,313 bp\n",
      "Total written (filtered):     21,174,254 bp (93.4%)\n",
      "  Read 1:    10,566,321 bp\n",
      "  Read 2:    10,607,933 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38413 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t626\t0.0\t1\t0 607 19\n",
      "20\t37657\t0.0\t2\t36737 907 13\n",
      "21\t65\t0.0\t2\t0 62 3\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38210 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t675\t0.0\t1\t0 675\n",
      "19\t37470\t0.0\t1\t36382 1088\n",
      "20\t65\t0.0\t1\t0 65\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S169_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S169_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S169_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S169_87_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S169_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S169_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S169_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S169_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        57,932 reads  @     62.5 µs/read;   0.96 M reads/minute\n",
      "Finished in 3.63 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,932\n",
      "  Read 1 with adapter:                  57,919 (100.0%)\n",
      "  Read 2 with adapter:                  57,618 (99.5%)\n",
      "Pairs written (passing filters):        57,932 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,179,072 bp\n",
      "  Read 1:    17,089,771 bp\n",
      "  Read 2:    17,089,301 bp\n",
      "Total written (filtered):     31,927,928 bp (93.4%)\n",
      "  Read 1:    15,932,312 bp\n",
      "  Read 2:    15,995,616 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57919 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t46\t0.0\t1\t0 0 46\n",
      "19\t1016\t0.0\t1\t0 982 34\n",
      "20\t56715\t0.0\t2\t55234 1448 33\n",
      "21\t97\t0.0\t2\t0 94 3\n",
      "22\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57618 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1167\t0.0\t1\t0 1167\n",
      "19\t56341\t0.0\t1\t54674 1667\n",
      "20\t110\t0.0\t1\t0 110\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S170_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S170_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S170_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S170_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S170_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S170_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S170_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S170_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        63,551 reads  @     65.8 µs/read;   0.91 M reads/minute\n",
      "Finished in 4.19 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,551\n",
      "  Read 1 with adapter:                  63,521 (100.0%)\n",
      "  Read 2 with adapter:                  63,186 (99.4%)\n",
      "Pairs written (passing filters):        63,551 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,493,855 bp\n",
      "  Read 1:    18,746,473 bp\n",
      "  Read 2:    18,747,382 bp\n",
      "Total written (filtered):     35,024,931 bp (93.4%)\n",
      "  Read 1:    17,477,017 bp\n",
      "  Read 2:    17,547,914 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63521 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t1098\t0.0\t1\t0 1058 40\n",
      "20\t62264\t0.0\t2\t60599 1641 24\n",
      "21\t88\t0.0\t2\t0 86 2\n",
      "22\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63186 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1208\t0.0\t1\t0 1208\n",
      "19\t61836\t0.0\t1\t60061 1775\n",
      "20\t142\t0.0\t1\t0 142\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S171_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S171_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S171_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S171_91_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S171_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S171_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S171_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S171_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        64,652 reads  @     64.2 µs/read;   0.93 M reads/minute\n",
      "Finished in 4.16 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,652\n",
      "  Read 1 with adapter:                  64,621 (100.0%)\n",
      "  Read 2 with adapter:                  64,341 (99.5%)\n",
      "Pairs written (passing filters):        64,651 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,139,806 bp\n",
      "  Read 1:    19,071,327 bp\n",
      "  Read 2:    19,068,479 bp\n",
      "Total written (filtered):     35,626,645 bp (93.4%)\n",
      "  Read 1:    17,779,629 bp\n",
      "  Read 2:    17,847,016 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64621 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t48\t0.0\t1\t0 0 48\n",
      "19\t1109\t0.0\t1\t0 1074 35\n",
      "20\t63309\t0.0\t2\t61623 1664 22\n",
      "21\t102\t0.0\t2\t0 101 1\n",
      "22\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 64341 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1131\t0.0\t1\t0 1131\n",
      "19\t63095\t0.0\t1\t61243 1852\n",
      "20\t115\t0.0\t1\t0 115\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S172_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S172_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S172_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S172_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S172_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S172_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S172_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S172_93_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        64,322 reads  @     64.9 µs/read;   0.92 M reads/minute\n",
      "Finished in 4.18 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,322\n",
      "  Read 1 with adapter:                  64,301 (100.0%)\n",
      "  Read 2 with adapter:                  63,934 (99.4%)\n",
      "Pairs written (passing filters):        64,322 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,937,106 bp\n",
      "  Read 1:    18,974,582 bp\n",
      "  Read 2:    18,962,524 bp\n",
      "Total written (filtered):     35,438,588 bp (93.4%)\n",
      "  Read 1:    17,689,555 bp\n",
      "  Read 2:    17,749,033 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64301 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t43\t0.0\t1\t0 0 43\n",
      "19\t1113\t0.0\t1\t0 1085 28\n",
      "20\t62996\t0.0\t2\t61389 1582 25\n",
      "21\t92\t0.0\t2\t0 91 1\n",
      "22\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63934 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1389\t0.0\t1\t0 1389\n",
      "19\t62411\t0.0\t1\t60589 1822\n",
      "20\t134\t0.0\t1\t0 134\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S173_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S173_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S173_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S173_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S173_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S173_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S173_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S173_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        65,837 reads  @     68.8 µs/read;   0.87 M reads/minute\n",
      "Finished in 4.54 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,837\n",
      "  Read 1 with adapter:                  65,811 (100.0%)\n",
      "  Read 2 with adapter:                  65,413 (99.4%)\n",
      "Pairs written (passing filters):        65,837 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,819,287 bp\n",
      "  Read 1:    19,421,148 bp\n",
      "  Read 2:    19,398,139 bp\n",
      "Total written (filtered):     36,262,317 bp (93.4%)\n",
      "  Read 1:    18,105,985 bp\n",
      "  Read 2:    18,156,332 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 65811 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t40\t0.0\t1\t0 0 40\n",
      "19\t1146\t0.0\t1\t0 1100 46\n",
      "20\t64487\t0.0\t2\t62836 1634 17\n",
      "21\t107\t0.0\t2\t0 105 2\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 65413 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1130\t0.0\t1\t0 1130\n",
      "19\t64193\t0.0\t1\t62588 1605\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S174_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S174_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S174_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S174_97_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S174_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S174_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S174_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S174_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        69,412 reads  @     71.3 µs/read;   0.84 M reads/minute\n",
      "Finished in 4.96 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,412\n",
      "  Read 1 with adapter:                  69,402 (100.0%)\n",
      "  Read 2 with adapter:                  67,801 (97.7%)\n",
      "Pairs written (passing filters):        69,412 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,866,038 bp\n",
      "  Read 1:    20,476,296 bp\n",
      "  Read 2:    20,389,742 bp\n",
      "Total written (filtered):     38,191,431 bp (93.5%)\n",
      "  Read 1:    19,089,357 bp\n",
      "  Read 2:    19,102,074 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 69402 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t45\t0.0\t1\t0 0 45\n",
      "19\t1224\t0.0\t1\t0 1176 48\n",
      "20\t67970\t0.0\t2\t66094 1849 27\n",
      "21\t113\t0.0\t2\t0 110 3\n",
      "22\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 67801 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t641\t0.0\t1\t0 641\n",
      "19\t67070\t0.0\t1\t65588 1482\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S175_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S175_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S175_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S175_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S175_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S175_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S175_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S175_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        33,226 reads  @     64.8 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.16 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,226\n",
      "  Read 1 with adapter:                  33,218 (100.0%)\n",
      "  Read 2 with adapter:                  33,017 (99.4%)\n",
      "Pairs written (passing filters):        33,226 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,600,462 bp\n",
      "  Read 1:     9,801,617 bp\n",
      "  Read 2:     9,798,845 bp\n",
      "Total written (filtered):     18,309,746 bp (93.4%)\n",
      "  Read 1:     9,137,623 bp\n",
      "  Read 2:     9,172,123 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33218 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t423\t0.0\t1\t0 415 8\n",
      "20\t32695\t0.0\t2\t31981 703 11\n",
      "21\t63\t0.0\t2\t0 62 1\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t670\t0.0\t1\t0 670\n",
      "19\t32278\t0.0\t1\t31476 802\n",
      "20\t69\t0.0\t1\t0 69\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S176_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S176_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S176_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S176_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S176_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S176_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S176_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S176_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        55,491 reads  @     60.0 µs/read;   1.00 M reads/minute\n",
      "Finished in 3.33 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,491\n",
      "  Read 1 with adapter:                  55,481 (100.0%)\n",
      "  Read 2 with adapter:                  55,199 (99.5%)\n",
      "Pairs written (passing filters):        55,491 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,738,191 bp\n",
      "  Read 1:    16,368,807 bp\n",
      "  Read 2:    16,369,384 bp\n",
      "Total written (filtered):     30,581,374 bp (93.4%)\n",
      "  Read 1:    15,259,847 bp\n",
      "  Read 2:    15,321,527 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55481 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t765\t0.0\t1\t0 742 23\n",
      "20\t54577\t0.0\t2\t53181 1368 28\n",
      "21\t81\t0.0\t2\t0 80 1\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55199 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1007\t0.0\t1\t0 1007\n",
      "19\t54109\t0.0\t1\t52686 1423\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S177_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S177_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S177_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S177_103_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S177_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S177_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S177_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S177_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        47,846 reads  @     60.6 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.90 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,846\n",
      "  Read 1 with adapter:                  47,835 (100.0%)\n",
      "  Read 2 with adapter:                  47,599 (99.5%)\n",
      "Pairs written (passing filters):        47,846 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,229,003 bp\n",
      "  Read 1:    14,114,549 bp\n",
      "  Read 2:    14,114,454 bp\n",
      "Total written (filtered):     26,369,391 bp (93.4%)\n",
      "  Read 1:    13,158,487 bp\n",
      "  Read 2:    13,210,904 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47835 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t693\t0.0\t1\t0 673 20\n",
      "20\t47020\t0.0\t2\t45833 1171 16\n",
      "21\t69\t0.0\t2\t0 68 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47599 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t919\t0.0\t1\t0 919\n",
      "19\t46592\t0.0\t1\t45343 1249\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S178_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S178_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S178_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S178_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S178_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S178_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S178_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S178_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,024 reads  @     66.9 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.42 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,024\n",
      "  Read 1 with adapter:                  36,017 (100.0%)\n",
      "  Read 2 with adapter:                  35,803 (99.4%)\n",
      "Pairs written (passing filters):        36,024 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,253,563 bp\n",
      "  Read 1:    10,626,832 bp\n",
      "  Read 2:    10,626,731 bp\n",
      "Total written (filtered):     19,854,050 bp (93.4%)\n",
      "  Read 1:     9,906,963 bp\n",
      "  Read 2:     9,947,087 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t521\t0.0\t1\t0 499 22\n",
      "20\t35396\t0.0\t2\t34462 919 15\n",
      "21\t54\t0.0\t2\t0 53 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35803 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t700\t0.0\t1\t0 700\n",
      "19\t35016\t0.0\t1\t34026 990\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S179_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S179_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S179_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S179_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S179_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S179_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S179_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S179_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        55,784 reads  @     69.1 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.86 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,784\n",
      "  Read 1 with adapter:                  55,776 (100.0%)\n",
      "  Read 2 with adapter:                  55,509 (99.5%)\n",
      "Pairs written (passing filters):        55,782 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,910,588 bp\n",
      "  Read 1:    16,456,180 bp\n",
      "  Read 2:    16,454,408 bp\n",
      "Total written (filtered):     30,741,371 bp (93.4%)\n",
      "  Read 1:    15,340,711 bp\n",
      "  Read 2:    15,400,660 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55776 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t709\t0.0\t1\t0 690 19\n",
      "20\t54928\t0.0\t2\t53493 1414 21\n",
      "21\t102\t0.0\t2\t0 99 3\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55509 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1021\t0.0\t1\t0 1021\n",
      "19\t54390\t0.0\t1\t52911 1479\n",
      "20\t98\t0.0\t1\t0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S180_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S180_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S180_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S180_109_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S180_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S180_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S180_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S180_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        61,635 reads  @     65.9 µs/read;   0.91 M reads/minute\n",
      "Finished in 4.07 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,635\n",
      "  Read 1 with adapter:                  61,624 (100.0%)\n",
      "  Read 2 with adapter:                  61,274 (99.4%)\n",
      "Pairs written (passing filters):        61,635 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,348,608 bp\n",
      "  Read 1:    18,182,217 bp\n",
      "  Read 2:    18,166,391 bp\n",
      "Total written (filtered):     33,953,881 bp (93.4%)\n",
      "  Read 1:    16,950,457 bp\n",
      "  Read 2:    17,003,424 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 61624 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t38\t0.0\t1\t0 0 38\n",
      "19\t856\t0.0\t1\t0 833 23\n",
      "20\t60564\t0.0\t2\t59022 1521 21\n",
      "21\t120\t0.0\t2\t0 118 2\n",
      "22\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61274 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1348\t0.0\t1\t0 1348\n",
      "19\t59817\t0.0\t1\t58217 1600\n",
      "20\t109\t0.0\t1\t0 109\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S181_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S181_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S181_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S181_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S181_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S181_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S181_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S181_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        51,181 reads  @     65.7 µs/read;   0.91 M reads/minute\n",
      "Finished in 3.37 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,181\n",
      "  Read 1 with adapter:                  51,169 (100.0%)\n",
      "  Read 2 with adapter:                  50,828 (99.3%)\n",
      "Pairs written (passing filters):        51,181 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,176,605 bp\n",
      "  Read 1:    15,098,234 bp\n",
      "  Read 2:    15,078,371 bp\n",
      "Total written (filtered):     28,188,917 bp (93.4%)\n",
      "  Read 1:    14,075,438 bp\n",
      "  Read 2:    14,113,479 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51169 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t659\t0.0\t1\t0 632 27\n",
      "20\t50375\t0.0\t2\t48984 1376 15\n",
      "21\t79\t0.0\t2\t0 79\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50828 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t907\t0.0\t1\t0 907\n",
      "19\t49854\t0.0\t1\t48595 1259\n",
      "20\t67\t0.0\t1\t0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S182_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S182_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S182_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S182_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S182_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S182_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S182_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S182_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        63,955 reads  @     69.0 µs/read;   0.87 M reads/minute\n",
      "Finished in 4.42 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,955\n",
      "  Read 1 with adapter:                  63,936 (100.0%)\n",
      "  Read 2 with adapter:                  62,274 (97.4%)\n",
      "Pairs written (passing filters):        63,955 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,637,405 bp\n",
      "  Read 1:    18,866,142 bp\n",
      "  Read 2:    18,771,263 bp\n",
      "Total written (filtered):     35,176,750 bp (93.5%)\n",
      "  Read 1:    17,588,174 bp\n",
      "  Read 2:    17,588,576 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63936 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t38\t0.0\t1\t0 0 38\n",
      "19\t892\t0.0\t1\t0 867 25\n",
      "20\t62841\t0.0\t2\t61116 1691 34\n",
      "21\t114\t0.0\t2\t0 112 2\n",
      "22\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 62274 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t600\t0.0\t1\t0 600\n",
      "19\t61593\t0.0\t1\t60185 1408\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S183_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S183_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S183_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S183_115_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S183_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S183_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S183_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S183_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        29,642 reads  @     62.6 µs/read;   0.96 M reads/minute\n",
      "Finished in 1.86 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,642\n",
      "  Read 1 with adapter:                  29,635 (100.0%)\n",
      "  Read 2 with adapter:                  29,457 (99.4%)\n",
      "Pairs written (passing filters):        29,642 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,488,554 bp\n",
      "  Read 1:     8,744,206 bp\n",
      "  Read 2:     8,744,348 bp\n",
      "Total written (filtered):     16,337,146 bp (93.4%)\n",
      "  Read 1:     8,151,967 bp\n",
      "  Read 2:     8,185,179 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 29635 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t512\t0.0\t1\t0 490 22\n",
      "20\t29045\t0.0\t2\t28256 779 10\n",
      "21\t45\t0.0\t2\t0 44 1\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 29457 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t580\t0.0\t1\t0 580\n",
      "19\t28811\t0.0\t1\t27987 824\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S184_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S184_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S184_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S184_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S184_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S184_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S184_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S184_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        71,086 reads  @     62.5 µs/read;   0.96 M reads/minute\n",
      "Finished in 4.45 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             71,086\n",
      "  Read 1 with adapter:                  71,063 (100.0%)\n",
      "  Read 2 with adapter:                  70,632 (99.4%)\n",
      "Pairs written (passing filters):        71,086 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,939,150 bp\n",
      "  Read 1:    20,969,071 bp\n",
      "  Read 2:    20,970,079 bp\n",
      "Total written (filtered):     39,178,233 bp (93.4%)\n",
      "  Read 1:    19,548,994 bp\n",
      "  Read 2:    19,629,239 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 71063 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t48\t0.0\t1\t0 0 48\n",
      "19\t1255\t0.0\t1\t0 1209 46\n",
      "20\t69625\t0.0\t2\t67662 1930 33\n",
      "21\t102\t0.0\t2\t0 98 4\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 70632 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1268\t0.0\t1\t0 1268\n",
      "19\t69264\t0.0\t1\t67265 1999\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S185_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S185_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S185_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S185_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S185_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S185_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S185_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S185_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,018 reads  @     60.8 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.62 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,018\n",
      "  Read 1 with adapter:                  43,010 (100.0%)\n",
      "  Read 2 with adapter:                  42,778 (99.4%)\n",
      "Pairs written (passing filters):        43,018 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,380,383 bp\n",
      "  Read 1:    12,690,159 bp\n",
      "  Read 2:    12,690,224 bp\n",
      "Total written (filtered):     23,708,848 bp (93.4%)\n",
      "  Read 1:    11,830,636 bp\n",
      "  Read 2:    11,878,212 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43010 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t726\t0.0\t1\t0 708 18\n",
      "20\t42168\t0.0\t2\t41005 1144 19\n",
      "21\t75\t0.0\t2\t0 74 1\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42778 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t844\t0.0\t1\t0 844\n",
      "19\t41860\t0.0\t1\t40717 1143\n",
      "20\t74\t0.0\t1\t0 74\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S186_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S186_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S186_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S186_121_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S186_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S186_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S186_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S186_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,822 reads  @     62.5 µs/read;   0.96 M reads/minute\n",
      "Finished in 3.18 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,822\n",
      "  Read 1 with adapter:                  50,808 (100.0%)\n",
      "  Read 2 with adapter:                  50,426 (99.2%)\n",
      "Pairs written (passing filters):        50,822 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,984,403 bp\n",
      "  Read 1:    14,991,966 bp\n",
      "  Read 2:    14,992,437 bp\n",
      "Total written (filtered):     28,011,842 bp (93.4%)\n",
      "  Read 1:    13,976,616 bp\n",
      "  Read 2:    14,035,226 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50808 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t34\t0.0\t1\t0 0 34\n",
      "19\t856\t0.0\t1\t0 814 42\n",
      "20\t49822\t0.0\t2\t48347 1457 18\n",
      "21\t78\t0.0\t2\t0 77 1\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50426 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t980\t0.0\t1\t0 980\n",
      "19\t49349\t0.0\t1\t47759 1590\n",
      "20\t97\t0.0\t1\t0 97\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S187_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S187_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S187_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S187_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S187_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S187_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S187_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S187_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        50,657 reads  @     66.6 µs/read;   0.90 M reads/minute\n",
      "Finished in 3.38 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,657\n",
      "  Read 1 with adapter:                  50,641 (100.0%)\n",
      "  Read 2 with adapter:                  50,262 (99.2%)\n",
      "Pairs written (passing filters):        50,655 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,884,202 bp\n",
      "  Read 1:    14,943,421 bp\n",
      "  Read 2:    14,940,781 bp\n",
      "Total written (filtered):     27,917,428 bp (93.4%)\n",
      "  Read 1:    13,930,833 bp\n",
      "  Read 2:    13,986,595 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50641 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t854\t0.0\t1\t0 827 27\n",
      "20\t49651\t0.0\t2\t48291 1340 20\n",
      "21\t80\t0.0\t2\t0 79 1\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50262 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t885\t0.0\t1\t0 885\n",
      "19\t49284\t0.0\t1\t47715 1569\n",
      "20\t93\t0.0\t1\t0 93\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S188_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S188_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S188_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S188_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S188_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S188_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S188_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S188_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        63,776 reads  @     66.2 µs/read;   0.91 M reads/minute\n",
      "Finished in 4.23 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,776\n",
      "  Read 1 with adapter:                  63,759 (100.0%)\n",
      "  Read 2 with adapter:                  63,283 (99.2%)\n",
      "Pairs written (passing filters):        63,776 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,612,015 bp\n",
      "  Read 1:    18,812,231 bp\n",
      "  Read 2:    18,799,784 bp\n",
      "Total written (filtered):     35,136,698 bp (93.4%)\n",
      "  Read 1:    17,538,044 bp\n",
      "  Read 2:    17,598,654 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63759 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t43\t0.0\t1\t0 0 43\n",
      "19\t1088\t0.0\t1\t0 1045 43\n",
      "20\t62484\t0.0\t2\t60788 1663 33\n",
      "21\t107\t0.0\t2\t0 104 3\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63283 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1333\t0.0\t1\t0 1333\n",
      "19\t61864\t0.0\t1\t60114 1750\n",
      "20\t86\t0.0\t1\t0 86\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S189_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S189_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S189_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S189_127_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S189_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S189_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S189_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S189_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        55,204 reads  @     70.5 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.89 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,204\n",
      "  Read 1 with adapter:                  55,189 (100.0%)\n",
      "  Read 2 with adapter:                  54,749 (99.2%)\n",
      "Pairs written (passing filters):        55,204 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,548,946 bp\n",
      "  Read 1:    16,283,487 bp\n",
      "  Read 2:    16,265,459 bp\n",
      "Total written (filtered):     30,406,749 bp (93.4%)\n",
      "  Read 1:    15,180,657 bp\n",
      "  Read 2:    15,226,092 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55189 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t46\t0.0\t1\t0 0 46\n",
      "19\t1001\t0.0\t1\t0 968 33\n",
      "20\t54028\t0.0\t2\t52364 1637 27\n",
      "21\t85\t0.0\t2\t0 80 5\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54749 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t943\t0.0\t1\t0 943\n",
      "19\t53727\t0.0\t1\t52271 1456\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S190_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S190_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S190_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S190_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S190_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S190_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S190_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S190_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        74,229 reads  @     67.7 µs/read;   0.89 M reads/minute\n",
      "Finished in 5.03 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,229\n",
      "  Read 1 with adapter:                  74,206 (100.0%)\n",
      "  Read 2 with adapter:                  71,965 (96.9%)\n",
      "Pairs written (passing filters):        74,229 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,664,560 bp\n",
      "  Read 1:    21,895,884 bp\n",
      "  Read 2:    21,768,676 bp\n",
      "Total written (filtered):     40,814,876 bp (93.5%)\n",
      "  Read 1:    20,412,933 bp\n",
      "  Read 2:    20,401,943 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 74206 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t53\t0.0\t1\t0 0 53\n",
      "19\t1247\t0.0\t1\t0 1204 43\n",
      "20\t72758\t0.0\t2\t70640 2077 41\n",
      "21\t112\t0.0\t2\t0 111 1\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 71965 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t698\t0.0\t1\t0 698\n",
      "19\t71171\t0.0\t1\t69418 1753\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S191_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S191_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S191_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S191_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S191_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S191_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S191_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S191_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        59,950 reads  @     66.0 µs/read;   0.91 M reads/minute\n",
      "Finished in 3.96 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,950\n",
      "  Read 1 with adapter:                  59,935 (100.0%)\n",
      "  Read 2 with adapter:                  59,608 (99.4%)\n",
      "Pairs written (passing filters):        59,950 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,367,015 bp\n",
      "  Read 1:    17,684,249 bp\n",
      "  Read 2:    17,682,766 bp\n",
      "Total written (filtered):     33,037,533 bp (93.4%)\n",
      "  Read 1:    16,486,194 bp\n",
      "  Read 2:    16,551,339 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59935 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t731\t0.0\t1\t0 712 19\n",
      "20\t59065\t0.0\t2\t57436 1610 19\n",
      "21\t88\t0.0\t2\t0 87 1\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59608 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1253\t0.0\t1\t0 1253\n",
      "19\t58227\t0.0\t1\t56659 1568\n",
      "20\t128\t0.0\t1\t0 128\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S192_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S192_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S192_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S192_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S192_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S192_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S192_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S192_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        69,485 reads  @     64.4 µs/read;   0.93 M reads/minute\n",
      "Finished in 4.48 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,485\n",
      "  Read 1 with adapter:                  69,456 (100.0%)\n",
      "  Read 2 with adapter:                  69,174 (99.6%)\n",
      "Pairs written (passing filters):        69,485 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,993,447 bp\n",
      "  Read 1:    20,495,559 bp\n",
      "  Read 2:    20,497,888 bp\n",
      "Total written (filtered):     38,291,850 bp (93.4%)\n",
      "  Read 1:    19,107,144 bp\n",
      "  Read 2:    19,184,706 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 69456 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t36\t0.0\t1\t0 0 36\n",
      "19\t791\t0.0\t1\t0 765 26\n",
      "20\t68499\t0.0\t2\t66641 1830 28\n",
      "21\t102\t0.0\t2\t0 101 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 69174 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1249\t0.0\t1\t0 1249\n",
      "19\t67800\t0.0\t1\t66065 1735\n",
      "20\t125\t0.0\t1\t0 125\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S193_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S193_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S193_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S193_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S193_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S193_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S193_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S193_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        59,955 reads  @     61.6 µs/read;   0.97 M reads/minute\n",
      "Finished in 3.70 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,955\n",
      "  Read 1 with adapter:                  59,933 (100.0%)\n",
      "  Read 2 with adapter:                  59,692 (99.6%)\n",
      "Pairs written (passing filters):        59,955 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,371,476 bp\n",
      "  Read 1:    17,685,914 bp\n",
      "  Read 2:    17,685,562 bp\n",
      "Total written (filtered):     33,040,374 bp (93.4%)\n",
      "  Read 1:    16,487,875 bp\n",
      "  Read 2:    16,552,499 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59933 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t701\t0.0\t1\t0 683 18\n",
      "20\t59093\t0.0\t2\t57493 1575 25\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59692 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1192\t0.0\t1\t0 1192\n",
      "19\t58393\t0.0\t1\t56910 1483\n",
      "20\t107\t0.0\t1\t0 107\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S194_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S194_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S194_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S194_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S194_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S194_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S194_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S194_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,825 reads  @     63.2 µs/read;   0.95 M reads/minute\n",
      "Finished in 3.41 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,825\n",
      "  Read 1 with adapter:                  53,807 (100.0%)\n",
      "  Read 2 with adapter:                  53,606 (99.6%)\n",
      "Pairs written (passing filters):        53,825 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,756,042 bp\n",
      "  Read 1:    15,877,711 bp\n",
      "  Read 2:    15,878,331 bp\n",
      "Total written (filtered):     29,662,891 bp (93.4%)\n",
      "  Read 1:    14,802,121 bp\n",
      "  Read 2:    14,860,770 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53807 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t624\t0.0\t1\t0 608 16\n",
      "20\t53063\t0.0\t2\t51569 1458 36\n",
      "21\t74\t0.0\t2\t0 72 2\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53606 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1067\t0.0\t1\t0 1067\n",
      "19\t52425\t0.0\t1\t51015 1410\n",
      "20\t114\t0.0\t1\t0 114\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S195_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S195_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S195_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S195_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S195_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S195_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S195_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S195_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        91,084 reads  @     62.9 µs/read;   0.95 M reads/minute\n",
      "Finished in 5.73 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             91,084\n",
      "  Read 1 with adapter:                  91,052 (100.0%)\n",
      "  Read 2 with adapter:                  90,718 (99.6%)\n",
      "Pairs written (passing filters):        91,084 (100.0%)\n",
      "\n",
      "Total basepairs processed:    53,736,283 bp\n",
      "  Read 1:    26,868,108 bp\n",
      "  Read 2:    26,868,175 bp\n",
      "Total written (filtered):     50,194,063 bp (93.4%)\n",
      "  Read 1:    25,048,059 bp\n",
      "  Read 2:    25,146,004 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 91052 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t52\t0.0\t1\t0 0 52\n",
      "19\t1083\t0.0\t1\t0 1044 39\n",
      "20\t89760\t0.0\t2\t87141 2575 44\n",
      "21\t118\t0.0\t2\t0 115 3\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 90718 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1607\t0.0\t1\t0 1607\n",
      "19\t88975\t0.0\t1\t86749 2226\n",
      "20\t136\t0.0\t1\t0 136\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S196_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S196_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S196_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S196_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S196_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S196_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S196_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S196_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        60,175 reads  @     63.7 µs/read;   0.94 M reads/minute\n",
      "Finished in 3.84 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,175\n",
      "  Read 1 with adapter:                  60,159 (100.0%)\n",
      "  Read 2 with adapter:                  59,890 (99.5%)\n",
      "Pairs written (passing filters):        60,175 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,489,132 bp\n",
      "  Read 1:    17,750,526 bp\n",
      "  Read 2:    17,738,606 bp\n",
      "Total written (filtered):     33,149,970 bp (93.4%)\n",
      "  Read 1:    16,548,065 bp\n",
      "  Read 2:    16,601,905 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60159 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t49\t0.0\t1\t0 0 49\n",
      "19\t768\t0.0\t1\t0 743 25\n",
      "20\t59226\t0.0\t2\t57494 1704 28\n",
      "21\t85\t0.0\t2\t0 85\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59890 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1312\t0.0\t1\t0 1312\n",
      "19\t58475\t0.0\t1\t57090 1385\n",
      "20\t103\t0.0\t1\t0 103\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S197_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S197_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S197_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S197_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S197_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S197_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S197_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S197_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        45,439 reads  @     62.8 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.86 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,439\n",
      "  Read 1 with adapter:                  45,425 (100.0%)\n",
      "  Read 2 with adapter:                  45,168 (99.4%)\n",
      "Pairs written (passing filters):        45,439 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,793,318 bp\n",
      "  Read 1:    13,403,449 bp\n",
      "  Read 2:    13,389,869 bp\n",
      "Total written (filtered):     25,027,805 bp (93.4%)\n",
      "  Read 1:    12,495,431 bp\n",
      "  Read 2:    12,532,374 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45425 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t516\t0.0\t1\t0 491 25\n",
      "20\t44816\t0.0\t2\t43512 1279 25\n",
      "21\t64\t0.0\t2\t0 62 2\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45168 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t763\t0.0\t1\t0 763\n",
      "19\t44339\t0.0\t1\t43393 946\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S198_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S198_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S198_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S198_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S198_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S198_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S198_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S198_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        62,513 reads  @     65.6 µs/read;   0.92 M reads/minute\n",
      "Finished in 4.10 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,513\n",
      "  Read 1 with adapter:                  62,488 (100.0%)\n",
      "  Read 2 with adapter:                  60,903 (97.4%)\n",
      "Pairs written (passing filters):        62,513 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,787,153 bp\n",
      "  Read 1:    18,438,883 bp\n",
      "  Read 2:    18,348,270 bp\n",
      "Total written (filtered):     34,381,392 bp (93.5%)\n",
      "  Read 1:    17,189,788 bp\n",
      "  Read 2:    17,191,604 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 62488 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t34\t0.0\t1\t0 0 34\n",
      "19\t743\t0.0\t1\t0 721 22\n",
      "20\t61591\t0.0\t2\t59802 1754 35\n",
      "21\t94\t0.0\t2\t0 89 5\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 60903 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t569\t0.0\t1\t0 569\n",
      "19\t60256\t0.0\t1\t59063 1193\n",
      "20\t78\t0.0\t1\t0 78\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S199_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S199_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S199_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S199_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S199_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S199_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S199_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S199_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        24,643 reads  @     61.4 µs/read;   0.98 M reads/minute\n",
      "Finished in 1.52 s (62 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,643\n",
      "  Read 1 with adapter:                  24,638 (100.0%)\n",
      "  Read 2 with adapter:                  24,535 (99.6%)\n",
      "Pairs written (passing filters):        24,643 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,539,220 bp\n",
      "  Read 1:     7,269,595 bp\n",
      "  Read 2:     7,269,625 bp\n",
      "Total written (filtered):     13,580,916 bp (93.4%)\n",
      "  Read 1:     6,776,962 bp\n",
      "  Read 2:     6,803,954 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 24638 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t217\t0.0\t1\t0 209 8\n",
      "20\t24332\t0.0\t2\t23857 471 4\n",
      "21\t52\t0.0\t2\t0 51 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 24535 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t540\t0.0\t1\t0 540\n",
      "19\t23949\t0.0\t1\t23336 613\n",
      "20\t46\t0.0\t1\t0 46\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S200_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S200_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S200_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S200_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S200_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S200_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S200_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S200_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        58,495 reads  @     58.4 µs/read;   1.03 M reads/minute\n",
      "Finished in 3.42 s (58 us/read; 1.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,495\n",
      "  Read 1 with adapter:                  58,485 (100.0%)\n",
      "  Read 2 with adapter:                  58,262 (99.6%)\n",
      "Pairs written (passing filters):        58,495 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,510,936 bp\n",
      "  Read 1:    17,255,716 bp\n",
      "  Read 2:    17,255,220 bp\n",
      "Total written (filtered):     32,235,515 bp (93.4%)\n",
      "  Read 1:    16,086,333 bp\n",
      "  Read 2:    16,149,182 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58485 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t526\t0.0\t1\t0 515 11\n",
      "20\t57754\t0.0\t2\t56535 1206 13\n",
      "21\t109\t0.0\t2\t0 106 3\n",
      "22\t73\t0.0\t2\t0 0 73\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 58262 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1043\t0.0\t1\t0 1043\n",
      "19\t57116\t0.0\t1\t55652 1464\n",
      "20\t103\t0.0\t1\t0 103\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S201_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S201_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S201_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S201_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S201_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S201_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S201_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S201_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        63,299 reads  @     58.1 µs/read;   1.03 M reads/minute\n",
      "Finished in 3.68 s (58 us/read; 1.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,299\n",
      "  Read 1 with adapter:                  63,291 (100.0%)\n",
      "  Read 2 with adapter:                  63,050 (99.6%)\n",
      "Pairs written (passing filters):        63,299 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,345,746 bp\n",
      "  Read 1:    18,672,625 bp\n",
      "  Read 2:    18,673,121 bp\n",
      "Total written (filtered):     34,883,508 bp (93.4%)\n",
      "  Read 1:    17,407,218 bp\n",
      "  Read 2:    17,476,290 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63291 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t567\t0.0\t1\t0 547 20\n",
      "20\t62520\t0.0\t2\t61223 1283 14\n",
      "21\t130\t0.0\t2\t0 127 3\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63050 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1241\t0.0\t1\t0 1241\n",
      "19\t61687\t0.0\t1\t60171 1516\n",
      "20\t122\t0.0\t1\t0 122\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S202_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S202_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S202_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S202_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S202_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S202_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S202_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S202_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,705 reads  @     60.8 µs/read;   0.99 M reads/minute\n",
      "Finished in 3.03 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,705\n",
      "  Read 1 with adapter:                  49,693 (100.0%)\n",
      "  Read 2 with adapter:                  49,469 (99.5%)\n",
      "Pairs written (passing filters):        49,705 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,325,458 bp\n",
      "  Read 1:    14,662,673 bp\n",
      "  Read 2:    14,662,785 bp\n",
      "Total written (filtered):     27,392,860 bp (93.4%)\n",
      "  Read 1:    13,669,108 bp\n",
      "  Read 2:    13,723,752 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49693 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t436\t0.0\t1\t0 422 14\n",
      "20\t49083\t0.0\t2\t47984 1091 8\n",
      "21\t87\t0.0\t2\t0 84 3\n",
      "22\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49469 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t986\t0.0\t1\t0 986\n",
      "19\t48375\t0.0\t1\t47093 1282\n",
      "20\t108\t0.0\t1\t0 108\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S203_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S203_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S203_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S203_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S203_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S203_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S203_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S203_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,858 reads  @     59.7 µs/read;   1.00 M reads/minute\n",
      "Finished in 3.16 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,858\n",
      "  Read 1 with adapter:                  52,848 (100.0%)\n",
      "  Read 2 with adapter:                  52,632 (99.6%)\n",
      "Pairs written (passing filters):        52,858 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,183,336 bp\n",
      "  Read 1:    15,592,511 bp\n",
      "  Read 2:    15,590,825 bp\n",
      "Total written (filtered):     29,127,473 bp (93.4%)\n",
      "  Read 1:    14,535,826 bp\n",
      "  Read 2:    14,591,647 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52848 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t468\t0.0\t1\t0 456 12\n",
      "20\t52196\t0.0\t2\t51078 1105 13\n",
      "21\t99\t0.0\t2\t0 95 4\n",
      "22\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52632 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t926\t0.0\t1\t0 926\n",
      "19\t51610\t0.0\t1\t50256 1354\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S204_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S204_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S204_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S204_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S204_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S204_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S204_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S204_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        48,770 reads  @     58.2 µs/read;   1.03 M reads/minute\n",
      "Finished in 2.84 s (58 us/read; 1.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,770\n",
      "  Read 1 with adapter:                  48,762 (100.0%)\n",
      "  Read 2 with adapter:                  48,490 (99.4%)\n",
      "Pairs written (passing filters):        48,770 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,764,865 bp\n",
      "  Read 1:    14,386,990 bp\n",
      "  Read 2:    14,377,875 bp\n",
      "Total written (filtered):     26,869,555 bp (93.4%)\n",
      "  Read 1:    13,412,030 bp\n",
      "  Read 2:    13,457,525 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48762 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t429\t0.0\t1\t0 419 10\n",
      "20\t48175\t0.0\t2\t47151 1016 8\n",
      "21\t91\t0.0\t2\t0 87 4\n",
      "22\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48490 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1062\t0.0\t1\t0 1062\n",
      "19\t47326\t0.0\t1\t46063 1263\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S205_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S205_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S205_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S205_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S205_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S205_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S205_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S205_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        49,180 reads  @     59.4 µs/read;   1.01 M reads/minute\n",
      "Finished in 2.92 s (59 us/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,180\n",
      "  Read 1 with adapter:                  49,164 (100.0%)\n",
      "  Read 2 with adapter:                  48,912 (99.5%)\n",
      "Pairs written (passing filters):        49,180 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,002,443 bp\n",
      "  Read 1:    14,507,742 bp\n",
      "  Read 2:    14,494,701 bp\n",
      "Total written (filtered):     27,090,877 bp (93.4%)\n",
      "  Read 1:    13,524,751 bp\n",
      "  Read 2:    13,566,126 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49164 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t446\t0.0\t1\t0 431 15\n",
      "20\t48559\t0.0\t2\t47468 1079 12\n",
      "21\t105\t0.0\t2\t0 102 3\n",
      "22\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48912 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t823\t0.0\t1\t0 823\n",
      "19\t48019\t0.0\t1\t46954 1065\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S208_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S208_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S208_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S208_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S208_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S208_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S208_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S208_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,371 reads  @     61.9 µs/read;   0.97 M reads/minute\n",
      "Finished in 3.37 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,371\n",
      "  Read 1 with adapter:                  54,361 (100.0%)\n",
      "  Read 2 with adapter:                  54,031 (99.4%)\n",
      "Pairs written (passing filters):        54,371 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,063,312 bp\n",
      "  Read 1:    16,038,444 bp\n",
      "  Read 2:    16,024,868 bp\n",
      "Total written (filtered):     29,951,364 bp (93.4%)\n",
      "  Read 1:    14,952,306 bp\n",
      "  Read 2:    14,999,058 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54361 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t50\t0.0\t1\t0 0 50\n",
      "19\t1119\t0.0\t1\t0 1091 28\n",
      "20\t53075\t0.0\t2\t51661 1398 16\n",
      "21\t97\t0.0\t2\t0 92 5\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54031 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t860\t0.0\t1\t0 860\n",
      "19\t53090\t0.0\t1\t51757 1333\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S209_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S209_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S209_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S209_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S209_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S209_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S209_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S209_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        58,380 reads  @     66.2 µs/read;   0.91 M reads/minute\n",
      "Finished in 3.87 s (66 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,380\n",
      "  Read 1 with adapter:                  58,361 (100.0%)\n",
      "  Read 2 with adapter:                  58,046 (99.4%)\n",
      "Pairs written (passing filters):        58,380 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,442,966 bp\n",
      "  Read 1:    17,221,844 bp\n",
      "  Read 2:    17,221,122 bp\n",
      "Total written (filtered):     32,174,968 bp (93.4%)\n",
      "  Read 1:    16,055,888 bp\n",
      "  Read 2:    16,119,080 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58361 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t67\t0.0\t1\t0 0 67\n",
      "19\t1277\t0.0\t1\t0 1240 37\n",
      "20\t56897\t0.0\t2\t55374 1499 24\n",
      "21\t93\t0.0\t2\t0 91 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 58046 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t919\t0.0\t1\t0 919\n",
      "19\t57040\t0.0\t1\t55423 1617\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S210_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S210_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S210_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S210_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S210_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S210_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S210_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S210_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,358 reads  @     66.1 µs/read;   0.91 M reads/minute\n",
      "Finished in 2.01 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,358\n",
      "  Read 1 with adapter:                  30,347 (100.0%)\n",
      "  Read 2 with adapter:                  30,209 (99.5%)\n",
      "Pairs written (passing filters):        30,358 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,909,544 bp\n",
      "  Read 1:     8,955,318 bp\n",
      "  Read 2:     8,954,226 bp\n",
      "Total written (filtered):     16,729,583 bp (93.4%)\n",
      "  Read 1:     8,349,047 bp\n",
      "  Read 2:     8,380,536 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30347 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t674\t0.0\t1\t0 659 15\n",
      "20\t29586\t0.0\t2\t28882 696 8\n",
      "21\t45\t0.0\t2\t0 45\n",
      "22\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30209 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t313\t0.0\t1\t0 313\n",
      "19\t29864\t0.0\t1\t29116 748\n",
      "20\t32\t0.0\t1\t0 32\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S212_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S212_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S212_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S212_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S212_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S212_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S212_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S212_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        63,554 reads  @     64.7 µs/read;   0.93 M reads/minute\n",
      "Finished in 4.12 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,554\n",
      "  Read 1 with adapter:                  63,543 (100.0%)\n",
      "  Read 2 with adapter:                  63,235 (99.5%)\n",
      "Pairs written (passing filters):        63,554 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,483,992 bp\n",
      "  Read 1:    18,748,019 bp\n",
      "  Read 2:    18,735,973 bp\n",
      "Total written (filtered):     35,013,710 bp (93.4%)\n",
      "  Read 1:    17,478,244 bp\n",
      "  Read 2:    17,535,466 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63543 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t61\t0.0\t1\t0 0 61\n",
      "19\t1124\t0.0\t1\t0 1091 33\n",
      "20\t62240\t0.0\t2\t60528 1684 28\n",
      "21\t75\t0.0\t2\t0 71 4\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63235 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1058\t0.0\t1\t0 1058\n",
      "19\t62077\t0.0\t1\t60516 1561\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S213_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S213_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S213_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S213_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S213_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S213_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S213_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S213_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,449 reads  @     68.0 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.16 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,449\n",
      "  Read 1 with adapter:                  46,435 (100.0%)\n",
      "  Read 2 with adapter:                  46,255 (99.6%)\n",
      "Pairs written (passing filters):        46,449 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,402,623 bp\n",
      "  Read 1:    13,700,541 bp\n",
      "  Read 2:    13,702,082 bp\n",
      "Total written (filtered):     25,596,410 bp (93.4%)\n",
      "  Read 1:    12,772,598 bp\n",
      "  Read 2:    12,823,812 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46435 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t834\t0.0\t1\t0 804 30\n",
      "20\t45473\t0.0\t2\t44124 1325 24\n",
      "21\t75\t0.0\t2\t0 74 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46255 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t654\t0.0\t1\t0 654\n",
      "19\t45522\t0.0\t1\t44356 1166\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S214_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S214_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S214_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S214_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S214_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S214_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S214_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S214_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:06        93,624 reads  @     71.3 µs/read;   0.84 M reads/minute\n",
      "Finished in 6.68 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,624\n",
      "  Read 1 with adapter:                  93,593 (100.0%)\n",
      "  Read 2 with adapter:                  93,346 (99.7%)\n",
      "Pairs written (passing filters):        93,624 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,233,050 bp\n",
      "  Read 1:    27,617,065 bp\n",
      "  Read 2:    27,615,985 bp\n",
      "Total written (filtered):     51,590,218 bp (93.4%)\n",
      "  Read 1:    25,746,818 bp\n",
      "  Read 2:    25,843,400 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 93593 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t71\t0.0\t1\t0 0 71\n",
      "19\t1746\t0.0\t1\t0 1693 53\n",
      "20\t91567\t0.0\t2\t88860 2664 43\n",
      "21\t143\t0.0\t2\t0 139 4\n",
      "22\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 93346 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1097\t0.0\t1\t0 1097\n",
      "19\t92141\t0.0\t1\t89782 2359\n",
      "20\t108\t0.0\t1\t0 108\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S216_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S216_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S216_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S216_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S216_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S216_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S216_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S216_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        58,307 reads  @     59.3 µs/read;   1.01 M reads/minute\n",
      "Finished in 3.46 s (59 us/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,307\n",
      "  Read 1 with adapter:                  58,290 (100.0%)\n",
      "  Read 2 with adapter:                  57,980 (99.4%)\n",
      "Pairs written (passing filters):        58,307 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,387,364 bp\n",
      "  Read 1:    17,200,034 bp\n",
      "  Read 2:    17,187,330 bp\n",
      "Total written (filtered):     32,121,307 bp (93.4%)\n",
      "  Read 1:    16,034,714 bp\n",
      "  Read 2:    16,086,593 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58290 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t586\t0.0\t1\t0 568 18\n",
      "20\t57560\t0.0\t2\t56036 1506 18\n",
      "21\t90\t0.0\t2\t0 87 3\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57980 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t989\t0.0\t1\t0 989\n",
      "19\t56885\t0.0\t1\t55534 1351\n",
      "20\t106\t0.0\t1\t0 106\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S217_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S217_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S217_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S217_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S217_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S217_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S217_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S217_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,490 reads  @     60.0 µs/read;   1.00 M reads/minute\n",
      "Finished in 3.22 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,490\n",
      "  Read 1 with adapter:                  53,483 (100.0%)\n",
      "  Read 2 with adapter:                  53,283 (99.6%)\n",
      "Pairs written (passing filters):        53,490 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,558,643 bp\n",
      "  Read 1:    15,779,204 bp\n",
      "  Read 2:    15,779,439 bp\n",
      "Total written (filtered):     29,477,803 bp (93.4%)\n",
      "  Read 1:    14,709,959 bp\n",
      "  Read 2:    14,767,844 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53483 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t490\t0.0\t1\t0 473 17\n",
      "20\t52873\t0.0\t2\t51531 1324 18\n",
      "21\t85\t0.0\t2\t0 82 3\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53283 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t845\t0.0\t1\t0 845\n",
      "19\t52375\t0.0\t1\t51130 1245\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S218_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S218_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S218_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S218_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S218_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S218_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S218_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S218_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,007 reads  @     69.5 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.34 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,007\n",
      "  Read 1 with adapter:                  47,988 (100.0%)\n",
      "  Read 2 with adapter:                  47,815 (99.6%)\n",
      "Pairs written (passing filters):        48,007 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,321,504 bp\n",
      "  Read 1:    14,160,824 bp\n",
      "  Read 2:    14,160,680 bp\n",
      "Total written (filtered):     26,454,138 bp (93.4%)\n",
      "  Read 1:    13,201,456 bp\n",
      "  Read 2:    13,252,682 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47988 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t459\t0.0\t1\t0 443 16\n",
      "20\t47413\t0.0\t2\t46184 1211 18\n",
      "21\t73\t0.0\t2\t0 72 1\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47815 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t542\t0.0\t1\t0 542\n",
      "19\t47218\t0.0\t1\t46067 1151\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S220_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S220_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S220_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S220_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S220_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S220_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S220_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S220_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,931 reads  @     70.5 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.38 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,931\n",
      "  Read 1 with adapter:                  47,918 (100.0%)\n",
      "  Read 2 with adapter:                  47,714 (99.5%)\n",
      "Pairs written (passing filters):        47,931 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,270,707 bp\n",
      "  Read 1:    14,139,584 bp\n",
      "  Read 2:    14,131,123 bp\n",
      "Total written (filtered):     26,407,465 bp (93.4%)\n",
      "  Read 1:    13,182,186 bp\n",
      "  Read 2:    13,225,279 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47918 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t47\t0.0\t1\t0 0 47\n",
      "19\t1008\t0.0\t1\t0 975 33\n",
      "20\t46753\t0.0\t2\t45722 1023 8\n",
      "21\t80\t0.0\t2\t0 78 2\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47714 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t810\t0.0\t1\t0 810\n",
      "19\t46816\t0.0\t1\t45749 1067\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S221_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S221_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S221_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S221_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S221_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S221_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S221_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S221_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,221 reads  @     80.5 µs/read;   0.75 M reads/minute\n",
      "Finished in 3.40 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,221\n",
      "  Read 1 with adapter:                  42,215 (100.0%)\n",
      "  Read 2 with adapter:                  42,087 (99.7%)\n",
      "Pairs written (passing filters):        42,221 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,909,770 bp\n",
      "  Read 1:    12,455,043 bp\n",
      "  Read 2:    12,454,727 bp\n",
      "Total written (filtered):     23,267,210 bp (93.4%)\n",
      "  Read 1:    11,611,582 bp\n",
      "  Read 2:    11,655,628 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42215 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t46\t0.0\t1\t0 0 46\n",
      "19\t870\t0.0\t1\t0 847 23\n",
      "20\t41202\t0.0\t2\t40327 863 12\n",
      "21\t71\t0.0\t2\t0 70 1\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42087 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t613\t0.0\t1\t0 613\n",
      "19\t41415\t0.0\t1\t40419 996\n",
      "20\t59\t0.0\t1\t0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S222_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S222_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S222_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S222_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S222_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S222_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S222_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S222_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:04        50,268 reads  @     83.1 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.18 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,268\n",
      "  Read 1 with adapter:                  50,260 (100.0%)\n",
      "  Read 2 with adapter:                  50,096 (99.7%)\n",
      "Pairs written (passing filters):        50,268 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,656,482 bp\n",
      "  Read 1:    14,829,037 bp\n",
      "  Read 2:    14,827,445 bp\n",
      "Total written (filtered):     27,701,039 bp (93.4%)\n",
      "  Read 1:    13,824,852 bp\n",
      "  Read 2:    13,876,187 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50260 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t49\t0.0\t1\t0 0 49\n",
      "19\t1067\t0.0\t1\t0 1041 26\n",
      "20\t49029\t0.0\t2\t47829 1187 13\n",
      "21\t80\t0.0\t2\t0 79 1\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50096 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t622\t0.0\t1\t0 622\n",
      "19\t49418\t0.0\t1\t48182 1236\n",
      "20\t56\t0.0\t1\t0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S224_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S224_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S224_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S224_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S224_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S224_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S224_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S224_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,302 reads  @     81.2 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.58 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,302\n",
      "  Read 1 with adapter:                  56,290 (100.0%)\n",
      "  Read 2 with adapter:                  55,932 (99.3%)\n",
      "Pairs written (passing filters):        56,302 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,203,735 bp\n",
      "  Read 1:    16,608,171 bp\n",
      "  Read 2:    16,595,564 bp\n",
      "Total written (filtered):     31,017,014 bp (93.4%)\n",
      "  Read 1:    15,483,287 bp\n",
      "  Read 2:    15,533,727 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56290 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t33\t0.0\t1\t0 0 33\n",
      "19\t988\t0.0\t1\t0 948 40\n",
      "20\t55158\t0.0\t2\t53541 1589 28\n",
      "21\t84\t0.0\t2\t0 83 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55932 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t946\t0.0\t1\t0 946\n",
      "19\t54911\t0.0\t1\t53323 1588\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S225_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S225_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S225_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S225_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S225_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S225_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S225_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S225_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        57,631 reads  @     67.8 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.91 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,631\n",
      "  Read 1 with adapter:                  57,619 (100.0%)\n",
      "  Read 2 with adapter:                  57,346 (99.5%)\n",
      "Pairs written (passing filters):        57,631 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,000,851 bp\n",
      "  Read 1:    17,000,512 bp\n",
      "  Read 2:    17,000,339 bp\n",
      "Total written (filtered):     31,760,548 bp (93.4%)\n",
      "  Read 1:    15,849,022 bp\n",
      "  Read 2:    15,911,526 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57619 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t965\t0.0\t1\t0 931 34\n",
      "20\t56518\t0.0\t2\t54976 1530 12\n",
      "21\t81\t0.0\t2\t0 80 1\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57346 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t841\t0.0\t1\t0 841\n",
      "19\t56425\t0.0\t1\t54805 1620\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S226_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S226_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S226_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S226_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S226_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S226_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S226_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S226_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        41,267 reads  @     67.9 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.81 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,267\n",
      "  Read 1 with adapter:                  41,263 (100.0%)\n",
      "  Read 2 with adapter:                  41,044 (99.5%)\n",
      "Pairs written (passing filters):        41,267 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,346,142 bp\n",
      "  Read 1:    12,173,475 bp\n",
      "  Read 2:    12,172,667 bp\n",
      "Total written (filtered):     22,742,056 bp (93.4%)\n",
      "  Read 1:    11,348,810 bp\n",
      "  Read 2:    11,393,246 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41263 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t665\t0.0\t1\t0 647 18\n",
      "20\t40486\t0.0\t2\t39440 1030 16\n",
      "21\t62\t0.0\t2\t0 61 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41044 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t471\t0.0\t1\t0 471\n",
      "19\t40517\t0.0\t1\t39279 1238\n",
      "20\t56\t0.0\t1\t0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S228_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S228_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S228_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S228_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S228_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S228_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S228_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S228_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,252 reads  @     79.8 µs/read;   0.75 M reads/minute\n",
      "Finished in 3.29 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,252\n",
      "  Read 1 with adapter:                  41,238 (100.0%)\n",
      "  Read 2 with adapter:                  40,915 (99.2%)\n",
      "Pairs written (passing filters):        41,252 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,325,738 bp\n",
      "  Read 1:    12,168,602 bp\n",
      "  Read 2:    12,157,136 bp\n",
      "Total written (filtered):     22,724,812 bp (93.4%)\n",
      "  Read 1:    11,344,464 bp\n",
      "  Read 2:    11,380,348 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41238 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t706\t0.0\t1\t0 686 20\n",
      "20\t40409\t0.0\t2\t39349 1043 17\n",
      "21\t58\t0.0\t2\t0 57 1\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40915 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t647\t0.0\t1\t0 647\n",
      "19\t40218\t0.0\t1\t39021 1197\n",
      "20\t50\t0.0\t1\t0 50\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S229_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S229_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S229_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S229_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S229_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S229_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S229_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S229_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,350 reads  @     81.4 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.69 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,350\n",
      "  Read 1 with adapter:                  45,319 (99.9%)\n",
      "  Read 2 with adapter:                  45,155 (99.6%)\n",
      "Pairs written (passing filters):        45,350 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,753,174 bp\n",
      "  Read 1:    13,375,247 bp\n",
      "  Read 2:    13,377,927 bp\n",
      "Total written (filtered):     24,990,149 bp (93.4%)\n",
      "  Read 1:    12,469,574 bp\n",
      "  Read 2:    12,520,575 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45319 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t777\t0.0\t1\t0 747 30\n",
      "20\t44407\t0.0\t2\t43281 1108 18\n",
      "21\t76\t0.0\t2\t0 75 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45155 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t677\t0.0\t1\t0 677\n",
      "19\t44394\t0.0\t1\t43148 1246\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S230_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S230_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S230_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S230_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S230_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S230_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S230_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S230_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        47,483 reads  @     68.0 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.23 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,483\n",
      "  Read 1 with adapter:                  47,232 (99.5%)\n",
      "  Read 2 with adapter:                  47,268 (99.5%)\n",
      "Pairs written (passing filters):        47,483 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,978,267 bp\n",
      "  Read 1:    13,973,060 bp\n",
      "  Read 2:    14,005,207 bp\n",
      "Total written (filtered):     26,136,810 bp (93.4%)\n",
      "  Read 1:    13,029,153 bp\n",
      "  Read 2:    13,107,657 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47232 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t812\t0.0\t1\t0 785 27\n",
      "20\t46288\t0.0\t2\t44949 1320 19\n",
      "21\t77\t0.0\t2\t0 76 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47268 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t588\t0.0\t1\t0 588\n",
      "19\t46634\t0.0\t1\t45292 1342\n",
      "20\t46\t0.0\t1\t0 46\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S232_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S232_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S232_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S232_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S232_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S232_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S232_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S232_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        55,551 reads  @     65.7 µs/read;   0.91 M reads/minute\n",
      "Finished in 3.66 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,551\n",
      "  Read 1 with adapter:                  55,538 (100.0%)\n",
      "  Read 2 with adapter:                  55,223 (99.4%)\n",
      "Pairs written (passing filters):        55,550 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,761,926 bp\n",
      "  Read 1:    16,387,477 bp\n",
      "  Read 2:    16,374,449 bp\n",
      "Total written (filtered):     30,603,103 bp (93.4%)\n",
      "  Read 1:    15,277,079 bp\n",
      "  Read 2:    15,326,024 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55538 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t744\t0.0\t1\t0 732 12\n",
      "20\t54656\t0.0\t2\t53289 1348 19\n",
      "21\t73\t0.0\t2\t0 72 1\n",
      "22\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55223 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t892\t0.0\t1\t0 892\n",
      "19\t54251\t0.0\t1\t52993 1258\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S233_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S233_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S233_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S233_199_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S233_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S233_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S233_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S233_199_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        64,202 reads  @     62.4 µs/read;   0.96 M reads/minute\n",
      "Finished in 4.01 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,202\n",
      "  Read 1 with adapter:                  64,191 (100.0%)\n",
      "  Read 2 with adapter:                  63,902 (99.5%)\n",
      "Pairs written (passing filters):        64,202 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,878,599 bp\n",
      "  Read 1:    18,939,565 bp\n",
      "  Read 2:    18,939,034 bp\n",
      "Total written (filtered):     35,382,311 bp (93.4%)\n",
      "  Read 1:    17,656,566 bp\n",
      "  Read 2:    17,725,745 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64191 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t36\t0.0\t1\t0 0 36\n",
      "19\t902\t0.0\t1\t0 876 26\n",
      "20\t63122\t0.0\t2\t61566 1529 27\n",
      "21\t109\t0.0\t2\t0 106 3\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63902 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t945\t0.0\t1\t0 945\n",
      "19\t62861\t0.0\t1\t61256 1605\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S234_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S234_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S234_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S234_201_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S234_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S234_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S234_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S234_201_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,030 reads  @     62.1 µs/read;   0.97 M reads/minute\n",
      "Finished in 2.68 s (62 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,030\n",
      "  Read 1 with adapter:                  43,022 (100.0%)\n",
      "  Read 2 with adapter:                  42,855 (99.6%)\n",
      "Pairs written (passing filters):        43,030 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,386,775 bp\n",
      "  Read 1:    12,693,601 bp\n",
      "  Read 2:    12,693,174 bp\n",
      "Total written (filtered):     23,713,103 bp (93.4%)\n",
      "  Read 1:    11,833,737 bp\n",
      "  Read 2:    11,879,366 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43022 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t33\t0.0\t1\t0 0 33\n",
      "19\t629\t0.0\t1\t0 604 25\n",
      "20\t42266\t0.0\t2\t41250 1006 10\n",
      "21\t69\t0.0\t2\t0 69\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42855 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t488\t0.0\t1\t0 488\n",
      "19\t42316\t0.0\t1\t41246 1070\n",
      "20\t51\t0.0\t1\t0 51\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S236_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S236_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S236_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S236_203_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S236_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S236_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S236_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S236_203_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        55,057 reads  @     63.1 µs/read;   0.95 M reads/minute\n",
      "Finished in 3.48 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,057\n",
      "  Read 1 with adapter:                  55,042 (100.0%)\n",
      "  Read 2 with adapter:                  54,843 (99.6%)\n",
      "Pairs written (passing filters):        55,057 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,476,738 bp\n",
      "  Read 1:    16,241,794 bp\n",
      "  Read 2:    16,234,944 bp\n",
      "Total written (filtered):     30,335,637 bp (93.4%)\n",
      "  Read 1:    15,141,839 bp\n",
      "  Read 2:    15,193,798 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55042 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t35\t0.0\t1\t0 0 35\n",
      "19\t934\t0.0\t1\t0 904 30\n",
      "20\t53973\t0.0\t2\t52483 1468 22\n",
      "21\t81\t0.0\t2\t0 79 2\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54843 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t951\t0.0\t1\t0 951\n",
      "19\t53812\t0.0\t1\t52674 1138\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S237_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S237_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S237_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S237_205_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S237_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S237_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S237_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S237_205_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,040 reads  @     62.7 µs/read;   0.96 M reads/minute\n",
      "Finished in 3.33 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,040\n",
      "  Read 1 with adapter:                  53,030 (100.0%)\n",
      "  Read 2 with adapter:                  52,812 (99.6%)\n",
      "Pairs written (passing filters):        53,040 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,293,111 bp\n",
      "  Read 1:    15,646,613 bp\n",
      "  Read 2:    15,646,498 bp\n",
      "Total written (filtered):     29,230,658 bp (93.4%)\n",
      "  Read 1:    14,586,860 bp\n",
      "  Read 2:    14,643,798 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53030 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t914\t0.0\t1\t0 879 35\n",
      "20\t51965\t0.0\t2\t50462 1479 24\n",
      "21\t79\t0.0\t2\t0 78 1\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52812 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t796\t0.0\t1\t0 796\n",
      "19\t51948\t0.0\t1\t50705 1243\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S238_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S238_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S238_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S238_207_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S238_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S238_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S238_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S238_207_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:03        61,373 reads  @     61.0 µs/read;   0.98 M reads/minute\n",
      "Finished in 3.75 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,373\n",
      "  Read 1 with adapter:                  61,361 (100.0%)\n",
      "  Read 2 with adapter:                  61,151 (99.6%)\n",
      "Pairs written (passing filters):        61,373 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,208,011 bp\n",
      "  Read 1:    18,104,521 bp\n",
      "  Read 2:    18,103,490 bp\n",
      "Total written (filtered):     33,820,607 bp (93.4%)\n",
      "  Read 1:    16,878,269 bp\n",
      "  Read 2:    16,942,338 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 61361 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t46\t0.0\t1\t0 0 46\n",
      "19\t1033\t0.0\t1\t0 1005 28\n",
      "20\t60143\t0.0\t2\t58468 1658 17\n",
      "21\t121\t0.0\t2\t0 119 2\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61151 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t773\t0.0\t1\t0 773\n",
      "19\t60322\t0.0\t1\t58784 1538\n",
      "20\t56\t0.0\t1\t0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S240_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S240_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S240_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S240_209_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S240_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S240_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S240_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S240_209_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,679 reads  @     59.8 µs/read;   1.00 M reads/minute\n",
      "Finished in 3.21 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,679\n",
      "  Read 1 with adapter:                  53,657 (100.0%)\n",
      "  Read 2 with adapter:                  53,364 (99.4%)\n",
      "Pairs written (passing filters):        53,679 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,655,363 bp\n",
      "  Read 1:    15,833,956 bp\n",
      "  Read 2:    15,821,407 bp\n",
      "Total written (filtered):     29,569,651 bp (93.4%)\n",
      "  Read 1:    14,761,390 bp\n",
      "  Read 2:    14,808,261 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53657 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t657\t0.0\t1\t0 637 20\n",
      "20\t52876\t0.0\t2\t51412 1450 14\n",
      "21\t77\t0.0\t2\t0 75 2\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53364 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t849\t0.0\t1\t0 849\n",
      "19\t52436\t0.0\t1\t51167 1269\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S241_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S241_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S241_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S241_211_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S241_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S241_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S241_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S241_211_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,890 reads  @     60.4 µs/read;   0.99 M reads/minute\n",
      "Finished in 3.08 s (60 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,890\n",
      "  Read 1 with adapter:                  50,869 (100.0%)\n",
      "  Read 2 with adapter:                  50,678 (99.6%)\n",
      "Pairs written (passing filters):        50,890 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,021,824 bp\n",
      "  Read 1:    15,010,184 bp\n",
      "  Read 2:    15,011,640 bp\n",
      "Total written (filtered):     28,042,798 bp (93.4%)\n",
      "  Read 1:    13,993,368 bp\n",
      "  Read 2:    14,049,430 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50869 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t623\t0.0\t1\t0 602 21\n",
      "20\t50129\t0.0\t2\t48791 1317 21\n",
      "21\t87\t0.0\t2\t0 80 7\n",
      "22\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50678 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t738\t0.0\t1\t0 738\n",
      "19\t49874\t0.0\t1\t48616 1258\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S242_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S242_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S242_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S242_213_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S242_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S242_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S242_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S242_213_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        42,536 reads  @     61.5 µs/read;   0.98 M reads/minute\n",
      "Finished in 2.62 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,536\n",
      "  Read 1 with adapter:                  42,517 (100.0%)\n",
      "  Read 2 with adapter:                  42,338 (99.5%)\n",
      "Pairs written (passing filters):        42,536 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,092,809 bp\n",
      "  Read 1:    12,545,823 bp\n",
      "  Read 2:    12,546,986 bp\n",
      "Total written (filtered):     23,438,983 bp (93.4%)\n",
      "  Read 1:    11,695,942 bp\n",
      "  Read 2:    11,743,041 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42517 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t490\t0.0\t1\t0 480 10\n",
      "20\t41937\t0.0\t2\t40777 1145 15\n",
      "21\t57\t0.0\t2\t0 57\n",
      "22\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42338 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t515\t0.0\t1\t0 515\n",
      "19\t41785\t0.0\t1\t40678 1107\n",
      "20\t38\t0.0\t1\t0 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S244_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S244_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S244_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S244_215_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S244_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S244_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S244_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S244_215_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        50,032 reads  @     59.0 µs/read;   1.02 M reads/minute\n",
      "Finished in 2.95 s (59 us/read; 1.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,032\n",
      "  Read 1 with adapter:                  50,022 (100.0%)\n",
      "  Read 2 with adapter:                  49,770 (99.5%)\n",
      "Pairs written (passing filters):        50,032 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,506,277 bp\n",
      "  Read 1:    14,759,239 bp\n",
      "  Read 2:    14,747,038 bp\n",
      "Total written (filtered):     27,561,257 bp (93.4%)\n",
      "  Read 1:    13,759,122 bp\n",
      "  Read 2:    13,802,135 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50022 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t460\t0.0\t1\t0 449 11\n",
      "20\t49427\t0.0\t2\t48390 1033 4\n",
      "21\t81\t0.0\t2\t0 81\n",
      "22\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49770 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t799\t0.0\t1\t0 799\n",
      "19\t48899\t0.0\t1\t47690 1209\n",
      "20\t72\t0.0\t1\t0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S245_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S245_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S245_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S245_217_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S245_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-xntnmpqk/S245_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S245_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mkmwmttr/d2280710-ca35-42de-b1fd-0f11d49dcc26/data/S245_217_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,651 reads  @     61.5 µs/read;   0.98 M reads/minute\n",
      "Finished in 3.06 s (62 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,651\n",
      "  Read 1 with adapter:                  49,641 (100.0%)\n",
      "  Read 2 with adapter:                  49,471 (99.6%)\n",
      "Pairs written (passing filters):        49,651 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,293,572 bp\n",
      "  Read 1:    14,646,904 bp\n",
      "  Read 2:    14,646,668 bp\n",
      "Total written (filtered):     27,361,772 bp (93.4%)\n",
      "  Read 1:    13,654,398 bp\n",
      "  Read 2:    13,707,374 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49641 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t435\t0.0\t1\t0 425 10\n",
      "20\t49067\t0.0\t2\t48154 905 8\n",
      "21\t77\t0.0\t2\t0 75 2\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49471 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t717\t0.0\t1\t0 717\n",
      "19\t48692\t0.0\t1\t47608 1084\n",
      "20\t62\t0.0\t1\t0 62\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: 18S-demux-pairedend-trimmed1.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S371_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S371_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S371_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S371_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S371_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S371_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S371_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S371_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        34,922 reads  @     63.4 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.22 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,922\n",
      "  Read 1 with adapter:                  34,886 (99.9%)\n",
      "  Read 2 with adapter:                  34,740 (99.5%)\n",
      "Pairs written (passing filters):        34,922 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,601,497 bp\n",
      "  Read 1:    10,301,808 bp\n",
      "  Read 2:    10,299,689 bp\n",
      "Total written (filtered):     19,244,877 bp (93.4%)\n",
      "  Read 1:     9,604,674 bp\n",
      "  Read 2:     9,640,203 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34886 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t32\t0.0\t1\t0 0 32\n",
      "19\t605\t0.0\t1\t0 575 30\n",
      "20\t34185\t0.0\t2\t32471 1668 46\n",
      "21\t45\t0.0\t2\t0 44 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34740 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t658\t0.0\t1\t0 658\n",
      "19\t33998\t0.0\t1\t33093 905\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S372_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S372_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S372_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S372_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S372_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S372_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S372_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S372_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,680 reads  @     62.7 µs/read;   0.96 M reads/minute\n",
      "Finished in 2.74 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,680\n",
      "  Read 1 with adapter:                  43,645 (99.9%)\n",
      "  Read 2 with adapter:                  43,430 (99.4%)\n",
      "Pairs written (passing filters):        43,680 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,770,322 bp\n",
      "  Read 1:    12,885,302 bp\n",
      "  Read 2:    12,885,020 bp\n",
      "Total written (filtered):     24,073,677 bp (93.4%)\n",
      "  Read 1:    12,013,201 bp\n",
      "  Read 2:    12,060,476 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43645 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t844\t0.0\t1\t0 814 30\n",
      "20\t42685\t0.0\t2\t40476 2169 40\n",
      "21\t67\t0.0\t2\t0 67\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43430 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t699\t0.0\t1\t0 699\n",
      "19\t42658\t0.0\t1\t41325 1333\n",
      "20\t73\t0.0\t1\t0 73\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S373_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S373_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S373_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S373_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S373_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S373_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S373_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S373_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        42,852 reads  @     65.0 µs/read;   0.92 M reads/minute\n",
      "Finished in 2.79 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,852\n",
      "  Read 1 with adapter:                  42,818 (99.9%)\n",
      "  Read 2 with adapter:                  42,544 (99.3%)\n",
      "Pairs written (passing filters):        42,852 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,281,156 bp\n",
      "  Read 1:    12,641,091 bp\n",
      "  Read 2:    12,640,065 bp\n",
      "Total written (filtered):     23,617,787 bp (93.4%)\n",
      "  Read 1:    11,785,521 bp\n",
      "  Read 2:    11,832,266 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42818 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t38\t0.0\t1\t0 0 38\n",
      "19\t833\t0.0\t1\t0 802 31\n",
      "20\t41850\t0.0\t2\t39563 2222 65\n",
      "21\t75\t0.0\t2\t0 74 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42544 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t615\t0.0\t1\t0 615\n",
      "19\t41851\t0.0\t1\t40551 1300\n",
      "20\t78\t0.0\t1\t0 78\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S374_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S374_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S374_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S374_7_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S374_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S374_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S374_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S374_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        56,712 reads  @     61.4 µs/read;   0.98 M reads/minute\n",
      "Finished in 3.49 s (62 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,712\n",
      "  Read 1 with adapter:                  56,663 (99.9%)\n",
      "  Read 2 with adapter:                  56,488 (99.6%)\n",
      "Pairs written (passing filters):        56,712 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,458,926 bp\n",
      "  Read 1:    16,729,117 bp\n",
      "  Read 2:    16,729,809 bp\n",
      "Total written (filtered):     31,254,037 bp (93.4%)\n",
      "  Read 1:    15,596,821 bp\n",
      "  Read 2:    15,657,216 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56663 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t46\t0.0\t1\t0 0 46\n",
      "19\t1010\t0.0\t1\t0 962 48\n",
      "20\t55495\t0.0\t2\t52671 2747 77\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56488 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t795\t0.0\t1\t0 795\n",
      "19\t55577\t0.0\t1\t54093 1484\n",
      "20\t116\t0.0\t1\t0 116\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S375_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S375_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S375_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S375_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S375_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S375_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S375_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S375_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        47,436 reads  @     62.4 µs/read;   0.96 M reads/minute\n",
      "Finished in 2.96 s (62 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,436\n",
      "  Read 1 with adapter:                  47,402 (99.9%)\n",
      "  Read 2 with adapter:                  47,238 (99.6%)\n",
      "Pairs written (passing filters):        47,436 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,986,206 bp\n",
      "  Read 1:    13,993,131 bp\n",
      "  Read 2:    13,993,075 bp\n",
      "Total written (filtered):     26,142,057 bp (93.4%)\n",
      "  Read 1:    13,045,914 bp\n",
      "  Read 2:    13,096,143 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47402 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t37\t0.0\t1\t0 0 37\n",
      "19\t873\t0.0\t1\t0 829 44\n",
      "20\t46395\t0.0\t2\t43970 2369 56\n",
      "21\t70\t0.0\t2\t0 69 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47238 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t667\t0.0\t1\t0 667\n",
      "19\t46494\t0.0\t1\t45195 1299\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S376_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S376_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S376_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S376_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S376_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S376_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S376_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S376_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,090 reads  @     64.6 µs/read;   0.93 M reads/minute\n",
      "Finished in 3.50 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,090\n",
      "  Read 1 with adapter:                  54,035 (99.9%)\n",
      "  Read 2 with adapter:                  53,716 (99.3%)\n",
      "Pairs written (passing filters):        54,090 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,898,141 bp\n",
      "  Read 1:    15,955,934 bp\n",
      "  Read 2:    15,942,207 bp\n",
      "Total written (filtered):     29,798,829 bp (93.4%)\n",
      "  Read 1:    14,876,281 bp\n",
      "  Read 2:    14,922,548 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54035 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t49\t0.0\t1\t0 0 49\n",
      "19\t1123\t0.0\t1\t0 1057 66\n",
      "20\t52724\t0.0\t2\t50032 2602 90\n",
      "21\t104\t0.0\t2\t0 100 4\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53716 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1040\t0.0\t1\t0 1040\n",
      "19\t52581\t0.0\t1\t51057 1524\n",
      "20\t95\t0.0\t1\t0 95\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S377_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S377_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S377_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S377_13_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S377_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S377_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S377_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S377_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        50,747 reads  @     65.2 µs/read;   0.92 M reads/minute\n",
      "Finished in 3.31 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,747\n",
      "  Read 1 with adapter:                  50,683 (99.9%)\n",
      "  Read 2 with adapter:                  50,401 (99.3%)\n",
      "Pairs written (passing filters):        50,747 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,918,145 bp\n",
      "  Read 1:    14,968,367 bp\n",
      "  Read 2:    14,949,778 bp\n",
      "Total written (filtered):     27,948,467 bp (93.4%)\n",
      "  Read 1:    13,955,626 bp\n",
      "  Read 2:    13,992,841 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50683 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t46\t0.0\t1\t0 0 46\n",
      "19\t958\t0.0\t1\t0 907 51\n",
      "20\t49578\t0.0\t2\t46988 2524 66\n",
      "21\t71\t0.0\t2\t0 69 2\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50401 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t740\t0.0\t1\t0 740\n",
      "19\t49603\t0.0\t1\t48387 1216\n",
      "20\t58\t0.0\t1\t0 58\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S378_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S378_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S378_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S378_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S378_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S378_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S378_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S378_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,798 reads  @     69.4 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.46 s (69 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,798\n",
      "  Read 1 with adapter:                  49,758 (99.9%)\n",
      "  Read 2 with adapter:                  48,699 (97.8%)\n",
      "Pairs written (passing filters):        49,798 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,321,258 bp\n",
      "  Read 1:    14,690,392 bp\n",
      "  Read 2:    14,630,866 bp\n",
      "Total written (filtered):     27,401,925 bp (93.5%)\n",
      "  Read 1:    13,696,030 bp\n",
      "  Read 2:    13,705,895 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49758 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t38\t0.0\t1\t0 0 38\n",
      "19\t853\t0.0\t1\t0 810 43\n",
      "20\t48760\t0.0\t2\t46412 2297 51\n",
      "21\t83\t0.0\t2\t0 81 2\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48699 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t367\t0.0\t1\t0 367\n",
      "19\t48275\t0.0\t1\t47206 1069\n",
      "20\t57\t0.0\t1\t0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S379_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S379_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S379_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S379_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S379_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S379_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S379_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S379_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,104 reads  @     61.5 µs/read;   0.98 M reads/minute\n",
      "Finished in 3.21 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,104\n",
      "  Read 1 with adapter:                  52,059 (99.9%)\n",
      "  Read 2 with adapter:                  51,881 (99.6%)\n",
      "Pairs written (passing filters):        52,104 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,740,591 bp\n",
      "  Read 1:    15,370,657 bp\n",
      "  Read 2:    15,369,934 bp\n",
      "Total written (filtered):     28,715,406 bp (93.4%)\n",
      "  Read 1:    14,330,230 bp\n",
      "  Read 2:    14,385,176 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52059 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t35\t0.0\t1\t0 0 35\n",
      "19\t809\t0.0\t1\t0 771 38\n",
      "20\t51111\t0.0\t2\t48360 2684 67\n",
      "21\t82\t0.0\t2\t0 81 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51881 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1080\t0.0\t1\t0 1080\n",
      "19\t50702\t0.0\t1\t49440 1262\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S380_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S380_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S380_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S380_19_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S380_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S380_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S380_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S380_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        66,880 reads  @     63.1 µs/read;   0.95 M reads/minute\n",
      "Finished in 4.22 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,880\n",
      "  Read 1 with adapter:                  66,822 (99.9%)\n",
      "  Read 2 with adapter:                  66,590 (99.6%)\n",
      "Pairs written (passing filters):        66,880 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,458,391 bp\n",
      "  Read 1:    19,729,376 bp\n",
      "  Read 2:    19,729,015 bp\n",
      "Total written (filtered):     36,858,743 bp (93.4%)\n",
      "  Read 1:    18,393,909 bp\n",
      "  Read 2:    18,464,834 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 66822 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t59\t0.0\t1\t0 0 59\n",
      "19\t1064\t0.0\t1\t0 1012 52\n",
      "20\t65543\t0.0\t2\t62010 3446 87\n",
      "21\t103\t0.0\t2\t0 103\n",
      "22\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 66590 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1139\t0.0\t1\t0 1139\n",
      "19\t65341\t0.0\t1\t63639 1702\n",
      "20\t110\t0.0\t1\t0 110\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S381_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S381_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S381_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S381_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S381_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S381_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S381_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S381_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        42,257 reads  @     60.5 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.56 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,257\n",
      "  Read 1 with adapter:                  42,214 (99.9%)\n",
      "  Read 2 with adapter:                  42,102 (99.6%)\n",
      "Pairs written (passing filters):        42,257 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,931,146 bp\n",
      "  Read 1:    12,465,790 bp\n",
      "  Read 2:    12,465,356 bp\n",
      "Total written (filtered):     23,288,175 bp (93.4%)\n",
      "  Read 1:    11,622,120 bp\n",
      "  Read 2:    11,666,055 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42214 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t673\t0.0\t1\t0 635 38\n",
      "20\t41426\t0.0\t2\t39115 2267 44\n",
      "21\t55\t0.0\t2\t0 54 1\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42102 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t728\t0.0\t1\t0 728\n",
      "19\t41283\t0.0\t1\t40300 983\n",
      "20\t91\t0.0\t1\t0 91\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S382_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S382_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S382_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S382_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S382_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S382_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S382_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S382_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        57,865 reads  @     62.4 µs/read;   0.96 M reads/minute\n",
      "Finished in 3.62 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,865\n",
      "  Read 1 with adapter:                  57,810 (99.9%)\n",
      "  Read 2 with adapter:                  57,659 (99.6%)\n",
      "Pairs written (passing filters):        57,865 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,138,047 bp\n",
      "  Read 1:    17,069,202 bp\n",
      "  Read 2:    17,068,845 bp\n",
      "Total written (filtered):     31,888,018 bp (93.4%)\n",
      "  Read 1:    15,913,888 bp\n",
      "  Read 2:    15,974,130 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57810 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t967\t0.0\t1\t0 913 54\n",
      "20\t56697\t0.0\t2\t53736 2891 70\n",
      "21\t87\t0.0\t2\t0 86 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57659 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t938\t0.0\t1\t0 938\n",
      "19\t56589\t0.0\t1\t55296 1293\n",
      "20\t132\t0.0\t1\t0 132\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S383_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S383_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S383_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S383_25_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S383_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S383_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S383_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S383_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        52,168 reads  @     63.9 µs/read;   0.94 M reads/minute\n",
      "Finished in 3.34 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,168\n",
      "  Read 1 with adapter:                  52,112 (99.9%)\n",
      "  Read 2 with adapter:                  52,015 (99.7%)\n",
      "Pairs written (passing filters):        52,168 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,777,913 bp\n",
      "  Read 1:    15,389,543 bp\n",
      "  Read 2:    15,388,370 bp\n",
      "Total written (filtered):     28,748,854 bp (93.4%)\n",
      "  Read 1:    14,348,105 bp\n",
      "  Read 2:    14,400,749 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52112 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t856\t0.0\t1\t0 811 45\n",
      "20\t51113\t0.0\t2\t48454 2592 67\n",
      "21\t76\t0.0\t2\t0 74 2\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52015 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t758\t0.0\t1\t0 758\n",
      "19\t51163\t0.0\t1\t50001 1162\n",
      "20\t94\t0.0\t1\t0 94\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S384_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S384_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S384_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S384_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S384_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S384_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S384_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S384_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        58,804 reads  @     67.2 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.96 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,804\n",
      "  Read 1 with adapter:                  58,748 (99.9%)\n",
      "  Read 2 with adapter:                  58,519 (99.5%)\n",
      "Pairs written (passing filters):        58,804 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,681,419 bp\n",
      "  Read 1:    17,346,959 bp\n",
      "  Read 2:    17,334,460 bp\n",
      "Total written (filtered):     32,396,604 bp (93.4%)\n",
      "  Read 1:    16,172,914 bp\n",
      "  Read 2:    16,223,690 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58748 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t32\t0.0\t1\t0 0 32\n",
      "19\t1007\t0.0\t1\t0 959 48\n",
      "20\t57592\t0.0\t2\t54523 2992 77\n",
      "21\t78\t0.0\t2\t0 78\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 58519 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1187\t0.0\t1\t0 1187\n",
      "19\t57236\t0.0\t1\t55866 1370\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S385_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S385_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S385_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S385_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S385_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S385_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S385_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S385_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,315 reads  @     65.1 µs/read;   0.92 M reads/minute\n",
      "Finished in 3.54 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,315\n",
      "  Read 1 with adapter:                  54,274 (99.9%)\n",
      "  Read 2 with adapter:                  53,938 (99.3%)\n",
      "Pairs written (passing filters):        54,315 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,021,825 bp\n",
      "  Read 1:    16,022,903 bp\n",
      "  Read 2:    15,998,922 bp\n",
      "Total written (filtered):     29,913,101 bp (93.4%)\n",
      "  Read 1:    14,938,249 bp\n",
      "  Read 2:    14,974,852 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54274 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t40\t0.0\t1\t0 0 40\n",
      "19\t892\t0.0\t1\t0 850 42\n",
      "20\t53230\t0.0\t2\t50308 2853 69\n",
      "21\t78\t0.0\t2\t0 77 1\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53938 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t820\t0.0\t1\t0 820\n",
      "19\t53050\t0.0\t1\t51775 1275\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S386_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S386_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S386_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S386_31_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S386_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S386_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S386_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S386_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,277 reads  @     71.2 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.87 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,277\n",
      "  Read 1 with adapter:                  54,231 (99.9%)\n",
      "  Read 2 with adapter:                  53,132 (97.9%)\n",
      "Pairs written (passing filters):        54,277 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,960,502 bp\n",
      "  Read 1:    16,011,201 bp\n",
      "  Read 2:    15,949,301 bp\n",
      "Total written (filtered):     29,867,461 bp (93.5%)\n",
      "  Read 1:    14,927,296 bp\n",
      "  Read 2:    14,940,165 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54231 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t807\t0.0\t1\t0 768 39\n",
      "20\t53282\t0.0\t2\t50403 2801 78\n",
      "21\t84\t0.0\t2\t0 82 2\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53132 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t427\t0.0\t1\t0 427\n",
      "19\t52650\t0.0\t1\t51611 1039\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S387_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S387_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S387_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S387_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S387_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S387_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S387_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S387_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        56,492 reads  @     63.3 µs/read;   0.95 M reads/minute\n",
      "Finished in 3.58 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,492\n",
      "  Read 1 with adapter:                  56,434 (99.9%)\n",
      "  Read 2 with adapter:                  56,179 (99.4%)\n",
      "Pairs written (passing filters):        56,492 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,325,713 bp\n",
      "  Read 1:    16,664,885 bp\n",
      "  Read 2:    16,660,828 bp\n",
      "Total written (filtered):     31,131,060 bp (93.4%)\n",
      "  Read 1:    15,536,591 bp\n",
      "  Read 2:    15,594,469 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56434 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t486\t0.0\t1\t0 467 19\n",
      "20\t55827\t0.0\t2\t53048 2718 61\n",
      "21\t70\t0.0\t2\t0 69 1\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56179 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1167\t0.0\t1\t0 1167\n",
      "19\t54887\t0.0\t1\t53391 1496\n",
      "20\t125\t0.0\t1\t0 125\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S388_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S388_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S388_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S388_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S388_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S388_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S388_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S388_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,210 reads  @     65.6 µs/read;   0.91 M reads/minute\n",
      "Finished in 3.30 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,210\n",
      "  Read 1 with adapter:                  50,152 (99.9%)\n",
      "  Read 2 with adapter:                  49,920 (99.4%)\n",
      "Pairs written (passing filters):        50,210 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,623,381 bp\n",
      "  Read 1:    14,811,935 bp\n",
      "  Read 2:    14,811,446 bp\n",
      "Total written (filtered):     27,673,014 bp (93.4%)\n",
      "  Read 1:    13,809,226 bp\n",
      "  Read 2:    13,863,788 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50152 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t430\t0.0\t1\t0 411 19\n",
      "20\t49609\t0.0\t2\t46997 2562 50\n",
      "21\t63\t0.0\t2\t0 62 1\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49920 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t895\t0.0\t1\t0 895\n",
      "19\t48952\t0.0\t1\t47286 1666\n",
      "20\t73\t0.0\t1\t0 73\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S389_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S389_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S389_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S389_37_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S389_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S389_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S389_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S389_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        45,850 reads  @     63.2 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.90 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,850\n",
      "  Read 1 with adapter:                  45,789 (99.9%)\n",
      "  Read 2 with adapter:                  45,566 (99.4%)\n",
      "Pairs written (passing filters):        45,850 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,050,254 bp\n",
      "  Read 1:    13,525,498 bp\n",
      "  Read 2:    13,524,756 bp\n",
      "Total written (filtered):     25,269,709 bp (93.4%)\n",
      "  Read 1:    12,610,032 bp\n",
      "  Read 2:    12,659,677 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45789 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t378\t0.0\t1\t0 363 15\n",
      "20\t45311\t0.0\t2\t42876 2367 68\n",
      "21\t56\t0.0\t2\t0 55 1\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45566 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t755\t0.0\t1\t0 755\n",
      "19\t44731\t0.0\t1\t43466 1265\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S390_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S390_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S390_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S390_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S390_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S390_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S390_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S390_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        57,374 reads  @     61.2 µs/read;   0.98 M reads/minute\n",
      "Finished in 3.52 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,374\n",
      "  Read 1 with adapter:                  57,318 (99.9%)\n",
      "  Read 2 with adapter:                  57,086 (99.5%)\n",
      "Pairs written (passing filters):        57,374 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,850,210 bp\n",
      "  Read 1:    16,924,985 bp\n",
      "  Read 2:    16,925,225 bp\n",
      "Total written (filtered):     31,620,453 bp (93.4%)\n",
      "  Read 1:    15,778,992 bp\n",
      "  Read 2:    15,841,461 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57318 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t469\t0.0\t1\t0 450 19\n",
      "20\t56711\t0.0\t2\t53779 2863 69\n",
      "21\t70\t0.0\t2\t0 67 3\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57086 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t981\t0.0\t1\t0 981\n",
      "19\t55994\t0.0\t1\t54359 1635\n",
      "20\t111\t0.0\t1\t0 111\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S391_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S391_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S391_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S391_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S391_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S391_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S391_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S391_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,652 reads  @     61.5 µs/read;   0.98 M reads/minute\n",
      "Finished in 2.69 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,652\n",
      "  Read 1 with adapter:                  43,601 (99.9%)\n",
      "  Read 2 with adapter:                  43,463 (99.6%)\n",
      "Pairs written (passing filters):        43,651 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,752,816 bp\n",
      "  Read 1:    12,877,327 bp\n",
      "  Read 2:    12,875,489 bp\n",
      "Total written (filtered):     24,055,574 bp (93.4%)\n",
      "  Read 1:    12,005,300 bp\n",
      "  Read 2:    12,050,274 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43601 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t335\t0.0\t1\t0 323 12\n",
      "20\t43166\t0.0\t2\t40978 2132 56\n",
      "21\t53\t0.0\t2\t0 52 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43463 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t649\t0.0\t1\t0 649\n",
      "19\t42747\t0.0\t1\t41496 1251\n",
      "20\t67\t0.0\t1\t0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S392_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S392_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S392_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S392_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S392_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S392_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S392_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S392_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        59,667 reads  @     63.0 µs/read;   0.95 M reads/minute\n",
      "Finished in 3.77 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,667\n",
      "  Read 1 with adapter:                  59,611 (99.9%)\n",
      "  Read 2 with adapter:                  59,255 (99.3%)\n",
      "Pairs written (passing filters):        59,667 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,191,654 bp\n",
      "  Read 1:    17,601,742 bp\n",
      "  Read 2:    17,589,912 bp\n",
      "Total written (filtered):     32,875,069 bp (93.4%)\n",
      "  Read 1:    16,409,921 bp\n",
      "  Read 2:    16,465,148 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59611 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t520\t0.0\t1\t0 505 15\n",
      "20\t58955\t0.0\t2\t55847 3031 77\n",
      "21\t71\t0.0\t2\t0 71\n",
      "22\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59255 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1158\t0.0\t1\t0 1158\n",
      "19\t58020\t0.0\t1\t56319 1701\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S393_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S393_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S393_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S393_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S393_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S393_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S393_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S393_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        57,235 reads  @     62.9 µs/read;   0.95 M reads/minute\n",
      "Finished in 3.61 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,235\n",
      "  Read 1 with adapter:                  57,182 (99.9%)\n",
      "  Read 2 with adapter:                  56,799 (99.2%)\n",
      "Pairs written (passing filters):        57,235 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,741,321 bp\n",
      "  Read 1:    16,884,116 bp\n",
      "  Read 2:    16,857,205 bp\n",
      "Total written (filtered):     31,519,760 bp (93.4%)\n",
      "  Read 1:    15,740,940 bp\n",
      "  Read 2:    15,778,820 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57182 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t538\t0.0\t1\t0 511 27\n",
      "20\t56525\t0.0\t2\t53697 2762 66\n",
      "21\t72\t0.0\t2\t0 69 3\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56799 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t871\t0.0\t1\t0 871\n",
      "19\t55853\t0.0\t1\t54437 1416\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S394_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S394_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S394_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S394_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S394_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S394_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S394_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S394_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        56,880 reads  @     65.6 µs/read;   0.92 M reads/minute\n",
      "Finished in 3.73 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,880\n",
      "  Read 1 with adapter:                  56,828 (99.9%)\n",
      "  Read 2 with adapter:                  55,006 (96.7%)\n",
      "Pairs written (passing filters):        56,880 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,455,116 bp\n",
      "  Read 1:    16,779,478 bp\n",
      "  Read 2:    16,675,638 bp\n",
      "Total written (filtered):     31,274,198 bp (93.5%)\n",
      "  Read 1:    15,643,283 bp\n",
      "  Read 2:    15,630,915 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56828 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t453\t0.0\t1\t0 434 19\n",
      "20\t56248\t0.0\t2\t53410 2761 77\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55006 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t464\t0.0\t1\t0 464\n",
      "19\t54469\t0.0\t1\t53010 1459\n",
      "20\t73\t0.0\t1\t0 73\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S395_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S395_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S395_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S395_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S395_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S395_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S395_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S395_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        55,921 reads  @     61.5 µs/read;   0.97 M reads/minute\n",
      "Finished in 3.45 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,921\n",
      "  Read 1 with adapter:                  55,870 (99.9%)\n",
      "  Read 2 with adapter:                  55,704 (99.6%)\n",
      "Pairs written (passing filters):        55,921 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,991,690 bp\n",
      "  Read 1:    16,496,684 bp\n",
      "  Read 2:    16,495,006 bp\n",
      "Total written (filtered):     30,817,742 bp (93.4%)\n",
      "  Read 1:    15,380,148 bp\n",
      "  Read 2:    15,437,594 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55870 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t35\t0.0\t1\t0 0 35\n",
      "19\t904\t0.0\t1\t0 865 39\n",
      "20\t54843\t0.0\t2\t52363 2431 49\n",
      "21\t66\t0.0\t2\t0 64 2\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55704 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1066\t0.0\t1\t0 1066\n",
      "19\t54536\t0.0\t1\t53259 1277\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S396_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S396_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S396_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S396_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S396_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S396_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S396_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S396_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        59,357 reads  @     60.5 µs/read;   0.99 M reads/minute\n",
      "Finished in 3.59 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,357\n",
      "  Read 1 with adapter:                  59,320 (99.9%)\n",
      "  Read 2 with adapter:                  59,125 (99.6%)\n",
      "Pairs written (passing filters):        59,357 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,019,715 bp\n",
      "  Read 1:    17,510,040 bp\n",
      "  Read 2:    17,509,675 bp\n",
      "Total written (filtered):     32,711,847 bp (93.4%)\n",
      "  Read 1:    16,324,665 bp\n",
      "  Read 2:    16,387,182 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59320 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t50\t0.0\t1\t0 0 50\n",
      "19\t1117\t0.0\t1\t0 1075 42\n",
      "20\t58008\t0.0\t2\t55251 2690 67\n",
      "21\t98\t0.0\t2\t0 96 2\n",
      "22\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59125 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t979\t0.0\t1\t0 979\n",
      "19\t58049\t0.0\t1\t56677 1372\n",
      "20\t97\t0.0\t1\t0 97\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S397_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S397_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S397_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S397_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S397_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S397_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S397_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S397_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,035 reads  @     59.8 µs/read;   1.00 M reads/minute\n",
      "Finished in 3.18 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,035\n",
      "  Read 1 with adapter:                  52,995 (99.9%)\n",
      "  Read 2 with adapter:                  52,856 (99.7%)\n",
      "Pairs written (passing filters):        53,035 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,290,042 bp\n",
      "  Read 1:    15,645,309 bp\n",
      "  Read 2:    15,644,733 bp\n",
      "Total written (filtered):     29,227,700 bp (93.4%)\n",
      "  Read 1:    14,586,436 bp\n",
      "  Read 2:    14,641,264 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52995 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t62\t0.0\t1\t0 0 62\n",
      "19\t1049\t0.0\t1\t0 980 69\n",
      "20\t51769\t0.0\t2\t49299 2407 63\n",
      "21\t84\t0.0\t2\t0 81 3\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52856 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t907\t0.0\t1\t0 907\n",
      "19\t51837\t0.0\t1\t50630 1207\n",
      "20\t112\t0.0\t1\t0 112\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S398_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S398_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S398_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S398_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S398_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S398_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S398_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S398_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        63,940 reads  @     60.1 µs/read;   1.00 M reads/minute\n",
      "Finished in 3.85 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,940\n",
      "  Read 1 with adapter:                  63,900 (99.9%)\n",
      "  Read 2 with adapter:                  63,692 (99.6%)\n",
      "Pairs written (passing filters):        63,940 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,724,095 bp\n",
      "  Read 1:    18,862,277 bp\n",
      "  Read 2:    18,861,818 bp\n",
      "Total written (filtered):     35,238,017 bp (93.4%)\n",
      "  Read 1:    17,585,415 bp\n",
      "  Read 2:    17,652,602 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63900 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t53\t0.0\t1\t0 0 53\n",
      "19\t1218\t0.0\t1\t0 1162 56\n",
      "20\t62483\t0.0\t2\t59491 2927 65\n",
      "21\t106\t0.0\t2\t0 105 1\n",
      "22\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63692 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1055\t0.0\t1\t0 1055\n",
      "19\t62514\t0.0\t1\t61044 1470\n",
      "20\t123\t0.0\t1\t0 123\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S399_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S399_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S399_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S399_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S399_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S399_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S399_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S399_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,818 reads  @     58.9 µs/read;   1.02 M reads/minute\n",
      "Finished in 3.17 s (59 us/read; 1.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,818\n",
      "  Read 1 with adapter:                  53,776 (99.9%)\n",
      "  Read 2 with adapter:                  53,656 (99.7%)\n",
      "Pairs written (passing filters):        53,818 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,751,106 bp\n",
      "  Read 1:    15,876,287 bp\n",
      "  Read 2:    15,874,819 bp\n",
      "Total written (filtered):     29,657,732 bp (93.4%)\n",
      "  Read 1:    14,801,693 bp\n",
      "  Read 2:    14,856,039 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53776 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t40\t0.0\t1\t0 0 40\n",
      "19\t994\t0.0\t1\t0 944 50\n",
      "20\t52632\t0.0\t2\t50189 2390 53\n",
      "21\t72\t0.0\t2\t0 71 1\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53656 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t761\t0.0\t1\t0 761\n",
      "19\t52818\t0.0\t1\t51671 1147\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S400_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S400_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S400_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S400_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S400_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S400_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S400_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S400_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        59,452 reads  @     60.1 µs/read;   1.00 M reads/minute\n",
      "Finished in 3.58 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,452\n",
      "  Read 1 with adapter:                  59,402 (99.9%)\n",
      "  Read 2 with adapter:                  59,199 (99.6%)\n",
      "Pairs written (passing filters):        59,452 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,070,275 bp\n",
      "  Read 1:    17,538,313 bp\n",
      "  Read 2:    17,531,962 bp\n",
      "Total written (filtered):     32,759,537 bp (93.4%)\n",
      "  Read 1:    16,351,375 bp\n",
      "  Read 2:    16,408,162 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59402 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t57\t0.0\t1\t0 0 57\n",
      "19\t1153\t0.0\t1\t0 1118 35\n",
      "20\t58064\t0.0\t2\t55359 2638 67\n",
      "21\t91\t0.0\t2\t0 90 1\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59199 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1076\t0.0\t1\t0 1076\n",
      "19\t58028\t0.0\t1\t56687 1341\n",
      "20\t95\t0.0\t1\t0 95\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S401_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S401_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S401_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S401_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S401_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S401_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S401_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S401_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:03        61,699 reads  @     59.0 µs/read;   1.02 M reads/minute\n",
      "Finished in 3.65 s (59 us/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,699\n",
      "  Read 1 with adapter:                  61,646 (99.9%)\n",
      "  Read 2 with adapter:                  61,342 (99.4%)\n",
      "Pairs written (passing filters):        61,699 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,380,670 bp\n",
      "  Read 1:    18,201,099 bp\n",
      "  Read 2:    18,179,571 bp\n",
      "Total written (filtered):     33,984,296 bp (93.4%)\n",
      "  Read 1:    16,969,381 bp\n",
      "  Read 2:    17,014,915 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 61646 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t63\t0.0\t1\t0 0 63\n",
      "19\t1229\t0.0\t1\t0 1176 53\n",
      "20\t60232\t0.0\t2\t57250 2921 61\n",
      "21\t91\t0.0\t2\t0 91\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61342 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t927\t0.0\t1\t0 927\n",
      "19\t60330\t0.0\t1\t58944 1386\n",
      "20\t85\t0.0\t1\t0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S402_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S402_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S402_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S402_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S402_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S402_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S402_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S402_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        57,496 reads  @     58.1 µs/read;   1.03 M reads/minute\n",
      "Finished in 3.34 s (58 us/read; 1.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,496\n",
      "  Read 1 with adapter:                  57,456 (99.9%)\n",
      "  Read 2 with adapter:                  56,224 (97.8%)\n",
      "Pairs written (passing filters):        57,496 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,852,633 bp\n",
      "  Read 1:    16,960,831 bp\n",
      "  Read 2:    16,891,802 bp\n",
      "Total written (filtered):     31,636,577 bp (93.5%)\n",
      "  Read 1:    15,812,648 bp\n",
      "  Read 2:    15,823,929 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57456 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t50\t0.0\t1\t0 0 50\n",
      "19\t969\t0.0\t1\t0 936 33\n",
      "20\t56335\t0.0\t2\t53701 2573 61\n",
      "21\t72\t0.0\t2\t0 71 1\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56224 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t466\t0.0\t1\t0 466\n",
      "19\t55675\t0.0\t1\t54590 1085\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S403_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S403_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S403_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S403_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S403_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S403_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S403_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S403_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,935 reads  @     59.7 µs/read;   1.01 M reads/minute\n",
      "Finished in 3.22 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,935\n",
      "  Read 1 with adapter:                  53,887 (99.9%)\n",
      "  Read 2 with adapter:                  53,678 (99.5%)\n",
      "Pairs written (passing filters):        53,935 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,819,583 bp\n",
      "  Read 1:    15,909,389 bp\n",
      "  Read 2:    15,910,194 bp\n",
      "Total written (filtered):     29,723,614 bp (93.4%)\n",
      "  Read 1:    14,832,351 bp\n",
      "  Read 2:    14,891,263 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53887 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t776\t0.0\t1\t0 747 29\n",
      "20\t52993\t0.0\t2\t50394 2539 60\n",
      "21\t82\t0.0\t2\t0 78 4\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53678 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1056\t0.0\t1\t0 1056\n",
      "19\t52517\t0.0\t1\t51199 1318\n",
      "20\t105\t0.0\t1\t0 105\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S404_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S404_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S404_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S404_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S404_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S404_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S404_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S404_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        48,421 reads  @     60.7 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.94 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,421\n",
      "  Read 1 with adapter:                  48,378 (99.9%)\n",
      "  Read 2 with adapter:                  48,201 (99.5%)\n",
      "Pairs written (passing filters):        48,421 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,567,768 bp\n",
      "  Read 1:    14,284,170 bp\n",
      "  Read 2:    14,283,598 bp\n",
      "Total written (filtered):     26,685,734 bp (93.4%)\n",
      "  Read 1:    13,317,241 bp\n",
      "  Read 2:    13,368,493 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48378 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t735\t0.0\t1\t0 702 33\n",
      "20\t47509\t0.0\t2\t45122 2325 62\n",
      "21\t76\t0.0\t2\t0 76\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48201 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t801\t0.0\t1\t0 801\n",
      "19\t47313\t0.0\t1\t46056 1257\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S405_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S405_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S405_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S405_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S405_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S405_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S405_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S405_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        48,890 reads  @     58.9 µs/read;   1.02 M reads/minute\n",
      "Finished in 2.88 s (59 us/read; 1.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,890\n",
      "  Read 1 with adapter:                  48,846 (99.9%)\n",
      "  Read 2 with adapter:                  48,659 (99.5%)\n",
      "Pairs written (passing filters):        48,890 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,844,407 bp\n",
      "  Read 1:    14,422,524 bp\n",
      "  Read 2:    14,421,883 bp\n",
      "Total written (filtered):     26,944,497 bp (93.4%)\n",
      "  Read 1:    13,446,343 bp\n",
      "  Read 2:    13,498,154 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48846 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t49\t0.0\t1\t0 0 49\n",
      "19\t764\t0.0\t1\t0 727 37\n",
      "20\t47929\t0.0\t2\t45385 2490 54\n",
      "21\t85\t0.0\t2\t0 83 2\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48659 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t888\t0.0\t1\t0 888\n",
      "19\t47675\t0.0\t1\t46521 1154\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S406_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S406_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S406_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S406_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S406_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S406_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S406_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S406_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,139 reads  @     63.5 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.11 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,139\n",
      "  Read 1 with adapter:                  33,115 (99.9%)\n",
      "  Read 2 with adapter:                  32,971 (99.5%)\n",
      "Pairs written (passing filters):        33,139 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,551,891 bp\n",
      "  Read 1:     9,775,992 bp\n",
      "  Read 2:     9,775,899 bp\n",
      "Total written (filtered):     18,264,176 bp (93.4%)\n",
      "  Read 1:     9,114,159 bp\n",
      "  Read 2:     9,150,017 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33115 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t556\t0.0\t1\t0 528 28\n",
      "20\t32442\t0.0\t2\t30505 1893 44\n",
      "21\t45\t0.0\t2\t0 45\n",
      "22\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32971 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t639\t0.0\t1\t0 639\n",
      "19\t32260\t0.0\t1\t31178 1082\n",
      "20\t72\t0.0\t1\t0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S407_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S407_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S407_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S407_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S407_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S407_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S407_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S407_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        50,919 reads  @     62.3 µs/read;   0.96 M reads/minute\n",
      "Finished in 3.18 s (62 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,919\n",
      "  Read 1 with adapter:                  50,880 (99.9%)\n",
      "  Read 2 with adapter:                  50,722 (99.6%)\n",
      "Pairs written (passing filters):        50,919 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,040,806 bp\n",
      "  Read 1:    15,021,078 bp\n",
      "  Read 2:    15,019,728 bp\n",
      "Total written (filtered):     28,060,965 bp (93.4%)\n",
      "  Read 1:    14,004,281 bp\n",
      "  Read 2:    14,056,684 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50880 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t43\t0.0\t1\t0 0 43\n",
      "19\t861\t0.0\t1\t0 816 45\n",
      "20\t49863\t0.0\t2\t47285 2516 62\n",
      "21\t82\t0.0\t2\t0 80 2\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50722 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t774\t0.0\t1\t0 774\n",
      "19\t49848\t0.0\t1\t48628 1220\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S408_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S408_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S408_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S408_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S408_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S408_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S408_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S408_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        67,605 reads  @     62.8 µs/read;   0.96 M reads/minute\n",
      "Finished in 4.25 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,605\n",
      "  Read 1 with adapter:                  67,554 (99.9%)\n",
      "  Read 2 with adapter:                  67,134 (99.3%)\n",
      "Pairs written (passing filters):        67,605 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,870,745 bp\n",
      "  Read 1:    19,942,924 bp\n",
      "  Read 2:    19,927,821 bp\n",
      "Total written (filtered):     37,246,268 bp (93.4%)\n",
      "  Read 1:    18,592,830 bp\n",
      "  Read 2:    18,653,438 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 67554 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t41\t0.0\t1\t0 0 41\n",
      "19\t1091\t0.0\t1\t0 1038 53\n",
      "20\t66285\t0.0\t2\t62674 3540 71\n",
      "21\t87\t0.0\t2\t0 83 4\n",
      "22\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 67134 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1268\t0.0\t1\t0 1268\n",
      "19\t65761\t0.0\t1\t64016 1745\n",
      "20\t105\t0.0\t1\t0 105\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S409_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S409_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S409_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S409_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S409_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S409_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S409_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S409_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        59,533 reads  @     62.6 µs/read;   0.96 M reads/minute\n",
      "Finished in 3.73 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,533\n",
      "  Read 1 with adapter:                  59,491 (99.9%)\n",
      "  Read 2 with adapter:                  59,094 (99.3%)\n",
      "Pairs written (passing filters):        59,533 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,095,733 bp\n",
      "  Read 1:    17,561,772 bp\n",
      "  Read 2:    17,533,961 bp\n",
      "Total written (filtered):     32,784,883 bp (93.4%)\n",
      "  Read 1:    16,372,863 bp\n",
      "  Read 2:    16,412,020 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59491 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t51\t0.0\t1\t0 0 51\n",
      "19\t1001\t0.0\t1\t0 956 45\n",
      "20\t58289\t0.0\t2\t55222 2993 74\n",
      "21\t108\t0.0\t2\t0 107 1\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59094 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t932\t0.0\t1\t0 932\n",
      "19\t58075\t0.0\t1\t56730 1345\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S410_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S410_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S410_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S410_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S410_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S410_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S410_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S410_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           137 reads  @    399.1 µs/read;   0.15 M reads/minute\n",
      "Finished in 0.06 s (416 us/read; 0.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                137\n",
      "  Read 1 with adapter:                     132 (96.4%)\n",
      "  Read 2 with adapter:                     131 (95.6%)\n",
      "Pairs written (passing filters):           137 (100.0%)\n",
      "\n",
      "Total basepairs processed:        80,074 bp\n",
      "  Read 1:        39,940 bp\n",
      "  Read 2:        40,134 bp\n",
      "Total written (filtered):         74,949 bp (93.6%)\n",
      "  Read 1:        37,303 bp\n",
      "  Read 2:        37,646 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 132 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t3\t0.0\t1\t0 3\n",
      "20\t129\t0.0\t2\t125 4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 131 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 1\n",
      "19\t130\t0.0\t1\t128 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S411_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S411_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S411_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S411_81_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S411_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S411_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S411_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S411_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        45,124 reads  @     64.1 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.89 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,124\n",
      "  Read 1 with adapter:                  45,031 (99.8%)\n",
      "  Read 2 with adapter:                  44,856 (99.4%)\n",
      "Pairs written (passing filters):        45,124 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,613,946 bp\n",
      "  Read 1:    13,303,456 bp\n",
      "  Read 2:    13,310,490 bp\n",
      "Total written (filtered):     24,862,426 bp (93.4%)\n",
      "  Read 1:    12,403,377 bp\n",
      "  Read 2:    12,459,049 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45031 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t614\t0.0\t1\t0 594 20\n",
      "20\t44300\t0.0\t2\t42123 2131 46\n",
      "21\t65\t0.0\t2\t0 64 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44856 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t911\t0.0\t1\t0 911\n",
      "19\t43857\t0.0\t1\t42626 1231\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S412_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S412_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S412_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S412_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S412_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S412_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S412_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S412_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,593 reads  @     63.3 µs/read;   0.95 M reads/minute\n",
      "Finished in 3.14 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,593\n",
      "  Read 1 with adapter:                  49,544 (99.9%)\n",
      "  Read 2 with adapter:                  49,284 (99.4%)\n",
      "Pairs written (passing filters):        49,593 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,258,913 bp\n",
      "  Read 1:    14,629,621 bp\n",
      "  Read 2:    14,629,292 bp\n",
      "Total written (filtered):     27,332,938 bp (93.4%)\n",
      "  Read 1:    13,639,317 bp\n",
      "  Read 2:    13,693,621 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49544 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t661\t0.0\t1\t0 636 25\n",
      "20\t48770\t0.0\t2\t46410 2297 63\n",
      "21\t65\t0.0\t2\t0 64 1\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49284 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t832\t0.0\t1\t0 832\n",
      "19\t48345\t0.0\t1\t46957 1388\n",
      "20\t107\t0.0\t1\t0 107\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S413_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S413_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S413_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S413_85_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S413_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S413_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S413_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S413_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,699 reads  @     62.9 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.25 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,699\n",
      "  Read 1 with adapter:                  35,672 (99.9%)\n",
      "  Read 2 with adapter:                  35,489 (99.4%)\n",
      "Pairs written (passing filters):        35,699 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,060,207 bp\n",
      "  Read 1:    10,530,821 bp\n",
      "  Read 2:    10,529,386 bp\n",
      "Total written (filtered):     19,673,538 bp (93.4%)\n",
      "  Read 1:     9,817,871 bp\n",
      "  Read 2:     9,855,667 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35672 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t538\t0.0\t1\t0 514 24\n",
      "20\t35044\t0.0\t2\t33207 1798 39\n",
      "21\t48\t0.0\t2\t0 47 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35489 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t631\t0.0\t1\t0 631\n",
      "19\t34799\t0.0\t1\t33795 1004\n",
      "20\t59\t0.0\t1\t0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S414_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S414_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S414_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S414_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S414_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S414_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S414_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S414_87_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,110 reads  @     61.5 µs/read;   0.98 M reads/minute\n",
      "Finished in 2.16 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,110\n",
      "  Read 1 with adapter:                  35,088 (99.9%)\n",
      "  Read 2 with adapter:                  34,930 (99.5%)\n",
      "Pairs written (passing filters):        35,110 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,714,421 bp\n",
      "  Read 1:    10,357,291 bp\n",
      "  Read 2:    10,357,130 bp\n",
      "Total written (filtered):     19,349,975 bp (93.4%)\n",
      "  Read 1:     9,655,974 bp\n",
      "  Read 2:     9,694,001 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35088 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t512\t0.0\t1\t0 482 30\n",
      "20\t34478\t0.0\t2\t32689 1755 34\n",
      "21\t51\t0.0\t2\t0 51\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34930 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t617\t0.0\t1\t0 617\n",
      "19\t34237\t0.0\t1\t33206 1031\n",
      "20\t76\t0.0\t1\t0 76\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S415_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S415_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S415_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S415_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S415_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S415_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S415_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S415_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        55,605 reads  @     62.2 µs/read;   0.97 M reads/minute\n",
      "Finished in 3.46 s (62 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,605\n",
      "  Read 1 with adapter:                  55,563 (99.9%)\n",
      "  Read 2 with adapter:                  55,368 (99.6%)\n",
      "Pairs written (passing filters):        55,603 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,804,784 bp\n",
      "  Read 1:    16,403,335 bp\n",
      "  Read 2:    16,401,449 bp\n",
      "Total written (filtered):     30,642,538 bp (93.4%)\n",
      "  Read 1:    15,292,253 bp\n",
      "  Read 2:    15,350,285 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55563 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t35\t0.0\t1\t0 0 35\n",
      "19\t847\t0.0\t1\t0 815 32\n",
      "20\t54545\t0.0\t2\t51790 2693 62\n",
      "21\t83\t0.0\t2\t0 81 2\n",
      "22\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55368 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t936\t0.0\t1\t0 936\n",
      "19\t54324\t0.0\t1\t52678 1646\n",
      "20\t108\t0.0\t1\t0 108\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S416_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S416_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S416_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S416_91_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S416_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S416_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S416_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S416_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,743 reads  @     64.5 µs/read;   0.93 M reads/minute\n",
      "Finished in 3.34 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,743\n",
      "  Read 1 with adapter:                  51,693 (99.9%)\n",
      "  Read 2 with adapter:                  51,383 (99.3%)\n",
      "Pairs written (passing filters):        51,743 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,515,553 bp\n",
      "  Read 1:    15,263,550 bp\n",
      "  Read 2:    15,252,003 bp\n",
      "Total written (filtered):     28,507,206 bp (93.4%)\n",
      "  Read 1:    14,230,498 bp\n",
      "  Read 2:    14,276,708 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51693 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t46\t0.0\t1\t0 0 46\n",
      "19\t852\t0.0\t1\t0 804 48\n",
      "20\t50702\t0.0\t2\t48041 2590 71\n",
      "21\t50\t0.0\t2\t0 48 2\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51383 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1066\t0.0\t1\t0 1066\n",
      "19\t50233\t0.0\t1\t48785 1448\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S417_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S417_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S417_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S417_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S417_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S417_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S417_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S417_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        50,262 reads  @     65.1 µs/read;   0.92 M reads/minute\n",
      "Finished in 3.27 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,262\n",
      "  Read 1 with adapter:                  50,223 (99.9%)\n",
      "  Read 2 with adapter:                  49,911 (99.3%)\n",
      "Pairs written (passing filters):        50,262 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,631,194 bp\n",
      "  Read 1:    14,827,132 bp\n",
      "  Read 2:    14,804,062 bp\n",
      "Total written (filtered):     27,679,963 bp (93.4%)\n",
      "  Read 1:    13,823,456 bp\n",
      "  Read 2:    13,856,507 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50223 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t34\t0.0\t1\t0 0 34\n",
      "19\t858\t0.0\t1\t0 825 33\n",
      "20\t49217\t0.0\t2\t46683 2462 72\n",
      "21\t86\t0.0\t2\t0 85 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49911 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t816\t0.0\t1\t0 816\n",
      "19\t49033\t0.0\t1\t47864 1169\n",
      "20\t62\t0.0\t1\t0 62\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S418_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S418_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S418_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S418_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S418_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S418_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S418_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S418_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,084 reads  @     72.8 µs/read;   0.82 M reads/minute\n",
      "Finished in 2.26 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,084\n",
      "  Read 1 with adapter:                  31,058 (99.9%)\n",
      "  Read 2 with adapter:                  30,385 (97.8%)\n",
      "Pairs written (passing filters):        31,084 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,302,634 bp\n",
      "  Read 1:     9,169,578 bp\n",
      "  Read 2:     9,133,056 bp\n",
      "Total written (filtered):     17,104,823 bp (93.5%)\n",
      "  Read 1:     8,548,865 bp\n",
      "  Read 2:     8,555,958 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31058 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t491\t0.0\t1\t0 475 16\n",
      "20\t30481\t0.0\t2\t28915 1528 38\n",
      "21\t48\t0.0\t2\t0 48\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30385 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t258\t0.0\t1\t0 258\n",
      "19\t30086\t0.0\t1\t29418 668\n",
      "20\t41\t0.0\t1\t0 41\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S419_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S419_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S419_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S419_97_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S419_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S419_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S419_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S419_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,802 reads  @     62.9 µs/read;   0.95 M reads/minute\n",
      "Finished in 3.45 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,802\n",
      "  Read 1 with adapter:                  54,753 (99.9%)\n",
      "  Read 2 with adapter:                  54,543 (99.5%)\n",
      "Pairs written (passing filters):        54,802 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,332,502 bp\n",
      "  Read 1:    16,166,574 bp\n",
      "  Read 2:    16,165,928 bp\n",
      "Total written (filtered):     30,202,623 bp (93.4%)\n",
      "  Read 1:    15,072,085 bp\n",
      "  Read 2:    15,130,538 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54753 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t38\t0.0\t1\t0 0 38\n",
      "19\t641\t0.0\t1\t0 607 34\n",
      "20\t53959\t0.0\t2\t51274 2638 47\n",
      "21\t84\t0.0\t2\t0 79 5\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54543 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1023\t0.0\t1\t0 1023\n",
      "19\t53424\t0.0\t1\t52129 1295\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S420_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S420_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S420_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S420_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S420_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S420_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S420_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S420_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,933 reads  @     63.6 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.22 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,933\n",
      "  Read 1 with adapter:                  34,897 (99.9%)\n",
      "  Read 2 with adapter:                  34,756 (99.5%)\n",
      "Pairs written (passing filters):        34,933 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,609,637 bp\n",
      "  Read 1:    10,304,924 bp\n",
      "  Read 2:    10,304,713 bp\n",
      "Total written (filtered):     19,252,275 bp (93.4%)\n",
      "  Read 1:     9,607,386 bp\n",
      "  Read 2:     9,644,889 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34897 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t446\t0.0\t1\t0 432 14\n",
      "20\t34358\t0.0\t2\t32551 1763 44\n",
      "21\t50\t0.0\t2\t0 50\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34756 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t590\t0.0\t1\t0 590\n",
      "19\t34116\t0.0\t1\t33234 882\n",
      "20\t50\t0.0\t1\t0 50\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S421_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S421_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S421_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S421_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S421_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S421_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S421_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S421_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        48,153 reads  @     60.4 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.91 s (60 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,153\n",
      "  Read 1 with adapter:                  48,104 (99.9%)\n",
      "  Read 2 with adapter:                  47,914 (99.5%)\n",
      "Pairs written (passing filters):        48,153 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,409,798 bp\n",
      "  Read 1:    14,204,965 bp\n",
      "  Read 2:    14,204,833 bp\n",
      "Total written (filtered):     26,538,666 bp (93.4%)\n",
      "  Read 1:    13,243,430 bp\n",
      "  Read 2:    13,295,236 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48104 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t619\t0.0\t1\t0 582 37\n",
      "20\t47350\t0.0\t2\t44776 2512 62\n",
      "21\t80\t0.0\t2\t0 79 1\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47914 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t853\t0.0\t1\t0 853\n",
      "19\t46977\t0.0\t1\t45795 1182\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S422_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S422_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S422_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S422_103_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S422_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S422_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S422_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S422_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        28,975 reads  @     64.4 µs/read;   0.93 M reads/minute\n",
      "Finished in 1.87 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,975\n",
      "  Read 1 with adapter:                  28,949 (99.9%)\n",
      "  Read 2 with adapter:                  28,836 (99.5%)\n",
      "Pairs written (passing filters):        28,975 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,094,652 bp\n",
      "  Read 1:     8,547,615 bp\n",
      "  Read 2:     8,547,037 bp\n",
      "Total written (filtered):     15,968,523 bp (93.4%)\n",
      "  Read 1:     7,968,971 bp\n",
      "  Read 2:     7,999,552 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 28949 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t383\t0.0\t1\t0 367 16\n",
      "20\t28478\t0.0\t2\t27015 1420 43\n",
      "21\t49\t0.0\t2\t0 48 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 28836 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t470\t0.0\t1\t0 470\n",
      "19\t28295\t0.0\t1\t27497 798\n",
      "20\t71\t0.0\t1\t0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S423_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S423_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S423_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S423_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S423_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S423_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S423_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S423_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        49,352 reads  @     64.6 µs/read;   0.93 M reads/minute\n",
      "Finished in 3.19 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,352\n",
      "  Read 1 with adapter:                  49,307 (99.9%)\n",
      "  Read 2 with adapter:                  49,135 (99.6%)\n",
      "Pairs written (passing filters):        49,352 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,115,438 bp\n",
      "  Read 1:    14,558,822 bp\n",
      "  Read 2:    14,556,616 bp\n",
      "Total written (filtered):     27,197,058 bp (93.4%)\n",
      "  Read 1:    13,573,276 bp\n",
      "  Read 2:    13,623,782 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49307 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t42\t0.0\t1\t0 0 42\n",
      "19\t651\t0.0\t1\t0 608 43\n",
      "20\t48508\t0.0\t2\t46002 2441 65\n",
      "21\t71\t0.0\t2\t0 68 3\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49135 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t802\t0.0\t1\t0 802\n",
      "19\t48262\t0.0\t1\t47014 1248\n",
      "20\t71\t0.0\t1\t0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S424_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S424_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S424_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S424_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S424_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S424_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S424_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S424_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,457 reads  @     68.5 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.50 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,457\n",
      "  Read 1 with adapter:                  36,408 (99.9%)\n",
      "  Read 2 with adapter:                  36,258 (99.5%)\n",
      "Pairs written (passing filters):        36,457 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,503,117 bp\n",
      "  Read 1:    10,754,802 bp\n",
      "  Read 2:    10,748,315 bp\n",
      "Total written (filtered):     20,087,084 bp (93.4%)\n",
      "  Read 1:    10,027,005 bp\n",
      "  Read 2:    10,060,079 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36408 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t433\t0.0\t1\t0 419 14\n",
      "20\t35853\t0.0\t2\t33974 1833 46\n",
      "21\t78\t0.0\t2\t0 76 2\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36258 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t730\t0.0\t1\t0 730\n",
      "19\t35464\t0.0\t1\t34507 957\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S425_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S425_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S425_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S425_109_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S425_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S425_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S425_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S425_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,354 reads  @     65.4 µs/read;   0.92 M reads/minute\n",
      "Finished in 3.43 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,354\n",
      "  Read 1 with adapter:                  52,304 (99.9%)\n",
      "  Read 2 with adapter:                  51,933 (99.2%)\n",
      "Pairs written (passing filters):        52,354 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,860,012 bp\n",
      "  Read 1:    15,444,077 bp\n",
      "  Read 2:    15,415,935 bp\n",
      "Total written (filtered):     28,828,559 bp (93.4%)\n",
      "  Read 1:    14,398,608 bp\n",
      "  Read 2:    14,429,951 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52304 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t32\t0.0\t1\t0 0 32\n",
      "19\t682\t0.0\t1\t0 648 34\n",
      "20\t51477\t0.0\t2\t48830 2577 70\n",
      "21\t91\t0.0\t2\t0 85 6\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51933 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t798\t0.0\t1\t0 798\n",
      "19\t51080\t0.0\t1\t49935 1145\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S426_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S426_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S426_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S426_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S426_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S426_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S426_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S426_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        55,457 reads  @     72.0 µs/read;   0.83 M reads/minute\n",
      "Finished in 4.00 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,457\n",
      "  Read 1 with adapter:                  55,421 (99.9%)\n",
      "  Read 2 with adapter:                  54,159 (97.7%)\n",
      "Pairs written (passing filters):        55,457 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,650,700 bp\n",
      "  Read 1:    16,359,361 bp\n",
      "  Read 2:    16,291,339 bp\n",
      "Total written (filtered):     30,514,207 bp (93.5%)\n",
      "  Read 1:    15,251,553 bp\n",
      "  Read 2:    15,262,654 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55421 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t686\t0.0\t1\t0 655 31\n",
      "20\t54595\t0.0\t2\t51826 2694 75\n",
      "21\t86\t0.0\t2\t0 85 1\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54159 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t408\t0.0\t1\t0 408\n",
      "19\t53679\t0.0\t1\t52566 1113\n",
      "20\t72\t0.0\t1\t0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S427_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S427_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S427_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S427_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S427_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S427_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S427_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S427_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        64,700 reads  @     63.2 µs/read;   0.95 M reads/minute\n",
      "Finished in 4.09 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,700\n",
      "  Read 1 with adapter:                  64,658 (99.9%)\n",
      "  Read 2 with adapter:                  64,312 (99.4%)\n",
      "Pairs written (passing filters):        64,700 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,168,691 bp\n",
      "  Read 1:    19,086,354 bp\n",
      "  Read 2:    19,082,337 bp\n",
      "Total written (filtered):     35,655,493 bp (93.4%)\n",
      "  Read 1:    17,794,008 bp\n",
      "  Read 2:    17,861,485 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64658 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t33\t0.0\t1\t0 0 33\n",
      "19\t874\t0.0\t1\t0 836 38\n",
      "20\t63648\t0.0\t2\t60522 3060 66\n",
      "21\t80\t0.0\t2\t0 80\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 64312 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1209\t0.0\t1\t0 1209\n",
      "19\t62970\t0.0\t1\t61362 1608\n",
      "20\t133\t0.0\t1\t0 133\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S428_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S428_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S428_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S428_115_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S428_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S428_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S428_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S428_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,801 reads  @     62.0 µs/read;   0.97 M reads/minute\n",
      "Finished in 2.72 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,801\n",
      "  Read 1 with adapter:                  43,771 (99.9%)\n",
      "  Read 2 with adapter:                  43,561 (99.5%)\n",
      "Pairs written (passing filters):        43,801 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,841,990 bp\n",
      "  Read 1:    12,921,042 bp\n",
      "  Read 2:    12,920,948 bp\n",
      "Total written (filtered):     24,140,041 bp (93.4%)\n",
      "  Read 1:    12,046,067 bp\n",
      "  Read 2:    12,093,974 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43771 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t531\t0.0\t1\t0 510 21\n",
      "20\t43123\t0.0\t2\t40898 2179 46\n",
      "21\t68\t0.0\t2\t0 68\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43561 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t745\t0.0\t1\t0 745\n",
      "19\t42756\t0.0\t1\t41621 1135\n",
      "20\t60\t0.0\t1\t0 60\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S429_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S429_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S429_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S429_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S429_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S429_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S429_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S429_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        50,363 reads  @     60.1 µs/read;   1.00 M reads/minute\n",
      "Finished in 3.03 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,363\n",
      "  Read 1 with adapter:                  50,338 (100.0%)\n",
      "  Read 2 with adapter:                  50,131 (99.5%)\n",
      "Pairs written (passing filters):        50,363 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,713,272 bp\n",
      "  Read 1:    14,856,863 bp\n",
      "  Read 2:    14,856,409 bp\n",
      "Total written (filtered):     27,755,467 bp (93.4%)\n",
      "  Read 1:    13,850,749 bp\n",
      "  Read 2:    13,904,718 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50338 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t699\t0.0\t1\t0 657 42\n",
      "20\t49522\t0.0\t2\t46866 2574 82\n",
      "21\t69\t0.0\t2\t0 67 2\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50131 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t904\t0.0\t1\t0 904\n",
      "19\t49121\t0.0\t1\t47878 1243\n",
      "20\t106\t0.0\t1\t0 106\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S430_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S430_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S430_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S430_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S430_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S430_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S430_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S430_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        45,285 reads  @     60.5 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.74 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,285\n",
      "  Read 1 with adapter:                  45,262 (99.9%)\n",
      "  Read 2 with adapter:                  45,066 (99.5%)\n",
      "Pairs written (passing filters):        45,285 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,717,707 bp\n",
      "  Read 1:    13,358,707 bp\n",
      "  Read 2:    13,359,000 bp\n",
      "Total written (filtered):     24,957,527 bp (93.4%)\n",
      "  Read 1:    12,454,059 bp\n",
      "  Read 2:    12,503,468 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45262 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t664\t0.0\t1\t0 630 34\n",
      "20\t44498\t0.0\t2\t42161 2267 70\n",
      "21\t64\t0.0\t2\t0 62 2\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45066 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t811\t0.0\t1\t0 811\n",
      "19\t44166\t0.0\t1\t42938 1228\n",
      "20\t89\t0.0\t1\t0 89\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S431_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S431_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S431_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S431_121_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S431_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S431_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S431_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S431_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        29,627 reads  @     63.2 µs/read;   0.95 M reads/minute\n",
      "Finished in 1.87 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,627\n",
      "  Read 1 with adapter:                  29,604 (99.9%)\n",
      "  Read 2 with adapter:                  29,471 (99.5%)\n",
      "Pairs written (passing filters):        29,626 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,477,872 bp\n",
      "  Read 1:     8,739,224 bp\n",
      "  Read 2:     8,738,648 bp\n",
      "Total written (filtered):     16,326,334 bp (93.4%)\n",
      "  Read 1:     8,147,256 bp\n",
      "  Read 2:     8,179,078 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 29604 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t427\t0.0\t1\t0 406 21\n",
      "20\t29101\t0.0\t2\t27553 1516 32\n",
      "21\t40\t0.0\t2\t0 40\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 29471 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t438\t0.0\t1\t0 438\n",
      "19\t28974\t0.0\t1\t28207 767\n",
      "20\t59\t0.0\t1\t0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S432_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S432_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S432_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S432_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S432_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S432_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S432_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S432_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        73,229 reads  @     62.4 µs/read;   0.96 M reads/minute\n",
      "Finished in 4.57 s (62 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,229\n",
      "  Read 1 with adapter:                  73,184 (99.9%)\n",
      "  Read 2 with adapter:                  72,802 (99.4%)\n",
      "Pairs written (passing filters):        73,229 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,191,336 bp\n",
      "  Read 1:    21,602,077 bp\n",
      "  Read 2:    21,589,259 bp\n",
      "Total written (filtered):     40,346,688 bp (93.4%)\n",
      "  Read 1:    20,139,400 bp\n",
      "  Read 2:    20,207,288 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 73184 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t53\t0.0\t1\t0 0 53\n",
      "19\t1109\t0.0\t1\t0 1069 40\n",
      "20\t71859\t0.0\t2\t67954 3814 91\n",
      "21\t114\t0.0\t2\t0 112 2\n",
      "22\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 72802 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1387\t0.0\t1\t0 1387\n",
      "19\t71295\t0.0\t1\t69374 1921\n",
      "20\t120\t0.0\t1\t0 120\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S433_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S433_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S433_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S433_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S433_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S433_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S433_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S433_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,117 reads  @     63.5 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.42 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,117\n",
      "  Read 1 with adapter:                  38,095 (99.9%)\n",
      "  Read 2 with adapter:                  37,882 (99.4%)\n",
      "Pairs written (passing filters):        38,117 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,475,571 bp\n",
      "  Read 1:    11,244,500 bp\n",
      "  Read 2:    11,231,071 bp\n",
      "Total written (filtered):     20,994,989 bp (93.4%)\n",
      "  Read 1:    10,483,148 bp\n",
      "  Read 2:    10,511,841 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38095 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t596\t0.0\t1\t0 569 27\n",
      "20\t37407\t0.0\t2\t35440 1919 48\n",
      "21\t56\t0.0\t2\t0 55 1\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37882 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t576\t0.0\t1\t0 576\n",
      "19\t37258\t0.0\t1\t36391 867\n",
      "20\t48\t0.0\t1\t0 48\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S434_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S434_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S434_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S434_127_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S434_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S434_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S434_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S434_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        56,593 reads  @     64.3 µs/read;   0.93 M reads/minute\n",
      "Finished in 3.64 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,593\n",
      "  Read 1 with adapter:                  56,564 (99.9%)\n",
      "  Read 2 with adapter:                  55,124 (97.4%)\n",
      "Pairs written (passing filters):        56,593 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,306,528 bp\n",
      "  Read 1:    16,694,914 bp\n",
      "  Read 2:    16,611,614 bp\n",
      "Total written (filtered):     31,128,997 bp (93.5%)\n",
      "  Read 1:    15,564,365 bp\n",
      "  Read 2:    15,564,632 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56564 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t40\t0.0\t1\t0 0 40\n",
      "19\t776\t0.0\t1\t0 739 37\n",
      "20\t55643\t0.0\t2\t52826 2744 73\n",
      "21\t85\t0.0\t2\t0 84 1\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55124 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t458\t0.0\t1\t0 458\n",
      "19\t54582\t0.0\t1\t53453 1129\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S435_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S435_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S435_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S435_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S435_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S435_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S435_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S435_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:05        80,131 reads  @     63.4 µs/read;   0.95 M reads/minute\n",
      "Finished in 5.08 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,131\n",
      "  Read 1 with adapter:                  80,042 (99.9%)\n",
      "  Read 2 with adapter:                  79,771 (99.6%)\n",
      "Pairs written (passing filters):        80,131 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,273,584 bp\n",
      "  Read 1:    23,636,959 bp\n",
      "  Read 2:    23,636,625 bp\n",
      "Total written (filtered):     44,159,032 bp (93.4%)\n",
      "  Read 1:    22,036,722 bp\n",
      "  Read 2:    22,122,310 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 80042 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t37\t0.0\t1\t0 0 37\n",
      "19\t709\t0.0\t1\t0 675 34\n",
      "20\t79146\t0.0\t2\t75326 3716 104\n",
      "21\t120\t0.0\t2\t0 118 2\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 79771 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1466\t0.0\t1\t0 1466\n",
      "19\t78173\t0.0\t1\t76344 1829\n",
      "20\t132\t0.0\t1\t0 132\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S436_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S436_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S436_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S436_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S436_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S436_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S436_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S436_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,702 reads  @     60.8 µs/read;   0.99 M reads/minute\n",
      "Finished in 3.21 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,702\n",
      "  Read 1 with adapter:                  52,654 (99.9%)\n",
      "  Read 2 with adapter:                  52,477 (99.6%)\n",
      "Pairs written (passing filters):        52,702 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,093,486 bp\n",
      "  Read 1:    15,546,594 bp\n",
      "  Read 2:    15,546,892 bp\n",
      "Total written (filtered):     29,044,657 bp (93.4%)\n",
      "  Read 1:    14,494,025 bp\n",
      "  Read 2:    14,550,632 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52654 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t562\t0.0\t1\t0 532 30\n",
      "20\t51976\t0.0\t2\t49191 2716 69\n",
      "21\t73\t0.0\t2\t0 71 2\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52477 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t895\t0.0\t1\t0 895\n",
      "19\t51490\t0.0\t1\t50178 1312\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S437_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S437_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S437_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S437_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S437_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S437_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S437_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S437_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        41,924 reads  @     61.2 µs/read;   0.98 M reads/minute\n",
      "Finished in 2.57 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,924\n",
      "  Read 1 with adapter:                  41,884 (99.9%)\n",
      "  Read 2 with adapter:                  41,747 (99.6%)\n",
      "Pairs written (passing filters):        41,924 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,733,460 bp\n",
      "  Read 1:    12,366,937 bp\n",
      "  Read 2:    12,366,523 bp\n",
      "Total written (filtered):     23,103,647 bp (93.4%)\n",
      "  Read 1:    11,529,632 bp\n",
      "  Read 2:    11,574,015 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41884 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t415\t0.0\t1\t0 401 14\n",
      "20\t41360\t0.0\t2\t39063 2235 62\n",
      "21\t62\t0.0\t2\t0 59 3\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41747 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t764\t0.0\t1\t0 764\n",
      "19\t40904\t0.0\t1\t39916 988\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S438_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S438_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S438_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S438_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S438_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S438_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S438_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S438_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,025 reads  @     62.9 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.02 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,025\n",
      "  Read 1 with adapter:                  31,991 (99.9%)\n",
      "  Read 2 with adapter:                  31,887 (99.6%)\n",
      "Pairs written (passing filters):        32,025 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,894,402 bp\n",
      "  Read 1:     9,447,093 bp\n",
      "  Read 2:     9,447,309 bp\n",
      "Total written (filtered):     17,649,461 bp (93.4%)\n",
      "  Read 1:     8,807,546 bp\n",
      "  Read 2:     8,841,915 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31991 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t327\t0.0\t1\t0 309 18\n",
      "20\t31591\t0.0\t2\t29939 1614 38\n",
      "21\t44\t0.0\t2\t0 42 2\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31887 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t523\t0.0\t1\t0 523\n",
      "19\t31300\t0.0\t1\t30504 796\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S439_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S439_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S439_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S439_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S439_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S439_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S439_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S439_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        41,341 reads  @     62.6 µs/read;   0.96 M reads/minute\n",
      "Finished in 2.59 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,341\n",
      "  Read 1 with adapter:                  41,292 (99.9%)\n",
      "  Read 2 with adapter:                  41,192 (99.6%)\n",
      "Pairs written (passing filters):        41,341 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,388,913 bp\n",
      "  Read 1:    12,194,844 bp\n",
      "  Read 2:    12,194,069 bp\n",
      "Total written (filtered):     22,781,391 bp (93.4%)\n",
      "  Read 1:    11,369,369 bp\n",
      "  Read 2:    11,412,022 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41292 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t413\t0.0\t1\t0 395 18\n",
      "20\t40778\t0.0\t2\t38606 2118 54\n",
      "21\t54\t0.0\t2\t0 53 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41192 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t669\t0.0\t1\t0 669\n",
      "19\t40455\t0.0\t1\t39514 941\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S440_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S440_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S440_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S440_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S440_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S440_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S440_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S440_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,917 reads  @     63.8 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.17 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,917\n",
      "  Read 1 with adapter:                  33,884 (99.9%)\n",
      "  Read 2 with adapter:                  33,754 (99.5%)\n",
      "Pairs written (passing filters):        33,917 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,003,753 bp\n",
      "  Read 1:    10,004,952 bp\n",
      "  Read 2:     9,998,801 bp\n",
      "Total written (filtered):     18,685,657 bp (93.4%)\n",
      "  Read 1:     9,327,593 bp\n",
      "  Read 2:     9,358,064 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33884 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t349\t0.0\t1\t0 332 17\n",
      "20\t33454\t0.0\t2\t31703 1706 45\n",
      "21\t38\t0.0\t2\t0 37 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33754 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t634\t0.0\t1\t0 634\n",
      "19\t33075\t0.0\t1\t32335 740\n",
      "20\t45\t0.0\t1\t0 45\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S441_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S441_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S441_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S441_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S441_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S441_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S441_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S441_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        50,737 reads  @     62.3 µs/read;   0.96 M reads/minute\n",
      "Finished in 3.16 s (62 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,737\n",
      "  Read 1 with adapter:                  50,672 (99.9%)\n",
      "  Read 2 with adapter:                  50,396 (99.3%)\n",
      "Pairs written (passing filters):        50,737 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,912,864 bp\n",
      "  Read 1:    14,966,065 bp\n",
      "  Read 2:    14,946,799 bp\n",
      "Total written (filtered):     27,943,082 bp (93.4%)\n",
      "  Read 1:    13,953,087 bp\n",
      "  Read 2:    13,989,995 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50672 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t513\t0.0\t1\t0 497 16\n",
      "20\t50049\t0.0\t2\t47331 2651 67\n",
      "21\t65\t0.0\t2\t0 61 4\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50396 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t793\t0.0\t1\t0 793\n",
      "19\t49530\t0.0\t1\t48466 1064\n",
      "20\t73\t0.0\t1\t0 73\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S442_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S442_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S442_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S442_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S442_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S442_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S442_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S442_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,723 reads  @     61.5 µs/read;   0.97 M reads/minute\n",
      "Finished in 3.12 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,723\n",
      "  Read 1 with adapter:                  50,659 (99.9%)\n",
      "  Read 2 with adapter:                  49,568 (97.7%)\n",
      "Pairs written (passing filters):        50,723 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,857,782 bp\n",
      "  Read 1:    14,960,957 bp\n",
      "  Read 2:    14,896,825 bp\n",
      "Total written (filtered):     27,903,631 bp (93.5%)\n",
      "  Read 1:    13,948,257 bp\n",
      "  Read 2:    13,955,374 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50659 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t524\t0.0\t1\t0 502 22\n",
      "20\t50036\t0.0\t2\t47436 2537 63\n",
      "21\t62\t0.0\t2\t0 61 1\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49568 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t398\t0.0\t1\t0 398\n",
      "19\t49113\t0.0\t1\t48121 992\n",
      "20\t57\t0.0\t1\t0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S443_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S443_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S443_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S443_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S443_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S443_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S443_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S443_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,664 reads  @     58.6 µs/read;   1.02 M reads/minute\n",
      "Finished in 3.15 s (59 us/read; 1.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,664\n",
      "  Read 1 with adapter:                  53,618 (99.9%)\n",
      "  Read 2 with adapter:                  53,423 (99.6%)\n",
      "Pairs written (passing filters):        53,664 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,661,244 bp\n",
      "  Read 1:    15,830,669 bp\n",
      "  Read 2:    15,830,575 bp\n",
      "Total written (filtered):     29,575,085 bp (93.4%)\n",
      "  Read 1:    14,758,570 bp\n",
      "  Read 2:    14,816,515 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53618 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t378\t0.0\t1\t0 360 18\n",
      "20\t53113\t0.0\t2\t50699 2371 43\n",
      "21\t81\t0.0\t2\t0 80 1\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53423 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1089\t0.0\t1\t0 1089\n",
      "19\t52222\t0.0\t1\t50910 1312\n",
      "20\t112\t0.0\t1\t0 112\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S444_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S444_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S444_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S444_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S444_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S444_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S444_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S444_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        52,375 reads  @     57.1 µs/read;   1.05 M reads/minute\n",
      "Finished in 3.00 s (57 us/read; 1.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,375\n",
      "  Read 1 with adapter:                  52,338 (99.9%)\n",
      "  Read 2 with adapter:                  52,172 (99.6%)\n",
      "Pairs written (passing filters):        52,375 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,900,207 bp\n",
      "  Read 1:    15,450,052 bp\n",
      "  Read 2:    15,450,155 bp\n",
      "Total written (filtered):     28,863,228 bp (93.4%)\n",
      "  Read 1:    14,403,534 bp\n",
      "  Read 2:    14,459,694 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52338 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t423\t0.0\t1\t0 406 17\n",
      "20\t51728\t0.0\t2\t49278 2408 42\n",
      "21\t97\t0.0\t2\t0 94 3\n",
      "22\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52172 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t906\t0.0\t1\t0 906\n",
      "19\t51167\t0.0\t1\t49866 1301\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S445_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S445_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S445_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S445_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S445_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S445_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S445_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S445_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:01        19,954 reads  @     58.7 µs/read;   1.02 M reads/minute\n",
      "Finished in 1.17 s (59 us/read; 1.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             19,954\n",
      "  Read 1 with adapter:                  19,939 (99.9%)\n",
      "  Read 2 with adapter:                  19,874 (99.6%)\n",
      "Pairs written (passing filters):        19,954 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,772,659 bp\n",
      "  Read 1:     5,886,425 bp\n",
      "  Read 2:     5,886,234 bp\n",
      "Total written (filtered):     10,996,675 bp (93.4%)\n",
      "  Read 1:     5,487,744 bp\n",
      "  Read 2:     5,508,931 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 19939 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t161\t0.0\t1\t0 155 6\n",
      "20\t19718\t0.0\t2\t18809 888 21\n",
      "21\t30\t0.0\t2\t0 29 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 19874 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t336\t0.0\t1\t0 336\n",
      "19\t19505\t0.0\t1\t19021 484\n",
      "20\t33\t0.0\t1\t0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S446_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S446_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S446_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S446_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S446_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S446_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S446_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S446_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,756 reads  @     63.3 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.20 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,756\n",
      "  Read 1 with adapter:                  34,725 (99.9%)\n",
      "  Read 2 with adapter:                  34,570 (99.5%)\n",
      "Pairs written (passing filters):        34,756 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,505,531 bp\n",
      "  Read 1:    10,252,529 bp\n",
      "  Read 2:    10,253,002 bp\n",
      "Total written (filtered):     19,154,795 bp (93.4%)\n",
      "  Read 1:     9,558,058 bp\n",
      "  Read 2:     9,596,737 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34725 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t312\t0.0\t1\t0 297 15\n",
      "20\t34211\t0.0\t2\t32356 1803 52\n",
      "21\t53\t0.0\t2\t0 50 3\n",
      "22\t132\t0.0\t2\t0 0 132\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34570 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t647\t0.0\t1\t0 647\n",
      "19\t33841\t0.0\t1\t32742 1099\n",
      "20\t82\t0.0\t1\t0 82\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S447_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S447_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S447_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S447_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S447_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S447_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S447_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S447_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        41,375 reads  @     60.7 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.51 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,375\n",
      "  Read 1 with adapter:                  41,342 (99.9%)\n",
      "  Read 2 with adapter:                  41,233 (99.7%)\n",
      "Pairs written (passing filters):        41,375 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,410,011 bp\n",
      "  Read 1:    12,205,096 bp\n",
      "  Read 2:    12,204,915 bp\n",
      "Total written (filtered):     22,800,588 bp (93.4%)\n",
      "  Read 1:    11,378,486 bp\n",
      "  Read 2:    11,422,102 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41342 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t376\t0.0\t1\t0 358 18\n",
      "20\t40822\t0.0\t2\t38864 1925 33\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41233 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t682\t0.0\t1\t0 682\n",
      "19\t40483\t0.0\t1\t39460 1023\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S448_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S448_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S448_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S448_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S448_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S448_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S448_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S448_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,688 reads  @     62.1 µs/read;   0.97 M reads/minute\n",
      "Finished in 3.28 s (62 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,688\n",
      "  Read 1 with adapter:                  52,650 (99.9%)\n",
      "  Read 2 with adapter:                  52,411 (99.5%)\n",
      "Pairs written (passing filters):        52,688 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,077,550 bp\n",
      "  Read 1:    15,542,894 bp\n",
      "  Read 2:    15,534,656 bp\n",
      "Total written (filtered):     29,029,974 bp (93.4%)\n",
      "  Read 1:    14,490,186 bp\n",
      "  Read 2:    14,539,788 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52650 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t475\t0.0\t1\t0 457 18\n",
      "20\t51971\t0.0\t2\t49574 2347 50\n",
      "21\t105\t0.0\t2\t0 103 2\n",
      "22\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52411 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1026\t0.0\t1\t0 1026\n",
      "19\t51300\t0.0\t1\t49996 1304\n",
      "20\t85\t0.0\t1\t0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S449_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S449_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S449_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S449_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S449_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S449_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S449_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S449_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        64,264 reads  @     63.5 µs/read;   0.95 M reads/minute\n",
      "Finished in 4.08 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,264\n",
      "  Read 1 with adapter:                  64,213 (99.9%)\n",
      "  Read 2 with adapter:                  63,864 (99.4%)\n",
      "Pairs written (passing filters):        64,264 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,885,747 bp\n",
      "  Read 1:    18,957,568 bp\n",
      "  Read 2:    18,928,179 bp\n",
      "Total written (filtered):     35,389,390 bp (93.4%)\n",
      "  Read 1:    17,673,694 bp\n",
      "  Read 2:    17,715,696 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64213 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t581\t0.0\t1\t0 551 30\n",
      "20\t63410\t0.0\t2\t60439 2910 61\n",
      "21\t137\t0.0\t2\t0 135 2\n",
      "22\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63864 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1020\t0.0\t1\t0 1020\n",
      "19\t62757\t0.0\t1\t61344 1413\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S450_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S450_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S450_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S450_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S450_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S450_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S450_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S450_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           269 reads  @    266.3 µs/read;   0.23 M reads/minute\n",
      "Finished in 0.07 s (274 us/read; 0.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                269\n",
      "  Read 1 with adapter:                     266 (98.9%)\n",
      "  Read 2 with adapter:                     261 (97.0%)\n",
      "Pairs written (passing filters):           269 (100.0%)\n",
      "\n",
      "Total basepairs processed:       158,454 bp\n",
      "  Read 1:        79,355 bp\n",
      "  Read 2:        79,099 bp\n",
      "Total written (filtered):        148,176 bp (93.5%)\n",
      "  Read 1:        74,035 bp\n",
      "  Read 2:        74,141 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 266 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t3\t0.0\t1\t0 2 1\n",
      "20\t261\t0.0\t2\t256 5\n",
      "21\t1\t0.0\t2\t0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 261 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t2\t0.0\t1\t0 2\n",
      "19\t258\t0.0\t1\t251 7\n",
      "20\t1\t0.0\t1\t0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S451_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S451_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S451_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S451_161_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S451_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S451_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S451_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S451_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,664 reads  @     64.3 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.10 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,664\n",
      "  Read 1 with adapter:                  32,632 (99.9%)\n",
      "  Read 2 with adapter:                  32,474 (99.4%)\n",
      "Pairs written (passing filters):        32,664 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,270,405 bp\n",
      "  Read 1:     9,635,173 bp\n",
      "  Read 2:     9,635,232 bp\n",
      "Total written (filtered):     18,001,407 bp (93.4%)\n",
      "  Read 1:     8,982,670 bp\n",
      "  Read 2:     9,018,737 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32632 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t210\t0.0\t1\t0 202 8\n",
      "20\t32321\t0.0\t2\t30808 1480 33\n",
      "21\t69\t0.0\t2\t0 69\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32474 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t591\t0.0\t1\t0 591\n",
      "19\t31803\t0.0\t1\t30916 887\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S452_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S452_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S452_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S452_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S452_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S452_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S452_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S452_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        57,066 reads  @     61.5 µs/read;   0.98 M reads/minute\n",
      "Finished in 3.51 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,066\n",
      "  Read 1 with adapter:                  57,008 (99.9%)\n",
      "  Read 2 with adapter:                  56,707 (99.4%)\n",
      "Pairs written (passing filters):        57,066 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,647,990 bp\n",
      "  Read 1:    16,833,862 bp\n",
      "  Read 2:    16,814,128 bp\n",
      "Total written (filtered):     31,432,151 bp (93.4%)\n",
      "  Read 1:    15,694,761 bp\n",
      "  Read 2:    15,737,390 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57008 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t55\t0.0\t1\t0 0 55\n",
      "19\t1060\t0.0\t1\t0 1001 59\n",
      "20\t55806\t0.0\t2\t52821 2920 65\n",
      "21\t63\t0.0\t2\t0 61 2\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56707 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t770\t0.0\t1\t0 770\n",
      "19\t55862\t0.0\t1\t54404 1458\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S453_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S453_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S453_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S453_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S453_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S453_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S453_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S453_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        23,439 reads  @     65.7 µs/read;   0.91 M reads/minute\n",
      "Finished in 1.54 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,439\n",
      "  Read 1 with adapter:                  23,417 (99.9%)\n",
      "  Read 2 with adapter:                  23,305 (99.4%)\n",
      "Pairs written (passing filters):        23,439 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,828,599 bp\n",
      "  Read 1:     6,914,407 bp\n",
      "  Read 2:     6,914,192 bp\n",
      "Total written (filtered):     12,918,136 bp (93.4%)\n",
      "  Read 1:     6,446,522 bp\n",
      "  Read 2:     6,471,614 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 23417 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t460\t0.0\t1\t0 435 25\n",
      "20\t22886\t0.0\t2\t21670 1176 40\n",
      "21\t37\t0.0\t2\t0 37\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 23305 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t235\t0.0\t1\t0 235\n",
      "19\t23052\t0.0\t1\t22447 605\n",
      "20\t18\t0.0\t1\t0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S454_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S454_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S454_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S454_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S454_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S454_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S454_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S454_167_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,918 reads  @     61.8 µs/read;   0.97 M reads/minute\n",
      "Finished in 3.40 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,918\n",
      "  Read 1 with adapter:                  54,865 (99.9%)\n",
      "  Read 2 with adapter:                  54,690 (99.6%)\n",
      "Pairs written (passing filters):        54,918 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,400,072 bp\n",
      "  Read 1:    16,200,322 bp\n",
      "  Read 2:    16,199,750 bp\n",
      "Total written (filtered):     30,265,210 bp (93.4%)\n",
      "  Read 1:    15,104,056 bp\n",
      "  Read 2:    15,161,154 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54865 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t55\t0.0\t1\t0 0 55\n",
      "19\t1050\t0.0\t1\t0 995 55\n",
      "20\t53658\t0.0\t2\t50831 2748 79\n",
      "21\t78\t0.0\t2\t0 78\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54690 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t580\t0.0\t1\t0 580\n",
      "19\t54044\t0.0\t1\t52481 1563\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S455_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S455_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S455_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S455_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S455_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S455_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S455_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S455_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        47,829 reads  @     61.6 µs/read;   0.97 M reads/minute\n",
      "Finished in 2.95 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,829\n",
      "  Read 1 with adapter:                  47,772 (99.9%)\n",
      "  Read 2 with adapter:                  47,471 (99.3%)\n",
      "Pairs written (passing filters):        47,829 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,201,978 bp\n",
      "  Read 1:    14,106,261 bp\n",
      "  Read 2:    14,095,717 bp\n",
      "Total written (filtered):     26,345,689 bp (93.4%)\n",
      "  Read 1:    13,151,046 bp\n",
      "  Read 2:    13,194,643 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47772 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t350\t0.0\t1\t0 336 14\n",
      "20\t47303\t0.0\t2\t44957 2293 53\n",
      "21\t81\t0.0\t2\t0 80 1\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47471 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t951\t0.0\t1\t0 951\n",
      "19\t46444\t0.0\t1\t45187 1257\n",
      "20\t76\t0.0\t1\t0 76\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S456_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S456_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S456_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S456_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S456_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S456_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S456_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S456_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        24,080 reads  @     66.0 µs/read;   0.91 M reads/minute\n",
      "Finished in 1.59 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,080\n",
      "  Read 1 with adapter:                  24,061 (99.9%)\n",
      "  Read 2 with adapter:                  23,917 (99.3%)\n",
      "Pairs written (passing filters):        24,080 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,199,805 bp\n",
      "  Read 1:     7,103,589 bp\n",
      "  Read 2:     7,096,216 bp\n",
      "Total written (filtered):     13,264,823 bp (93.4%)\n",
      "  Read 1:     6,622,682 bp\n",
      "  Read 2:     6,642,141 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 24061 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t359\t0.0\t1\t0 346 13\n",
      "20\t23618\t0.0\t2\t22311 1276 31\n",
      "21\t42\t0.0\t2\t0 38 4\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 23917 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t384\t0.0\t1\t0 384\n",
      "19\t23497\t0.0\t1\t22887 610\n",
      "20\t36\t0.0\t1\t0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S457_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S457_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S457_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S457_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S457_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S457_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S457_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S457_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        25,314 reads  @     65.7 µs/read;   0.91 M reads/minute\n",
      "Finished in 1.66 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,314\n",
      "  Read 1 with adapter:                  25,290 (99.9%)\n",
      "  Read 2 with adapter:                  25,191 (99.5%)\n",
      "Pairs written (passing filters):        25,314 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,934,839 bp\n",
      "  Read 1:     7,467,431 bp\n",
      "  Read 2:     7,467,408 bp\n",
      "Total written (filtered):     13,951,083 bp (93.4%)\n",
      "  Read 1:     6,961,995 bp\n",
      "  Read 2:     6,989,088 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 25290 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t422\t0.0\t1\t0 407 15\n",
      "20\t24791\t0.0\t2\t23499 1261 31\n",
      "21\t44\t0.0\t2\t0 43 1\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 25191 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t343\t0.0\t1\t0 343\n",
      "19\t24814\t0.0\t1\t24131 683\n",
      "20\t34\t0.0\t1\t0 34\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S458_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S458_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S458_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S458_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S458_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S458_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S458_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S458_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,026 reads  @     69.4 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.34 s (69 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,026\n",
      "  Read 1 with adapter:                  47,980 (99.9%)\n",
      "  Read 2 with adapter:                  47,855 (99.6%)\n",
      "Pairs written (passing filters):        48,026 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,331,681 bp\n",
      "  Read 1:    14,164,906 bp\n",
      "  Read 2:    14,166,775 bp\n",
      "Total written (filtered):     26,463,966 bp (93.4%)\n",
      "  Read 1:    13,205,971 bp\n",
      "  Read 2:    13,257,995 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47980 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t740\t0.0\t1\t0 701 39\n",
      "20\t47119\t0.0\t2\t44659 2401 59\n",
      "21\t67\t0.0\t2\t0 64 3\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47855 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t537\t0.0\t1\t0 537\n",
      "19\t47246\t0.0\t1\t45984 1262\n",
      "20\t72\t0.0\t1\t0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S459_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S459_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S459_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S459_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S459_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S459_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S459_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S459_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        57,590 reads  @     62.1 µs/read;   0.97 M reads/minute\n",
      "Finished in 3.58 s (62 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,590\n",
      "  Read 1 with adapter:                  57,528 (99.9%)\n",
      "  Read 2 with adapter:                  57,283 (99.5%)\n",
      "Pairs written (passing filters):        57,590 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,974,104 bp\n",
      "  Read 1:    16,986,243 bp\n",
      "  Read 2:    16,987,861 bp\n",
      "Total written (filtered):     31,736,285 bp (93.4%)\n",
      "  Read 1:    15,835,907 bp\n",
      "  Read 2:    15,900,378 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57528 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t376\t0.0\t1\t0 365 11\n",
      "20\t56982\t0.0\t2\t54194 2724 64\n",
      "21\t100\t0.0\t2\t0 97 3\n",
      "22\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57283 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t993\t0.0\t1\t0 993\n",
      "19\t56191\t0.0\t1\t54799 1392\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S460_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S460_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S460_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S460_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S460_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S460_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S460_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S460_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        60,568 reads  @     61.5 µs/read;   0.98 M reads/minute\n",
      "Finished in 3.73 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,568\n",
      "  Read 1 with adapter:                  60,520 (99.9%)\n",
      "  Read 2 with adapter:                  60,257 (99.5%)\n",
      "Pairs written (passing filters):        60,568 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,720,578 bp\n",
      "  Read 1:    17,867,550 bp\n",
      "  Read 2:    17,853,028 bp\n",
      "Total written (filtered):     33,366,542 bp (93.4%)\n",
      "  Read 1:    16,657,601 bp\n",
      "  Read 2:    16,708,941 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60520 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t556\t0.0\t1\t0 528 28\n",
      "20\t59833\t0.0\t2\t56645 3120 68\n",
      "21\t73\t0.0\t2\t0 70 3\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 60257 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t907\t0.0\t1\t0 907\n",
      "19\t59239\t0.0\t1\t57891 1348\n",
      "20\t111\t0.0\t1\t0 111\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S461_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S461_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S461_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S461_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S461_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S461_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S461_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S461_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        61,276 reads  @     60.4 µs/read;   0.99 M reads/minute\n",
      "Finished in 3.70 s (60 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,276\n",
      "  Read 1 with adapter:                  61,222 (99.9%)\n",
      "  Read 2 with adapter:                  61,055 (99.6%)\n",
      "Pairs written (passing filters):        61,276 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,152,031 bp\n",
      "  Read 1:    18,075,969 bp\n",
      "  Read 2:    18,076,062 bp\n",
      "Total written (filtered):     33,768,542 bp (93.4%)\n",
      "  Read 1:    16,851,911 bp\n",
      "  Read 2:    16,916,631 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 61222 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t503\t0.0\t1\t0 484 19\n",
      "20\t60561\t0.0\t2\t57420 3058 83\n",
      "21\t103\t0.0\t2\t0 101 2\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61055 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t701\t0.0\t1\t0 701\n",
      "19\t60267\t0.0\t1\t58962 1305\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S462_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S462_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S462_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S462_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S462_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S462_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S462_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S462_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,983 reads  @     62.5 µs/read;   0.96 M reads/minute\n",
      "Finished in 2.75 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,983\n",
      "  Read 1 with adapter:                  43,940 (99.9%)\n",
      "  Read 2 with adapter:                  43,812 (99.6%)\n",
      "Pairs written (passing filters):        43,983 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,949,409 bp\n",
      "  Read 1:    12,974,734 bp\n",
      "  Read 2:    12,974,675 bp\n",
      "Total written (filtered):     24,238,948 bp (93.4%)\n",
      "  Read 1:    12,096,277 bp\n",
      "  Read 2:    12,142,671 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43940 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t422\t0.0\t1\t0 393 29\n",
      "20\t43423\t0.0\t2\t41143 2232 48\n",
      "21\t55\t0.0\t2\t0 53 2\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43812 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t480\t0.0\t1\t0 480\n",
      "19\t43276\t0.0\t1\t42241 1035\n",
      "20\t56\t0.0\t1\t0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S463_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S463_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S463_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S463_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S463_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S463_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S463_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S463_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04        65,890 reads  @     64.4 µs/read;   0.93 M reads/minute\n",
      "Finished in 4.24 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,890\n",
      "  Read 1 with adapter:                  65,824 (99.9%)\n",
      "  Read 2 with adapter:                  65,452 (99.3%)\n",
      "Pairs written (passing filters):        65,890 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,842,127 bp\n",
      "  Read 1:    19,435,185 bp\n",
      "  Read 2:    19,406,942 bp\n",
      "Total written (filtered):     36,283,217 bp (93.4%)\n",
      "  Read 1:    18,118,939 bp\n",
      "  Read 2:    18,164,278 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 65824 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t399\t0.0\t1\t0 378 21\n",
      "20\t65248\t0.0\t2\t62026 3151 71\n",
      "21\t121\t0.0\t2\t0 118 3\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 65452 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1001\t0.0\t1\t0 1001\n",
      "19\t64374\t0.0\t1\t62951 1423\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S464_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S464_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S464_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S464_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S464_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S464_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S464_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S464_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        60,330 reads  @     68.4 µs/read;   0.88 M reads/minute\n",
      "Finished in 4.13 s (69 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,330\n",
      "  Read 1 with adapter:                  60,278 (99.9%)\n",
      "  Read 2 with adapter:                  59,980 (99.4%)\n",
      "Pairs written (passing filters):        60,330 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,577,871 bp\n",
      "  Read 1:    17,797,223 bp\n",
      "  Read 2:    17,780,648 bp\n",
      "Total written (filtered):     33,234,691 bp (93.4%)\n",
      "  Read 1:    16,592,780 bp\n",
      "  Read 2:    16,641,911 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60278 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t56\t0.0\t1\t0 0 56\n",
      "19\t1173\t0.0\t1\t0 1128 45\n",
      "20\t58920\t0.0\t2\t56144 2719 57\n",
      "21\t90\t0.0\t2\t0 87 3\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59980 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t976\t0.0\t1\t0 976\n",
      "19\t58911\t0.0\t1\t57538 1373\n",
      "20\t93\t0.0\t1\t0 93\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S465_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S465_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S465_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S465_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S465_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S465_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S465_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S465_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,628 reads  @     65.2 µs/read;   0.92 M reads/minute\n",
      "Finished in 3.17 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,628\n",
      "  Read 1 with adapter:                  48,601 (99.9%)\n",
      "  Read 2 with adapter:                  48,423 (99.6%)\n",
      "Pairs written (passing filters):        48,628 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,690,448 bp\n",
      "  Read 1:    14,345,248 bp\n",
      "  Read 2:    14,345,200 bp\n",
      "Total written (filtered):     26,799,847 bp (93.4%)\n",
      "  Read 1:    13,374,097 bp\n",
      "  Read 2:    13,425,750 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48601 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t959\t0.0\t1\t0 926 33\n",
      "20\t47475\t0.0\t2\t45343 2095 37\n",
      "21\t88\t0.0\t2\t0 86 2\n",
      "22\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48423 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t655\t0.0\t1\t0 655\n",
      "19\t47700\t0.0\t1\t46610 1090\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S466_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S466_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S466_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S466_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S466_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S466_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S466_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S466_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,672 reads  @     70.8 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.67 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,672\n",
      "  Read 1 with adapter:                  51,638 (99.9%)\n",
      "  Read 2 with adapter:                  51,452 (99.6%)\n",
      "Pairs written (passing filters):        51,672 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,480,750 bp\n",
      "  Read 1:    15,243,227 bp\n",
      "  Read 2:    15,237,523 bp\n",
      "Total written (filtered):     28,471,959 bp (93.4%)\n",
      "  Read 1:    14,211,483 bp\n",
      "  Read 2:    14,260,476 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51638 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t57\t0.0\t1\t0 0 57\n",
      "19\t1082\t0.0\t1\t0 1029 53\n",
      "20\t50365\t0.0\t2\t47966 2330 69\n",
      "21\t88\t0.0\t2\t0 88\n",
      "22\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51452 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t607\t0.0\t1\t0 607\n",
      "19\t50779\t0.0\t1\t49588 1191\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S467_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S467_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S467_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S467_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S467_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S467_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S467_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S467_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        66,128 reads  @     62.8 µs/read;   0.96 M reads/minute\n",
      "Finished in 4.16 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,128\n",
      "  Read 1 with adapter:                  66,065 (99.9%)\n",
      "  Read 2 with adapter:                  65,657 (99.3%)\n",
      "Pairs written (passing filters):        66,128 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,011,614 bp\n",
      "  Read 1:    19,506,791 bp\n",
      "  Read 2:    19,504,823 bp\n",
      "Total written (filtered):     36,444,197 bp (93.4%)\n",
      "  Read 1:    18,185,784 bp\n",
      "  Read 2:    18,258,413 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 66065 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t468\t0.0\t1\t0 447 21\n",
      "20\t65423\t0.0\t2\t62067 3283 73\n",
      "21\t117\t0.0\t2\t0 115 2\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 65657 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1175\t0.0\t1\t0 1175\n",
      "19\t64380\t0.0\t1\t62448 1932\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S468_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S468_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S468_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S468_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S468_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S468_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S468_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S468_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        55,723 reads  @     61.7 µs/read;   0.97 M reads/minute\n",
      "Finished in 3.44 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,723\n",
      "  Read 1 with adapter:                  55,679 (99.9%)\n",
      "  Read 2 with adapter:                  55,328 (99.3%)\n",
      "Pairs written (passing filters):        55,723 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,857,054 bp\n",
      "  Read 1:    16,438,265 bp\n",
      "  Read 2:    16,418,789 bp\n",
      "Total written (filtered):     30,693,849 bp (93.4%)\n",
      "  Read 1:    15,325,511 bp\n",
      "  Read 2:    15,368,338 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55679 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t41\t0.0\t1\t0 0 41\n",
      "19\t903\t0.0\t1\t0 868 35\n",
      "20\t54609\t0.0\t2\t51596 2933 80\n",
      "21\t93\t0.0\t2\t0 89 4\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55328 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t870\t0.0\t1\t0 870\n",
      "19\t54369\t0.0\t1\t52846 1523\n",
      "20\t89\t0.0\t1\t0 89\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S469_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S469_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S469_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S469_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S469_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S469_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S469_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S469_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        56,804 reads  @     60.9 µs/read;   0.98 M reads/minute\n",
      "Finished in 3.47 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,804\n",
      "  Read 1 with adapter:                  56,753 (99.9%)\n",
      "  Read 2 with adapter:                  56,556 (99.6%)\n",
      "Pairs written (passing filters):        56,804 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,513,974 bp\n",
      "  Read 1:    16,757,076 bp\n",
      "  Read 2:    16,756,898 bp\n",
      "Total written (filtered):     31,305,774 bp (93.4%)\n",
      "  Read 1:    15,622,899 bp\n",
      "  Read 2:    15,682,875 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56753 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t978\t0.0\t1\t0 929 49\n",
      "20\t55598\t0.0\t2\t52600 2917 81\n",
      "21\t103\t0.0\t2\t0 100 3\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56556 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t624\t0.0\t1\t0 624\n",
      "19\t55849\t0.0\t1\t54341 1508\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S470_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S470_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S470_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S470_199_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S470_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S470_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S470_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S470_199_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        61,681 reads  @     61.4 µs/read;   0.98 M reads/minute\n",
      "Finished in 3.79 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,681\n",
      "  Read 1 with adapter:                  61,639 (99.9%)\n",
      "  Read 2 with adapter:                  61,377 (99.5%)\n",
      "Pairs written (passing filters):        61,681 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,390,075 bp\n",
      "  Read 1:    18,195,653 bp\n",
      "  Read 2:    18,194,422 bp\n",
      "Total written (filtered):     33,992,724 bp (93.4%)\n",
      "  Read 1:    16,963,840 bp\n",
      "  Read 2:    17,028,884 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 61639 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t55\t0.0\t1\t0 0 55\n",
      "19\t1053\t0.0\t1\t0 996 57\n",
      "20\t60379\t0.0\t2\t57059 3244 76\n",
      "21\t108\t0.0\t2\t0 107 1\n",
      "22\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61377 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t703\t0.0\t1\t0 703\n",
      "19\t60596\t0.0\t1\t58797 1799\n",
      "20\t78\t0.0\t1\t0 78\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S471_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S471_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S471_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S471_201_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S471_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S471_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S471_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S471_201_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        67,892 reads  @     63.4 µs/read;   0.95 M reads/minute\n",
      "Finished in 4.31 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,892\n",
      "  Read 1 with adapter:                  67,822 (99.9%)\n",
      "  Read 2 with adapter:                  65,765 (96.9%)\n",
      "Pairs written (passing filters):        67,892 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,933,723 bp\n",
      "  Read 1:    20,026,579 bp\n",
      "  Read 2:    19,907,144 bp\n",
      "Total written (filtered):     37,328,479 bp (93.5%)\n",
      "  Read 1:    18,670,399 bp\n",
      "  Read 2:    18,658,080 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 67822 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t439\t0.0\t1\t0 419 20\n",
      "20\t67201\t0.0\t2\t63803 3316 82\n",
      "21\t117\t0.0\t2\t0 115 2\n",
      "22\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 65765 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t559\t0.0\t1\t0 559\n",
      "19\t65118\t0.0\t1\t63390 1728\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S472_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S472_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S472_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S472_203_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S472_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S472_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S472_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S472_203_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,730 reads  @     64.2 µs/read;   0.93 M reads/minute\n",
      "Finished in 3.07 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,730\n",
      "  Read 1 with adapter:                  47,644 (99.8%)\n",
      "  Read 2 with adapter:                  47,314 (99.1%)\n",
      "Pairs written (passing filters):        47,730 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,130,658 bp\n",
      "  Read 1:    14,072,627 bp\n",
      "  Read 2:    14,058,031 bp\n",
      "Total written (filtered):     26,280,144 bp (93.4%)\n",
      "  Read 1:    13,120,370 bp\n",
      "  Read 2:    13,159,774 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47644 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t706\t0.0\t1\t0 675 31\n",
      "20\t46798\t0.0\t2\t44364 2369 65\n",
      "21\t81\t0.0\t2\t0 78 3\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47314 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t772\t0.0\t1\t0 772\n",
      "19\t46479\t0.0\t1\t45044 1435\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S473_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S473_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S473_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S473_205_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S473_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S473_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S473_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S473_205_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,927 reads  @     63.2 µs/read;   0.95 M reads/minute\n",
      "Finished in 3.29 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,927\n",
      "  Read 1 with adapter:                  51,863 (99.9%)\n",
      "  Read 2 with adapter:                  51,706 (99.6%)\n",
      "Pairs written (passing filters):        51,927 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,634,460 bp\n",
      "  Read 1:    15,316,213 bp\n",
      "  Read 2:    15,318,247 bp\n",
      "Total written (filtered):     28,616,257 bp (93.4%)\n",
      "  Read 1:    14,279,714 bp\n",
      "  Read 2:    14,336,543 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51863 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t850\t0.0\t1\t0 813 37\n",
      "20\t50842\t0.0\t2\t48310 2465 67\n",
      "21\t97\t0.0\t2\t0 96 1\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51706 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t780\t0.0\t1\t0 780\n",
      "19\t50856\t0.0\t1\t49447 1409\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S474_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S474_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S474_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S474_207_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S474_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S474_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S474_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S474_207_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        70,404 reads  @     64.3 µs/read;   0.93 M reads/minute\n",
      "Finished in 4.53 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             70,404\n",
      "  Read 1 with adapter:                  69,493 (98.7%)\n",
      "  Read 2 with adapter:                  70,061 (99.5%)\n",
      "Pairs written (passing filters):        70,404 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,409,967 bp\n",
      "  Read 1:    20,646,407 bp\n",
      "  Read 2:    20,763,560 bp\n",
      "Total written (filtered):     38,690,741 bp (93.4%)\n",
      "  Read 1:    19,257,610 bp\n",
      "  Read 2:    19,433,131 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 69493 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t62\t0.0\t1\t0 0 62\n",
      "19\t1142\t0.0\t1\t0 1079 63\n",
      "20\t68137\t0.0\t2\t64543 3498 96\n",
      "21\t101\t0.0\t2\t0 99 2\n",
      "22\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 70061 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t807\t0.0\t1\t0 807\n",
      "19\t69177\t0.0\t1\t67032 2145\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S475_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S475_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S475_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S475_209_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S475_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S475_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S475_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S475_209_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:03        60,916 reads  @     62.2 µs/read;   0.96 M reads/minute\n",
      "Finished in 3.80 s (62 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,916\n",
      "  Read 1 with adapter:                  60,866 (99.9%)\n",
      "  Read 2 with adapter:                  60,664 (99.6%)\n",
      "Pairs written (passing filters):        60,916 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,940,343 bp\n",
      "  Read 1:    17,970,194 bp\n",
      "  Read 2:    17,970,149 bp\n",
      "Total written (filtered):     33,571,551 bp (93.4%)\n",
      "  Read 1:    16,753,132 bp\n",
      "  Read 2:    16,818,419 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60866 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t415\t0.0\t1\t0 397 18\n",
      "20\t60291\t0.0\t2\t57245 2979 67\n",
      "21\t103\t0.0\t2\t0 103\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 60664 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1017\t0.0\t1\t0 1017\n",
      "19\t59516\t0.0\t1\t57987 1529\n",
      "20\t131\t0.0\t1\t0 131\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S476_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S476_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S476_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S476_211_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S476_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S476_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S476_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S476_211_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        60,780 reads  @     59.4 µs/read;   1.01 M reads/minute\n",
      "Finished in 3.61 s (59 us/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,780\n",
      "  Read 1 with adapter:                  60,740 (99.9%)\n",
      "  Read 2 with adapter:                  60,508 (99.6%)\n",
      "Pairs written (passing filters):        60,779 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,847,690 bp\n",
      "  Read 1:    17,929,980 bp\n",
      "  Read 2:    17,917,710 bp\n",
      "Total written (filtered):     33,484,484 bp (93.4%)\n",
      "  Read 1:    16,715,557 bp\n",
      "  Read 2:    16,768,927 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60740 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t750\t0.0\t1\t0 720 30\n",
      "20\t59820\t0.0\t2\t56736 2989 95\n",
      "21\t118\t0.0\t2\t0 115 3\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 60508 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t971\t0.0\t1\t0 971\n",
      "19\t59435\t0.0\t1\t58146 1289\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S477_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S477_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S477_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S477_213_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S477_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S477_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S477_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S477_213_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,614 reads  @     61.0 µs/read;   0.98 M reads/minute\n",
      "Finished in 3.09 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,614\n",
      "  Read 1 with adapter:                  50,573 (99.9%)\n",
      "  Read 2 with adapter:                  50,402 (99.6%)\n",
      "Pairs written (passing filters):        50,614 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,861,997 bp\n",
      "  Read 1:    14,931,012 bp\n",
      "  Read 2:    14,930,985 bp\n",
      "Total written (filtered):     27,894,085 bp (93.4%)\n",
      "  Read 1:    13,920,147 bp\n",
      "  Read 2:    13,973,938 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50573 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t711\t0.0\t1\t0 669 42\n",
      "20\t49704\t0.0\t2\t47069 2564 71\n",
      "21\t96\t0.0\t2\t0 91 5\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50402 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t658\t0.0\t1\t0 658\n",
      "19\t49677\t0.0\t1\t48507 1170\n",
      "20\t67\t0.0\t1\t0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S478_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S478_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S478_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S478_215_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S478_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S478_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S478_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S478_215_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        49,249 reads  @     60.6 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.99 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,249\n",
      "  Read 1 with adapter:                  49,197 (99.9%)\n",
      "  Read 2 with adapter:                  49,105 (99.7%)\n",
      "Pairs written (passing filters):        49,249 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,055,528 bp\n",
      "  Read 1:    14,528,424 bp\n",
      "  Read 2:    14,527,104 bp\n",
      "Total written (filtered):     27,139,660 bp (93.4%)\n",
      "  Read 1:    13,545,072 bp\n",
      "  Read 2:    13,594,588 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49197 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t688\t0.0\t1\t0 649 39\n",
      "20\t48377\t0.0\t2\t45886 2434 57\n",
      "21\t84\t0.0\t2\t0 81 3\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49105 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t542\t0.0\t1\t0 542\n",
      "19\t48500\t0.0\t1\t47410 1090\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S479_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S479_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S479_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S479_217_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S479_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S479_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S479_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S479_217_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        45,189 reads  @     62.4 µs/read;   0.96 M reads/minute\n",
      "Finished in 2.82 s (62 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,189\n",
      "  Read 1 with adapter:                  45,140 (99.9%)\n",
      "  Read 2 with adapter:                  44,948 (99.5%)\n",
      "Pairs written (passing filters):        45,189 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,642,227 bp\n",
      "  Read 1:    13,329,570 bp\n",
      "  Read 2:    13,312,657 bp\n",
      "Total written (filtered):     24,886,282 bp (93.4%)\n",
      "  Read 1:    12,427,005 bp\n",
      "  Read 2:    12,459,277 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45140 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t306\t0.0\t1\t0 289 17\n",
      "20\t44728\t0.0\t2\t42471 2200 57\n",
      "21\t65\t0.0\t2\t0 64 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44948 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t699\t0.0\t1\t0 699\n",
      "19\t44182\t0.0\t1\t43257 925\n",
      "20\t67\t0.0\t1\t0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S480_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S480_219_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S480_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S480_219_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S480_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S480_219_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S480_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S480_219_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,736 reads  @     59.6 µs/read;   1.01 M reads/minute\n",
      "Finished in 3.02 s (60 us/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,736\n",
      "  Read 1 with adapter:                  50,694 (99.9%)\n",
      "  Read 2 with adapter:                  50,496 (99.5%)\n",
      "Pairs written (passing filters):        50,736 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,918,398 bp\n",
      "  Read 1:    14,966,843 bp\n",
      "  Read 2:    14,951,555 bp\n",
      "Total written (filtered):     27,946,408 bp (93.4%)\n",
      "  Read 1:    13,953,583 bp\n",
      "  Read 2:    13,992,825 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50694 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t36\t0.0\t1\t0 0 36\n",
      "19\t697\t0.0\t1\t0 665 32\n",
      "20\t49844\t0.0\t2\t47278 2508 58\n",
      "21\t85\t0.0\t2\t0 84 1\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50496 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t756\t0.0\t1\t0 756\n",
      "19\t49678\t0.0\t1\t48615 1063\n",
      "20\t62\t0.0\t1\t0 62\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S481_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S481_221_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S481_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S481_221_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S481_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S481_221_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S481_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S481_221_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        51,030 reads  @     62.1 µs/read;   0.97 M reads/minute\n",
      "Finished in 3.17 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,030\n",
      "  Read 1 with adapter:                  50,986 (99.9%)\n",
      "  Read 2 with adapter:                  50,831 (99.6%)\n",
      "Pairs written (passing filters):        51,030 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,107,380 bp\n",
      "  Read 1:    15,053,717 bp\n",
      "  Read 2:    15,053,663 bp\n",
      "Total written (filtered):     28,123,297 bp (93.4%)\n",
      "  Read 1:    14,034,740 bp\n",
      "  Read 2:    14,088,557 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50986 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t35\t0.0\t1\t0 0 35\n",
      "19\t785\t0.0\t1\t0 759 26\n",
      "20\t50076\t0.0\t2\t47409 2589 78\n",
      "21\t68\t0.0\t2\t0 67 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50831 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t758\t0.0\t1\t0 758\n",
      "19\t49998\t0.0\t1\t48811 1187\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S482_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S482_223_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S482_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S482_223_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S482_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S482_223_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S482_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S482_223_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,001 reads  @     57.8 µs/read;   1.04 M reads/minute\n",
      "Finished in 2.49 s (58 us/read; 1.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,001\n",
      "  Read 1 with adapter:                  42,970 (99.9%)\n",
      "  Read 2 with adapter:                  42,860 (99.7%)\n",
      "Pairs written (passing filters):        43,001 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,367,936 bp\n",
      "  Read 1:    12,684,459 bp\n",
      "  Read 2:    12,683,477 bp\n",
      "Total written (filtered):     23,695,242 bp (93.4%)\n",
      "  Read 1:    11,825,666 bp\n",
      "  Read 2:    11,869,576 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42970 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t32\t0.0\t1\t0 0 32\n",
      "19\t658\t0.0\t1\t0 634 24\n",
      "20\t42186\t0.0\t2\t40009 2118 59\n",
      "21\t73\t0.0\t2\t0 72 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42860 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t497\t0.0\t1\t0 497\n",
      "19\t42305\t0.0\t1\t41320 985\n",
      "20\t58\t0.0\t1\t0 58\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S483_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S483_225_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S483_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S483_225_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S483_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S483_225_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S483_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S483_225_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        67,979 reads  @     58.3 µs/read;   1.03 M reads/minute\n",
      "Finished in 3.97 s (58 us/read; 1.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,979\n",
      "  Read 1 with adapter:                  67,922 (99.9%)\n",
      "  Read 2 with adapter:                  67,676 (99.6%)\n",
      "Pairs written (passing filters):        67,977 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,104,489 bp\n",
      "  Read 1:    20,052,875 bp\n",
      "  Read 2:    20,051,614 bp\n",
      "Total written (filtered):     37,460,979 bp (93.4%)\n",
      "  Read 1:    18,694,199 bp\n",
      "  Read 2:    18,766,780 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 67922 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t481\t0.0\t1\t0 459 22\n",
      "20\t67256\t0.0\t2\t63954 3214 88\n",
      "21\t119\t0.0\t2\t0 119\n",
      "22\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 67676 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1131\t0.0\t1\t0 1131\n",
      "19\t66424\t0.0\t1\t64678 1746\n",
      "20\t121\t0.0\t1\t0 121\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S484_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S484_227_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S484_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S484_227_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S484_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S484_227_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S484_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S484_227_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00        15,565 reads  @     60.2 µs/read;   1.00 M reads/minute\n",
      "Finished in 0.94 s (60 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             15,565\n",
      "  Read 1 with adapter:                  15,549 (99.9%)\n",
      "  Read 2 with adapter:                  15,469 (99.4%)\n",
      "Pairs written (passing filters):        15,565 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,176,152 bp\n",
      "  Read 1:     4,591,097 bp\n",
      "  Read 2:     4,585,055 bp\n",
      "Total written (filtered):      8,571,640 bp (93.4%)\n",
      "  Read 1:     4,280,253 bp\n",
      "  Read 2:     4,291,387 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 15549 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t149\t0.0\t1\t0 143 6\n",
      "20\t15360\t0.0\t2\t14468 868 24\n",
      "21\t23\t0.0\t2\t0 23\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 15469 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t270\t0.0\t1\t0 270\n",
      "19\t15172\t0.0\t1\t14830 342\n",
      "20\t27\t0.0\t1\t0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S485_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S485_229_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S485_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S485_229_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S485_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S485_229_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S485_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S485_229_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        56,760 reads  @     60.2 µs/read;   1.00 M reads/minute\n",
      "Finished in 3.42 s (60 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,760\n",
      "  Read 1 with adapter:                  56,693 (99.9%)\n",
      "  Read 2 with adapter:                  56,476 (99.5%)\n",
      "Pairs written (passing filters):        56,760 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,484,588 bp\n",
      "  Read 1:    16,740,898 bp\n",
      "  Read 2:    16,743,690 bp\n",
      "Total written (filtered):     31,278,862 bp (93.4%)\n",
      "  Read 1:    15,607,545 bp\n",
      "  Read 2:    15,671,317 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56693 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t581\t0.0\t1\t0 550 31\n",
      "20\t55984\t0.0\t2\t52955 2947 82\n",
      "21\t70\t0.0\t2\t0 68 2\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56476 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t759\t0.0\t1\t0 759\n",
      "19\t55629\t0.0\t1\t54204 1425\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S486_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S486_231_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S486_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S486_231_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S486_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S486_231_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S486_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S486_231_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        74,811 reads  @     63.9 µs/read;   0.94 M reads/minute\n",
      "Finished in 4.79 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,811\n",
      "  Read 1 with adapter:                  74,442 (99.5%)\n",
      "  Read 2 with adapter:                  74,205 (99.2%)\n",
      "Pairs written (passing filters):        74,811 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,135,376 bp\n",
      "  Read 1:    22,067,800 bp\n",
      "  Read 2:    22,067,576 bp\n",
      "Total written (filtered):     41,238,078 bp (93.4%)\n",
      "  Read 1:    20,579,553 bp\n",
      "  Read 2:    20,658,525 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 74442 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t815\t0.0\t1\t0 768 47\n",
      "20\t73390\t0.0\t2\t68935 4335 120\n",
      "21\t96\t0.0\t2\t0 93 3\n",
      "22\t102\t0.0\t2\t0 0 102\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 74205 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t927\t0.0\t1\t0 927\n",
      "19\t73195\t0.0\t1\t70828 2367\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S487_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S487_233_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S487_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S487_233_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S487_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S487_233_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S487_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S487_233_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        55,242 reads  @     62.1 µs/read;   0.97 M reads/minute\n",
      "Finished in 3.43 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,242\n",
      "  Read 1 with adapter:                  55,176 (99.9%)\n",
      "  Read 2 with adapter:                  54,954 (99.5%)\n",
      "Pairs written (passing filters):        55,242 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,586,688 bp\n",
      "  Read 1:    16,290,917 bp\n",
      "  Read 2:    16,295,771 bp\n",
      "Total written (filtered):     30,439,964 bp (93.4%)\n",
      "  Read 1:    15,187,607 bp\n",
      "  Read 2:    15,252,357 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55176 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t355\t0.0\t1\t0 339 16\n",
      "20\t54651\t0.0\t2\t51875 2709 67\n",
      "21\t107\t0.0\t2\t0 107\n",
      "22\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54954 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t791\t0.0\t1\t0 791\n",
      "19\t54084\t0.0\t1\t52712 1372\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S488_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S488_235_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S488_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S488_235_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S488_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S488_235_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S488_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S488_235_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        66,067 reads  @     62.5 µs/read;   0.96 M reads/minute\n",
      "Finished in 4.13 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,067\n",
      "  Read 1 with adapter:                  66,016 (99.9%)\n",
      "  Read 2 with adapter:                  65,602 (99.3%)\n",
      "Pairs written (passing filters):        66,066 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,956,896 bp\n",
      "  Read 1:    19,489,462 bp\n",
      "  Read 2:    19,467,434 bp\n",
      "Total written (filtered):     36,391,197 bp (93.4%)\n",
      "  Read 1:    18,169,260 bp\n",
      "  Read 2:    18,221,937 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 66016 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t33\t0.0\t1\t0 0 33\n",
      "19\t579\t0.0\t1\t0 542 37\n",
      "20\t65214\t0.0\t2\t62059 3078 77\n",
      "21\t128\t0.0\t2\t0 125 3\n",
      "22\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 65602 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1047\t0.0\t1\t0 1047\n",
      "19\t64449\t0.0\t1\t62839 1610\n",
      "20\t106\t0.0\t1\t0 106\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S489_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S489_237_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S489_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S489_237_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S489_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9pmssg_d/S489_237_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S489_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uujmq_o/693ccc21-bdcc-4276-b0c3-c36aba76502c/data/S489_237_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00        14,325 reads  @     65.7 µs/read;   0.91 M reads/minute\n",
      "Finished in 0.94 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             14,325\n",
      "  Read 1 with adapter:                  14,310 (99.9%)\n",
      "  Read 2 with adapter:                  14,268 (99.6%)\n",
      "Pairs written (passing filters):        14,325 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,451,705 bp\n",
      "  Read 1:     4,225,872 bp\n",
      "  Read 2:     4,225,833 bp\n",
      "Total written (filtered):      7,894,703 bp (93.4%)\n",
      "  Read 1:     3,939,729 bp\n",
      "  Read 2:     3,954,974 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 14310 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t2\t0.0\t1\t0 0 2\n",
      "19\t121\t0.0\t1\t0 116 5\n",
      "20\t14137\t0.0\t2\t13497 628 12\n",
      "21\t32\t0.0\t2\t0 31 1\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 14268 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t250\t0.0\t1\t0 250\n",
      "19\t14001\t0.0\t1\t13660 341\n",
      "20\t17\t0.0\t1\t0 17\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: 18S-demux-pairedend-trimmed2.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s501_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s501_136_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s501_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s501_136_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s501_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s501_136_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s501_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s501_136_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,154 reads  @     70.7 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.42 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,154\n",
      "  Read 1 with adapter:                  34,142 (100.0%)\n",
      "  Read 2 with adapter:                  34,086 (99.8%)\n",
      "Pairs written (passing filters):        34,154 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,150,240 bp\n",
      "  Read 1:    10,075,422 bp\n",
      "  Read 2:    10,074,818 bp\n",
      "Total written (filtered):     18,819,993 bp (93.4%)\n",
      "  Read 1:     9,392,678 bp\n",
      "  Read 2:     9,427,315 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34142 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t173\t0.0\t1\t0 167 6\n",
      "20\t33887\t0.0\t2\t33337 542 8\n",
      "21\t51\t0.0\t2\t0 51\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34086 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t182\t0.0\t1\t0 182\n",
      "19\t33853\t0.0\t1\t33425 428\n",
      "20\t51\t0.0\t1\t0 51\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s502_1_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s502_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s502_1_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s502_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s502_1_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s502_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s502_1_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s502_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        44,465 reads  @     66.0 µs/read;   0.91 M reads/minute\n",
      "Finished in 2.94 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,465\n",
      "  Read 1 with adapter:                  44,442 (99.9%)\n",
      "  Read 2 with adapter:                  44,313 (99.7%)\n",
      "Pairs written (passing filters):        44,465 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,233,081 bp\n",
      "  Read 1:    13,116,271 bp\n",
      "  Read 2:    13,116,810 bp\n",
      "Total written (filtered):     24,502,610 bp (93.4%)\n",
      "  Read 1:    12,227,610 bp\n",
      "  Read 2:    12,275,000 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44442 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t277\t0.0\t1\t0 270 7\n",
      "20\t44037\t0.0\t2\t43339 689 9\n",
      "21\t94\t0.0\t2\t0 93 1\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44313 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t218\t0.0\t1\t0 218\n",
      "19\t44014\t0.0\t1\t43089 925\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s503_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s503_138_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s503_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s503_138_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s503_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s503_138_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s503_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s503_138_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        33,415 reads  @     59.8 µs/read;   1.00 M reads/minute\n",
      "Finished in 2.00 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,415\n",
      "  Read 1 with adapter:                  33,399 (100.0%)\n",
      "  Read 2 with adapter:                  33,295 (99.6%)\n",
      "Pairs written (passing filters):        33,415 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,714,196 bp\n",
      "  Read 1:     9,857,177 bp\n",
      "  Read 2:     9,857,019 bp\n",
      "Total written (filtered):     18,413,824 bp (93.4%)\n",
      "  Read 1:     9,189,318 bp\n",
      "  Read 2:     9,224,506 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33399 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t188\t0.0\t1\t0 184 4\n",
      "20\t33125\t0.0\t2\t32660 462 3\n",
      "21\t49\t0.0\t2\t0 49\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33295 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t160\t0.0\t1\t0 160\n",
      "19\t33067\t0.0\t1\t32478 589\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s504_3_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s504_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s504_3_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s504_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s504_3_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s504_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s504_3_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s504_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,418 reads  @     64.9 µs/read;   0.92 M reads/minute\n",
      "Finished in 2.30 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,418\n",
      "  Read 1 with adapter:                  35,407 (100.0%)\n",
      "  Read 2 with adapter:                  35,183 (99.3%)\n",
      "Pairs written (passing filters):        35,418 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,896,348 bp\n",
      "  Read 1:    10,448,302 bp\n",
      "  Read 2:    10,448,046 bp\n",
      "Total written (filtered):     19,519,971 bp (93.4%)\n",
      "  Read 1:     9,740,275 bp\n",
      "  Read 2:     9,779,696 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35407 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t199\t0.0\t1\t0 195 4\n",
      "20\t35121\t0.0\t2\t34608 507 6\n",
      "21\t60\t0.0\t2\t0 57 3\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35183 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t210\t0.0\t1\t0 210\n",
      "19\t34890\t0.0\t1\t33865 1025\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s506_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s506_140_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s506_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s506_140_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s506_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s506_140_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s506_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s506_140_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        44,587 reads  @     60.4 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.70 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,587\n",
      "  Read 1 with adapter:                  44,576 (100.0%)\n",
      "  Read 2 with adapter:                  44,473 (99.7%)\n",
      "Pairs written (passing filters):        44,587 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,302,598 bp\n",
      "  Read 1:    13,152,825 bp\n",
      "  Read 2:    13,149,773 bp\n",
      "Total written (filtered):     24,566,321 bp (93.4%)\n",
      "  Read 1:    12,261,444 bp\n",
      "  Read 2:    12,304,877 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44576 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t264\t0.0\t1\t0 262 2\n",
      "20\t44175\t0.0\t2\t43437 735 3\n",
      "21\t85\t0.0\t2\t0 85\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44473 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t183\t0.0\t1\t0 183\n",
      "19\t44198\t0.0\t1\t43419 779\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s507_5_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s507_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s507_5_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s507_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s507_5_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s507_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s507_5_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s507_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        40,773 reads  @     60.1 µs/read;   1.00 M reads/minute\n",
      "Finished in 2.45 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,773\n",
      "  Read 1 with adapter:                  40,755 (100.0%)\n",
      "  Read 2 with adapter:                  40,648 (99.7%)\n",
      "Pairs written (passing filters):        40,773 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,051,910 bp\n",
      "  Read 1:    12,027,692 bp\n",
      "  Read 2:    12,024,218 bp\n",
      "Total written (filtered):     22,464,746 bp (93.4%)\n",
      "  Read 1:    11,212,749 bp\n",
      "  Read 2:    11,251,997 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40755 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t216\t0.0\t1\t0 214 2\n",
      "20\t40433\t0.0\t2\t39859 570 4\n",
      "21\t73\t0.0\t2\t0 73\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40648 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t174\t0.0\t1\t0 174\n",
      "19\t40391\t0.0\t1\t39748 643\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s508_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s508_142_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s508_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s508_142_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s508_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s508_142_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s508_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s508_142_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,196 reads  @     67.2 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.50 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,196\n",
      "  Read 1 with adapter:                  37,184 (100.0%)\n",
      "  Read 2 with adapter:                  36,748 (98.8%)\n",
      "Pairs written (passing filters):        37,196 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,922,151 bp\n",
      "  Read 1:    10,972,813 bp\n",
      "  Read 2:    10,949,338 bp\n",
      "Total written (filtered):     20,480,495 bp (93.4%)\n",
      "  Read 1:    10,229,263 bp\n",
      "  Read 2:    10,251,232 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37184 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t229\t0.0\t1\t0 227 2\n",
      "20\t36847\t0.0\t2\t36200 636 11\n",
      "21\t77\t0.0\t2\t0 76 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36748 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t172\t0.0\t1\t0 172\n",
      "19\t36510\t0.0\t1\t35929 581\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s509_7_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s509_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s509_7_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s509_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s509_7_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s509_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s509_7_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s509_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        26,553 reads  @     69.8 µs/read;   0.86 M reads/minute\n",
      "Finished in 1.86 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,553\n",
      "  Read 1 with adapter:                  26,546 (100.0%)\n",
      "  Read 2 with adapter:                  26,509 (99.8%)\n",
      "Pairs written (passing filters):        26,553 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,665,865 bp\n",
      "  Read 1:     7,833,129 bp\n",
      "  Read 2:     7,832,736 bp\n",
      "Total written (filtered):     14,631,449 bp (93.4%)\n",
      "  Read 1:     7,302,306 bp\n",
      "  Read 2:     7,329,143 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 26546 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t166\t0.0\t1\t0 164 2\n",
      "20\t26306\t0.0\t2\t25833 469 4\n",
      "21\t39\t0.0\t2\t0 38 1\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 26509 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t128\t0.0\t1\t0 128\n",
      "19\t26331\t0.0\t1\t25953 378\n",
      "20\t50\t0.0\t1\t0 50\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s510_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s510_144_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s510_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s510_144_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s510_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s510_144_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s510_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s510_144_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        32,361 reads  @     59.1 µs/read;   1.01 M reads/minute\n",
      "Finished in 1.92 s (59 us/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,361\n",
      "  Read 1 with adapter:                  32,349 (100.0%)\n",
      "  Read 2 with adapter:                  32,280 (99.7%)\n",
      "Pairs written (passing filters):        32,361 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,092,532 bp\n",
      "  Read 1:     9,546,491 bp\n",
      "  Read 2:     9,546,041 bp\n",
      "Total written (filtered):     17,832,433 bp (93.4%)\n",
      "  Read 1:     8,899,598 bp\n",
      "  Read 2:     8,932,835 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32349 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t174\t0.0\t1\t0 170 4\n",
      "20\t32090\t0.0\t2\t31608 476 6\n",
      "21\t51\t0.0\t2\t0 51\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32280 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t172\t0.0\t1\t0 172\n",
      "19\t32050\t0.0\t1\t31472 578\n",
      "20\t58\t0.0\t1\t0 58\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s511_9_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s511_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s511_9_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s511_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s511_9_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s511_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s511_9_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s511_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        26,380 reads  @     61.7 µs/read;   0.97 M reads/minute\n",
      "Finished in 1.63 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,380\n",
      "  Read 1 with adapter:                  26,374 (100.0%)\n",
      "  Read 2 with adapter:                  26,309 (99.7%)\n",
      "Pairs written (passing filters):        26,380 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,563,558 bp\n",
      "  Read 1:     7,782,090 bp\n",
      "  Read 2:     7,781,468 bp\n",
      "Total written (filtered):     14,536,372 bp (93.4%)\n",
      "  Read 1:     7,254,700 bp\n",
      "  Read 2:     7,281,672 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 26374 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t166\t0.0\t1\t0 162 4\n",
      "20\t26137\t0.0\t2\t25732 401 4\n",
      "21\t46\t0.0\t2\t0 46\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 26309 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t122\t0.0\t1\t0 122\n",
      "19\t26140\t0.0\t1\t25702 438\n",
      "20\t47\t0.0\t1\t0 47\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s512_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s512_146_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s512_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s512_146_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s512_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s512_146_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s512_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s512_146_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        30,575 reads  @     63.9 µs/read;   0.94 M reads/minute\n",
      "Finished in 1.96 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,575\n",
      "  Read 1 with adapter:                  30,568 (100.0%)\n",
      "  Read 2 with adapter:                  30,395 (99.4%)\n",
      "Pairs written (passing filters):        30,575 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,038,528 bp\n",
      "  Read 1:     9,019,616 bp\n",
      "  Read 2:     9,018,912 bp\n",
      "Total written (filtered):     16,849,909 bp (93.4%)\n",
      "  Read 1:     8,408,380 bp\n",
      "  Read 2:     8,441,529 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30568 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t198\t0.0\t1\t0 198\n",
      "20\t30277\t0.0\t2\t29834 440 3\n",
      "21\t52\t0.0\t2\t0 50 2\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30395 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t192\t0.0\t1\t0 192\n",
      "19\t30133\t0.0\t1\t29360 773\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s514_11_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s514_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s514_11_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s514_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s514_11_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s514_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s514_11_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s514_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        31,975 reads  @     60.8 µs/read;   0.99 M reads/minute\n",
      "Finished in 1.95 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,975\n",
      "  Read 1 with adapter:                  31,960 (100.0%)\n",
      "  Read 2 with adapter:                  31,894 (99.7%)\n",
      "Pairs written (passing filters):        31,975 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,862,555 bp\n",
      "  Read 1:     9,432,608 bp\n",
      "  Read 2:     9,429,947 bp\n",
      "Total written (filtered):     17,617,527 bp (93.4%)\n",
      "  Read 1:     8,793,484 bp\n",
      "  Read 2:     8,824,043 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31960 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t181\t0.0\t1\t0 178 3\n",
      "20\t31675\t0.0\t2\t31195 474 6\n",
      "21\t63\t0.0\t2\t0 61 2\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31894 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t137\t0.0\t1\t0 137\n",
      "19\t31702\t0.0\t1\t31209 493\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s515_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s515_148_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s515_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s515_148_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s515_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s515_148_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s515_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s515_148_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,598 reads  @     60.7 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.04 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,598\n",
      "  Read 1 with adapter:                  33,587 (100.0%)\n",
      "  Read 2 with adapter:                  33,508 (99.7%)\n",
      "Pairs written (passing filters):        33,598 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,820,347 bp\n",
      "  Read 1:     9,911,110 bp\n",
      "  Read 2:     9,909,237 bp\n",
      "Total written (filtered):     18,512,167 bp (93.4%)\n",
      "  Read 1:     9,239,513 bp\n",
      "  Read 2:     9,272,654 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33587 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t212\t0.0\t1\t0 209 3\n",
      "20\t33304\t0.0\t2\t32842 453 9\n",
      "21\t45\t0.0\t2\t0 44 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33508 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t137\t0.0\t1\t0 137\n",
      "19\t33303\t0.0\t1\t32796 507\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s516_13_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s516_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s516_13_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s516_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s516_13_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s516_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s516_13_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s516_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        36,616 reads  @     60.5 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.22 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,616\n",
      "  Read 1 with adapter:                  36,602 (100.0%)\n",
      "  Read 2 with adapter:                  36,205 (98.9%)\n",
      "Pairs written (passing filters):        36,616 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,581,137 bp\n",
      "  Read 1:    10,801,712 bp\n",
      "  Read 2:    10,779,425 bp\n",
      "Total written (filtered):     20,161,392 bp (93.4%)\n",
      "  Read 1:    10,069,752 bp\n",
      "  Read 2:    10,091,640 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36602 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t194\t0.0\t1\t0 193 1\n",
      "20\t36299\t0.0\t2\t35746 549 4\n",
      "21\t60\t0.0\t2\t0 59 1\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36205 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t173\t0.0\t1\t0 173\n",
      "19\t35969\t0.0\t1\t35425 544\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s517_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s517_150_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s517_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s517_150_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s517_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s517_150_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s517_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s517_150_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        40,298 reads  @     62.8 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.54 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,298\n",
      "  Read 1 with adapter:                  40,278 (100.0%)\n",
      "  Read 2 with adapter:                  40,221 (99.8%)\n",
      "Pairs written (passing filters):        40,298 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,775,788 bp\n",
      "  Read 1:    11,887,894 bp\n",
      "  Read 2:    11,887,894 bp\n",
      "Total written (filtered):     22,206,230 bp (93.4%)\n",
      "  Read 1:    11,082,401 bp\n",
      "  Read 2:    11,123,829 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40278 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t192\t0.0\t1\t0 184 8\n",
      "20\t39961\t0.0\t2\t39358 599 4\n",
      "21\t89\t0.0\t2\t0 84 5\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40221 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t209\t0.0\t1\t0 209\n",
      "19\t39937\t0.0\t1\t39320 617\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s518_15_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s518_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s518_15_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s518_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s518_15_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s518_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s518_15_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s518_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,636 reads  @     59.3 µs/read;   1.01 M reads/minute\n",
      "Finished in 2.35 s (59 us/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,636\n",
      "  Read 1 with adapter:                  39,626 (100.0%)\n",
      "  Read 2 with adapter:                  39,541 (99.8%)\n",
      "Pairs written (passing filters):        39,636 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,384,904 bp\n",
      "  Read 1:    11,692,612 bp\n",
      "  Read 2:    11,692,292 bp\n",
      "Total written (filtered):     21,841,328 bp (93.4%)\n",
      "  Read 1:    10,900,176 bp\n",
      "  Read 2:    10,941,152 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t203\t0.0\t1\t0 201 2\n",
      "20\t39291\t0.0\t2\t38715 572 4\n",
      "21\t77\t0.0\t2\t0 76 1\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39541 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t200\t0.0\t1\t0 200\n",
      "19\t39280\t0.0\t1\t38538 742\n",
      "20\t61\t0.0\t1\t0 61\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s519_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s519_152_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s519_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s519_152_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s519_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s519_152_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s519_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s519_152_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        33,023 reads  @     59.8 µs/read;   1.00 M reads/minute\n",
      "Finished in 1.98 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,023\n",
      "  Read 1 with adapter:                  33,014 (100.0%)\n",
      "  Read 2 with adapter:                  32,929 (99.7%)\n",
      "Pairs written (passing filters):        33,023 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,483,206 bp\n",
      "  Read 1:     9,741,750 bp\n",
      "  Read 2:     9,741,456 bp\n",
      "Total written (filtered):     18,197,474 bp (93.4%)\n",
      "  Read 1:     9,081,577 bp\n",
      "  Read 2:     9,115,897 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33014 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t175\t0.0\t1\t0 172 3\n",
      "20\t32750\t0.0\t2\t32239 508 3\n",
      "21\t58\t0.0\t2\t0 55 3\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32929 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t160\t0.0\t1\t0 160\n",
      "19\t32701\t0.0\t1\t32156 545\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s520_17_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s520_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s520_17_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s520_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s520_17_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s520_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s520_17_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s520_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,583 reads  @     64.4 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.10 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,583\n",
      "  Read 1 with adapter:                  32,571 (100.0%)\n",
      "  Read 2 with adapter:                  32,309 (99.2%)\n",
      "Pairs written (passing filters):        32,583 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,223,806 bp\n",
      "  Read 1:     9,611,981 bp\n",
      "  Read 2:     9,611,825 bp\n",
      "Total written (filtered):     17,958,721 bp (93.4%)\n",
      "  Read 1:     8,960,647 bp\n",
      "  Read 2:     8,998,074 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32571 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t164\t0.0\t1\t0 161 3\n",
      "20\t32323\t0.0\t2\t31810 509 4\n",
      "21\t54\t0.0\t2\t0 52 2\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32309 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t190\t0.0\t1\t0 190\n",
      "19\t32049\t0.0\t1\t31178 871\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s522_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s522_154_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s522_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s522_154_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s522_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s522_154_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s522_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s522_154_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        42,642 reads  @     60.3 µs/read;   1.00 M reads/minute\n",
      "Finished in 2.57 s (60 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,642\n",
      "  Read 1 with adapter:                  42,626 (100.0%)\n",
      "  Read 2 with adapter:                  42,510 (99.7%)\n",
      "Pairs written (passing filters):        42,642 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,154,142 bp\n",
      "  Read 1:    12,579,368 bp\n",
      "  Read 2:    12,574,774 bp\n",
      "Total written (filtered):     23,494,199 bp (93.4%)\n",
      "  Read 1:    11,727,006 bp\n",
      "  Read 2:    11,767,193 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t234\t0.0\t1\t0 227 7\n",
      "20\t42277\t0.0\t2\t41596 671 10\n",
      "21\t62\t0.0\t2\t0 60 2\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42510 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t195\t0.0\t1\t0 195\n",
      "19\t42229\t0.0\t1\t41468 761\n",
      "20\t86\t0.0\t1\t0 86\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s523_19_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s523_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s523_19_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s523_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s523_19_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s523_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s523_19_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s523_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        44,391 reads  @     66.3 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.95 s (66 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,391\n",
      "  Read 1 with adapter:                  44,368 (99.9%)\n",
      "  Read 2 with adapter:                  44,236 (99.7%)\n",
      "Pairs written (passing filters):        44,391 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,184,543 bp\n",
      "  Read 1:    13,095,233 bp\n",
      "  Read 2:    13,089,310 bp\n",
      "Total written (filtered):     24,456,916 bp (93.4%)\n",
      "  Read 1:    12,208,005 bp\n",
      "  Read 2:    12,248,911 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44368 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t249\t0.0\t1\t0 246 3\n",
      "20\t43990\t0.0\t2\t43301 682 7\n",
      "21\t93\t0.0\t2\t0 90 3\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44236 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t181\t0.0\t1\t0 181\n",
      "19\t43959\t0.0\t1\t43200 759\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s524_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s524_156_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s524_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s524_156_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s524_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s524_156_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s524_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s524_156_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,307 reads  @     68.1 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.02 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,307\n",
      "  Read 1 with adapter:                  44,290 (100.0%)\n",
      "  Read 2 with adapter:                  43,485 (98.1%)\n",
      "Pairs written (passing filters):        44,307 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,096,540 bp\n",
      "  Read 1:    13,070,555 bp\n",
      "  Read 2:    13,025,985 bp\n",
      "Total written (filtered):     24,384,777 bp (93.4%)\n",
      "  Read 1:    12,184,872 bp\n",
      "  Read 2:    12,199,905 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44290 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t242\t0.0\t1\t0 241 1\n",
      "20\t43917\t0.0\t2\t43161 748 8\n",
      "21\t93\t0.0\t2\t0 91 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43485 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t205\t0.0\t1\t0 205\n",
      "19\t43210\t0.0\t1\t42461 749\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s525_21_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s525_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s525_21_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s525_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s525_21_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s525_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s525_21_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s525_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,821 reads  @     75.8 µs/read;   0.79 M reads/minute\n",
      "Finished in 2.41 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,821\n",
      "  Read 1 with adapter:                  31,814 (100.0%)\n",
      "  Read 2 with adapter:                  31,763 (99.8%)\n",
      "Pairs written (passing filters):        31,821 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,772,154 bp\n",
      "  Read 1:     9,387,190 bp\n",
      "  Read 2:     9,384,964 bp\n",
      "Total written (filtered):     17,532,575 bp (93.4%)\n",
      "  Read 1:     8,751,021 bp\n",
      "  Read 2:     8,781,554 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31814 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t174\t0.0\t1\t0 170 4\n",
      "20\t31569\t0.0\t2\t30833 722 14\n",
      "21\t51\t0.0\t2\t0 50 1\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31763 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t151\t0.0\t1\t0 151\n",
      "19\t31548\t0.0\t1\t31173 375\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s526_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s526_158_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s526_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s526_158_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s526_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s526_158_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s526_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s526_158_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        44,776 reads  @     62.9 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.82 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,776\n",
      "  Read 1 with adapter:                  44,766 (100.0%)\n",
      "  Read 2 with adapter:                  44,681 (99.8%)\n",
      "Pairs written (passing filters):        44,776 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,417,757 bp\n",
      "  Read 1:    13,208,901 bp\n",
      "  Read 2:    13,208,856 bp\n",
      "Total written (filtered):     24,673,765 bp (93.4%)\n",
      "  Read 1:    12,313,774 bp\n",
      "  Read 2:    12,359,991 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44766 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t292\t0.0\t1\t0 289 3\n",
      "20\t44361\t0.0\t2\t43698 657 6\n",
      "21\t83\t0.0\t2\t0 82 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44681 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t176\t0.0\t1\t0 176\n",
      "19\t44403\t0.0\t1\t43724 679\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s527_23_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s527_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s527_23_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s527_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s527_23_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s527_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s527_23_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s527_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        28,192 reads  @     67.5 µs/read;   0.89 M reads/minute\n",
      "Finished in 1.91 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,192\n",
      "  Read 1 with adapter:                  28,186 (100.0%)\n",
      "  Read 2 with adapter:                  28,119 (99.7%)\n",
      "Pairs written (passing filters):        28,192 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,631,560 bp\n",
      "  Read 1:     8,316,637 bp\n",
      "  Read 2:     8,314,923 bp\n",
      "Total written (filtered):     15,533,745 bp (93.4%)\n",
      "  Read 1:     7,753,021 bp\n",
      "  Read 2:     7,780,724 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 28186 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t183\t0.0\t1\t0 181 2\n",
      "20\t27921\t0.0\t2\t27479 436 6\n",
      "21\t53\t0.0\t2\t0 53\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 28119 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t124\t0.0\t1\t0 124\n",
      "19\t27933\t0.0\t1\t27523 410\n",
      "20\t62\t0.0\t1\t0 62\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s528_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s528_160_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s528_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s528_160_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s528_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s528_160_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s528_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s528_160_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,058 reads  @     72.2 µs/read;   0.83 M reads/minute\n",
      "Finished in 2.25 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,058\n",
      "  Read 1 with adapter:                  31,047 (100.0%)\n",
      "  Read 2 with adapter:                  30,898 (99.5%)\n",
      "Pairs written (passing filters):        31,058 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,322,534 bp\n",
      "  Read 1:     9,162,084 bp\n",
      "  Read 2:     9,160,450 bp\n",
      "Total written (filtered):     17,114,823 bp (93.4%)\n",
      "  Read 1:     8,541,307 bp\n",
      "  Read 2:     8,573,516 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31047 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t215\t0.0\t1\t0 214 1\n",
      "20\t30762\t0.0\t2\t30285 472 5\n",
      "21\t48\t0.0\t2\t0 48\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30898 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t202\t0.0\t1\t0 202\n",
      "19\t30622\t0.0\t1\t29850 772\n",
      "20\t74\t0.0\t1\t0 74\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s530_25_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s530_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s530_25_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s530_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s530_25_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s530_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s530_25_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s530_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        44,271 reads  @     68.0 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.01 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,271\n",
      "  Read 1 with adapter:                  44,252 (100.0%)\n",
      "  Read 2 with adapter:                  44,135 (99.7%)\n",
      "Pairs written (passing filters):        44,271 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,115,281 bp\n",
      "  Read 1:    13,059,932 bp\n",
      "  Read 2:    13,055,349 bp\n",
      "Total written (filtered):     24,391,922 bp (93.4%)\n",
      "  Read 1:    12,175,024 bp\n",
      "  Read 2:    12,216,898 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44252 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t267\t0.0\t1\t0 264 3\n",
      "20\t43851\t0.0\t2\t43212 630 9\n",
      "21\t101\t0.0\t2\t0 99 2\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44135 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t191\t0.0\t1\t0 191\n",
      "19\t43867\t0.0\t1\t43167 700\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s531_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s531_162_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s531_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s531_162_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s531_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s531_162_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s531_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s531_162_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        28,288 reads  @     88.7 µs/read;   0.68 M reads/minute\n",
      "Finished in 2.51 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,288\n",
      "  Read 1 with adapter:                  28,276 (100.0%)\n",
      "  Read 2 with adapter:                  27,703 (97.9%)\n",
      "Pairs written (passing filters):        28,288 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,670,830 bp\n",
      "  Read 1:     8,344,462 bp\n",
      "  Read 2:     8,326,368 bp\n",
      "Total written (filtered):     15,579,145 bp (93.5%)\n",
      "  Read 1:     7,779,064 bp\n",
      "  Read 2:     7,800,081 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 28276 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t186\t0.0\t1\t0 184 2\n",
      "20\t28018\t0.0\t2\t27445 561 12\n",
      "21\t48\t0.0\t2\t0 48\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 27703 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t124\t0.0\t1\t0 124\n",
      "19\t27525\t0.0\t1\t25791 1734\n",
      "20\t54\t0.0\t1\t0 54\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s532_27_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s532_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s532_27_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s532_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s532_27_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s532_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s532_27_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s532_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,330 reads  @     68.5 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.18 s (69 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,330\n",
      "  Read 1 with adapter:                  46,314 (100.0%)\n",
      "  Read 2 with adapter:                  45,702 (98.6%)\n",
      "Pairs written (passing filters):        46,330 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,299,453 bp\n",
      "  Read 1:    13,667,344 bp\n",
      "  Read 2:    13,632,109 bp\n",
      "Total written (filtered):     25,505,210 bp (93.4%)\n",
      "  Read 1:    12,741,269 bp\n",
      "  Read 2:    12,763,941 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46314 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t294\t0.0\t1\t0 292 2\n",
      "20\t45904\t0.0\t2\t45103 793 8\n",
      "21\t71\t0.0\t2\t0 70 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45702 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t235\t0.0\t1\t0 235\n",
      "19\t45402\t0.0\t1\t44716 686\n",
      "20\t65\t0.0\t1\t0 65\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s533_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s533_164_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s533_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s533_164_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s533_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s533_164_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s533_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s533_164_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,821 reads  @     71.2 µs/read;   0.84 M reads/minute\n",
      "Finished in 2.63 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,821\n",
      "  Read 1 with adapter:                  36,807 (100.0%)\n",
      "  Read 2 with adapter:                  36,756 (99.8%)\n",
      "Pairs written (passing filters):        36,821 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,723,796 bp\n",
      "  Read 1:    10,862,175 bp\n",
      "  Read 2:    10,861,621 bp\n",
      "Total written (filtered):     20,289,574 bp (93.4%)\n",
      "  Read 1:    10,126,183 bp\n",
      "  Read 2:    10,163,391 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36807 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t235\t0.0\t1\t0 231 4\n",
      "20\t36485\t0.0\t2\t35839 635 11\n",
      "21\t55\t0.0\t2\t0 55\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36756 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t200\t0.0\t1\t0 200\n",
      "19\t36490\t0.0\t1\t36011 479\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s534_29_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s534_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s534_29_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s534_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s534_29_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s534_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s534_29_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s534_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        41,627 reads  @     66.7 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.78 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,627\n",
      "  Read 1 with adapter:                  41,620 (100.0%)\n",
      "  Read 2 with adapter:                  41,526 (99.8%)\n",
      "Pairs written (passing filters):        41,627 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,558,967 bp\n",
      "  Read 1:    12,279,582 bp\n",
      "  Read 2:    12,279,385 bp\n",
      "Total written (filtered):     22,937,845 bp (93.4%)\n",
      "  Read 1:    11,447,335 bp\n",
      "  Read 2:    11,490,510 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41620 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t252\t0.0\t1\t0 249 3\n",
      "20\t41264\t0.0\t2\t40603 650 11\n",
      "21\t69\t0.0\t2\t0 69\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41526 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t193\t0.0\t1\t0 193\n",
      "19\t41259\t0.0\t1\t40550 709\n",
      "20\t74\t0.0\t1\t0 74\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s535_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s535_166_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s535_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s535_166_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s535_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s535_166_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s535_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s535_166_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,208 reads  @     67.5 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.18 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,208\n",
      "  Read 1 with adapter:                  32,193 (100.0%)\n",
      "  Read 2 with adapter:                  32,126 (99.7%)\n",
      "Pairs written (passing filters):        32,208 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,998,709 bp\n",
      "  Read 1:     9,500,561 bp\n",
      "  Read 2:     9,498,148 bp\n",
      "Total written (filtered):     17,744,643 bp (93.4%)\n",
      "  Read 1:     8,856,797 bp\n",
      "  Read 2:     8,887,846 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32193 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t170\t0.0\t1\t0 169 1\n",
      "20\t31944\t0.0\t2\t31437 505 2\n",
      "21\t48\t0.0\t2\t0 48\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32126 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t150\t0.0\t1\t0 150\n",
      "19\t31918\t0.0\t1\t31414 504\n",
      "20\t58\t0.0\t1\t0 58\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s536_31_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s536_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s536_31_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s536_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s536_31_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s536_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s536_31_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s536_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        31,955 reads  @     70.2 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.25 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,955\n",
      "  Read 1 with adapter:                  31,942 (100.0%)\n",
      "  Read 2 with adapter:                  31,762 (99.4%)\n",
      "Pairs written (passing filters):        31,955 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,852,754 bp\n",
      "  Read 1:     9,426,293 bp\n",
      "  Read 2:     9,426,461 bp\n",
      "Total written (filtered):     17,610,689 bp (93.4%)\n",
      "  Read 1:     8,787,587 bp\n",
      "  Read 2:     8,823,102 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31942 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t219\t0.0\t1\t0 216 3\n",
      "20\t31633\t0.0\t2\t31069 557 7\n",
      "21\t63\t0.0\t2\t0 61 2\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31762 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t211\t0.0\t1\t0 211\n",
      "19\t31459\t0.0\t1\t30647 812\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s538_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s538_168_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s538_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s538_168_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s538_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s538_168_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s538_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s538_168_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,133 reads  @     70.2 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.52 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,133\n",
      "  Read 1 with adapter:                  50,106 (99.9%)\n",
      "  Read 2 with adapter:                  50,016 (99.8%)\n",
      "Pairs written (passing filters):        50,133 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,572,499 bp\n",
      "  Read 1:    14,787,342 bp\n",
      "  Read 2:    14,785,157 bp\n",
      "Total written (filtered):     27,620,381 bp (93.4%)\n",
      "  Read 1:    13,785,414 bp\n",
      "  Read 2:    13,834,967 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50106 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t308\t0.0\t1\t0 301 7\n",
      "20\t49665\t0.0\t2\t48777 882 6\n",
      "21\t82\t0.0\t2\t0 82\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50016 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t214\t0.0\t1\t0 214\n",
      "19\t49702\t0.0\t1\t48916 786\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s539_33_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s539_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s539_33_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s539_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s539_33_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s539_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s539_33_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s539_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:01        12,448 reads  @     92.9 µs/read;   0.65 M reads/minute\n",
      "Finished in 1.16 s (93 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             12,448\n",
      "  Read 1 with adapter:                  12,438 (99.9%)\n",
      "  Read 2 with adapter:                  12,381 (99.5%)\n",
      "Pairs written (passing filters):        12,448 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,339,259 bp\n",
      "  Read 1:     3,672,153 bp\n",
      "  Read 2:     3,667,106 bp\n",
      "Total written (filtered):      6,855,334 bp (93.4%)\n",
      "  Read 1:     3,423,418 bp\n",
      "  Read 2:     3,431,916 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 12438 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t80\t0.0\t1\t0 77 3\n",
      "20\t12310\t0.0\t2\t12038 270 2\n",
      "21\t29\t0.0\t2\t0 28 1\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 12381 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t79\t0.0\t1\t0 79\n",
      "19\t12272\t0.0\t1\t11981 291\n",
      "20\t30\t0.0\t1\t0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s540_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s540_170_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s540_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s540_170_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s540_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s540_170_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s540_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s540_170_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        49,138 reads  @     60.8 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.99 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,138\n",
      "  Read 1 with adapter:                  49,122 (100.0%)\n",
      "  Read 2 with adapter:                  48,516 (98.7%)\n",
      "Pairs written (passing filters):        49,138 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,957,223 bp\n",
      "  Read 1:    14,495,684 bp\n",
      "  Read 2:    14,461,539 bp\n",
      "Total written (filtered):     27,053,247 bp (93.4%)\n",
      "  Read 1:    13,513,394 bp\n",
      "  Read 2:    13,539,853 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49122 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t252\t0.0\t1\t0 239 13\n",
      "20\t48760\t0.0\t2\t47862 882 16\n",
      "21\t78\t0.0\t2\t0 77 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48516 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t211\t0.0\t1\t0 211\n",
      "19\t48212\t0.0\t1\t47410 802\n",
      "20\t93\t0.0\t1\t0 93\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s541_35_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s541_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s541_35_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s541_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s541_35_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s541_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s541_35_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s541_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,570 reads  @     68.4 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.09 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,570\n",
      "  Read 1 with adapter:                  30,565 (100.0%)\n",
      "  Read 2 with adapter:                  30,537 (99.9%)\n",
      "Pairs written (passing filters):        30,570 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,035,991 bp\n",
      "  Read 1:     9,018,139 bp\n",
      "  Read 2:     9,017,852 bp\n",
      "Total written (filtered):     16,844,622 bp (93.4%)\n",
      "  Read 1:     8,406,884 bp\n",
      "  Read 2:     8,437,738 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30565 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t146\t0.0\t1\t0 139 7\n",
      "20\t30312\t0.0\t2\t29748 559 5\n",
      "21\t81\t0.0\t2\t0 81\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30537 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t146\t0.0\t1\t0 146\n",
      "19\t30334\t0.0\t1\t29958 376\n",
      "20\t57\t0.0\t1\t0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s542_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s542_172_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s542_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s542_172_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s542_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s542_172_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s542_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s542_172_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,995 reads  @     61.5 µs/read;   0.98 M reads/minute\n",
      "Finished in 2.22 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,995\n",
      "  Read 1 with adapter:                  35,985 (100.0%)\n",
      "  Read 2 with adapter:                  35,881 (99.7%)\n",
      "Pairs written (passing filters):        35,995 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,236,774 bp\n",
      "  Read 1:    10,618,514 bp\n",
      "  Read 2:    10,618,260 bp\n",
      "Total written (filtered):     19,835,516 bp (93.4%)\n",
      "  Read 1:     9,898,893 bp\n",
      "  Read 2:     9,936,623 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35985 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t203\t0.0\t1\t0 201 2\n",
      "20\t35668\t0.0\t2\t35133 529 6\n",
      "21\t84\t0.0\t2\t0 83 1\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35881 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t168\t0.0\t1\t0 168\n",
      "19\t35647\t0.0\t1\t34973 674\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s543_37_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s543_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s543_37_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s543_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s543_37_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s543_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s543_37_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s543_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        27,701 reads  @     61.6 µs/read;   0.97 M reads/minute\n",
      "Finished in 1.71 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,701\n",
      "  Read 1 with adapter:                  27,693 (100.0%)\n",
      "  Read 2 with adapter:                  27,633 (99.8%)\n",
      "Pairs written (passing filters):        27,701 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,343,228 bp\n",
      "  Read 1:     8,171,785 bp\n",
      "  Read 2:     8,171,443 bp\n",
      "Total written (filtered):     15,264,533 bp (93.4%)\n",
      "  Read 1:     7,618,035 bp\n",
      "  Read 2:     7,646,498 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 27693 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t160\t0.0\t1\t0 157 3\n",
      "20\t27453\t0.0\t2\t27068 381 4\n",
      "21\t54\t0.0\t2\t0 53 1\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 27633 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t130\t0.0\t1\t0 130\n",
      "19\t27455\t0.0\t1\t26976 479\n",
      "20\t48\t0.0\t1\t0 48\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s544_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s544_174_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s544_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s544_174_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s544_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s544_174_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s544_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s544_174_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,240 reads  @     67.0 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.10 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,240\n",
      "  Read 1 with adapter:                  31,228 (100.0%)\n",
      "  Read 2 with adapter:                  31,023 (99.3%)\n",
      "Pairs written (passing filters):        31,240 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,431,210 bp\n",
      "  Read 1:     9,215,701 bp\n",
      "  Read 2:     9,215,509 bp\n",
      "Total written (filtered):     17,217,438 bp (93.4%)\n",
      "  Read 1:     8,591,239 bp\n",
      "  Read 2:     8,626,199 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31228 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t186\t0.0\t1\t0 182 4\n",
      "20\t30949\t0.0\t2\t30443 496 10\n",
      "21\t70\t0.0\t2\t0 68 2\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31023 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t191\t0.0\t1\t0 191\n",
      "19\t30768\t0.0\t1\t29857 911\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s546_39_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s546_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s546_39_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s546_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s546_39_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s546_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s546_39_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s546_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,713 reads  @     61.6 µs/read;   0.97 M reads/minute\n",
      "Finished in 2.26 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,713\n",
      "  Read 1 with adapter:                  36,702 (100.0%)\n",
      "  Read 2 with adapter:                  36,608 (99.7%)\n",
      "Pairs written (passing filters):        36,713 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,656,405 bp\n",
      "  Read 1:    10,830,244 bp\n",
      "  Read 2:    10,826,161 bp\n",
      "Total written (filtered):     20,226,957 bp (93.4%)\n",
      "  Read 1:    10,096,270 bp\n",
      "  Read 2:    10,130,687 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36702 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t164\t0.0\t1\t0 162 2\n",
      "20\t36430\t0.0\t2\t35854 571 5\n",
      "21\t82\t0.0\t2\t0 81 1\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36608 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t162\t0.0\t1\t0 162\n",
      "19\t36362\t0.0\t1\t35688 674\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s547_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s547_176_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s547_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s547_176_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s547_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s547_176_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s547_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s547_176_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,494 reads  @     61.8 µs/read;   0.97 M reads/minute\n",
      "Finished in 2.32 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,494\n",
      "  Read 1 with adapter:                  37,478 (100.0%)\n",
      "  Read 2 with adapter:                  37,401 (99.8%)\n",
      "Pairs written (passing filters):        37,494 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,116,801 bp\n",
      "  Read 1:    11,060,697 bp\n",
      "  Read 2:    11,056,104 bp\n",
      "Total written (filtered):     20,656,802 bp (93.4%)\n",
      "  Read 1:    10,311,221 bp\n",
      "  Read 2:    10,345,581 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37478 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t197\t0.0\t1\t0 195 2\n",
      "20\t37160\t0.0\t2\t36592 564 4\n",
      "21\t97\t0.0\t2\t0 96 1\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37401 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t169\t0.0\t1\t0 169\n",
      "19\t37159\t0.0\t1\t36539 620\n",
      "20\t73\t0.0\t1\t0 73\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s548_41_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s548_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s548_41_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s548_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s548_41_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s548_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s548_41_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s548_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,488 reads  @     63.1 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.24 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,488\n",
      "  Read 1 with adapter:                  35,473 (100.0%)\n",
      "  Read 2 with adapter:                  34,972 (98.5%)\n",
      "Pairs written (passing filters):        35,488 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,908,907 bp\n",
      "  Read 1:    10,468,948 bp\n",
      "  Read 2:    10,439,959 bp\n",
      "Total written (filtered):     19,535,185 bp (93.4%)\n",
      "  Read 1:     9,759,587 bp\n",
      "  Read 2:     9,775,598 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35473 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t204\t0.0\t1\t0 200 4\n",
      "20\t35155\t0.0\t2\t34608 539 8\n",
      "21\t79\t0.0\t2\t0 79\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34972 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t171\t0.0\t1\t0 171\n",
      "19\t34737\t0.0\t1\t34183 554\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s549_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s549_178_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s549_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s549_178_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s549_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s549_178_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s549_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s549_178_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:01        19,695 reads  @     70.8 µs/read;   0.85 M reads/minute\n",
      "Finished in 1.40 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             19,695\n",
      "  Read 1 with adapter:                  19,693 (100.0%)\n",
      "  Read 2 with adapter:                  19,662 (99.8%)\n",
      "Pairs written (passing filters):        19,695 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,620,022 bp\n",
      "  Read 1:     5,810,018 bp\n",
      "  Read 2:     5,810,004 bp\n",
      "Total written (filtered):     10,852,733 bp (93.4%)\n",
      "  Read 1:     5,416,228 bp\n",
      "  Read 2:     5,436,505 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 19693 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t110\t0.0\t1\t0 107 3\n",
      "20\t19525\t0.0\t2\t18937 578 10\n",
      "21\t40\t0.0\t2\t0 37 3\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 19662 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t103\t0.0\t1\t0 103\n",
      "19\t19535\t0.0\t1\t19275 260\n",
      "20\t24\t0.0\t1\t0 24\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s550_43_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s550_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s550_43_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s550_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s550_43_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s550_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s550_43_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s550_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        42,727 reads  @     67.8 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.90 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,727\n",
      "  Read 1 with adapter:                  33,915 (79.4%)\n",
      "  Read 2 with adapter:                  42,473 (99.4%)\n",
      "Pairs written (passing filters):        42,727 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,121,084 bp\n",
      "  Read 1:    10,517,341 bp\n",
      "  Read 2:    12,603,743 bp\n",
      "Total written (filtered):     21,636,047 bp (93.6%)\n",
      "  Read 1:     9,839,178 bp\n",
      "  Read 2:    11,796,869 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33915 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t210\t0.0\t1\t0 207 3\n",
      "20\t33600\t0.0\t2\t32936 657 7\n",
      "21\t81\t0.0\t2\t0 79 2\n",
      "22\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42473 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t193\t0.0\t1\t0 193\n",
      "19\t42200\t0.0\t1\t41032 1168\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s551_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s551_180_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s551_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s551_180_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s551_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s551_180_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s551_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s551_180_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,328 reads  @     70.0 µs/read;   0.86 M reads/minute\n",
      "Finished in 2.62 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,328\n",
      "  Read 1 with adapter:                  37,311 (100.0%)\n",
      "  Read 2 with adapter:                  37,278 (99.9%)\n",
      "Pairs written (passing filters):        37,328 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,023,209 bp\n",
      "  Read 1:    11,011,747 bp\n",
      "  Read 2:    11,011,462 bp\n",
      "Total written (filtered):     20,568,993 bp (93.4%)\n",
      "  Read 1:    10,265,697 bp\n",
      "  Read 2:    10,303,296 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37311 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t256\t0.0\t1\t0 252 4\n",
      "20\t36950\t0.0\t2\t36244 700 6\n",
      "21\t68\t0.0\t2\t0 67 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37278 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t185\t0.0\t1\t0 185\n",
      "19\t37024\t0.0\t1\t36556 468\n",
      "20\t69\t0.0\t1\t0 69\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s552_45_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s552_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s552_45_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s552_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s552_45_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s552_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s552_45_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s552_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,354 reads  @     64.2 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.79 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,354\n",
      "  Read 1 with adapter:                  43,329 (99.9%)\n",
      "  Read 2 with adapter:                  43,238 (99.7%)\n",
      "Pairs written (passing filters):        43,354 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,578,425 bp\n",
      "  Read 1:    12,789,169 bp\n",
      "  Read 2:    12,789,256 bp\n",
      "Total written (filtered):     23,890,627 bp (93.4%)\n",
      "  Read 1:    11,922,752 bp\n",
      "  Read 2:    11,967,875 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43329 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t267\t0.0\t1\t0 263 4\n",
      "20\t42942\t0.0\t2\t42213 722 7\n",
      "21\t80\t0.0\t2\t0 77 3\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43238 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t216\t0.0\t1\t0 216\n",
      "19\t42947\t0.0\t1\t42187 760\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s553_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s553_182_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s553_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s553_182_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s553_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s553_182_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s553_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s553_182_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,703 reads  @     59.4 µs/read;   1.01 M reads/minute\n",
      "Finished in 2.06 s (59 us/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,703\n",
      "  Read 1 with adapter:                  34,689 (100.0%)\n",
      "  Read 2 with adapter:                  34,617 (99.8%)\n",
      "Pairs written (passing filters):        34,703 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,474,263 bp\n",
      "  Read 1:    10,237,373 bp\n",
      "  Read 2:    10,236,890 bp\n",
      "Total written (filtered):     19,122,966 bp (93.4%)\n",
      "  Read 1:     9,543,698 bp\n",
      "  Read 2:     9,579,268 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34689 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t210\t0.0\t1\t0 207 3\n",
      "20\t34379\t0.0\t2\t33806 565 8\n",
      "21\t59\t0.0\t2\t0 59\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34617 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t164\t0.0\t1\t0 164\n",
      "19\t34390\t0.0\t1\t33816 574\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s554_47_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s554_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s554_47_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s554_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s554_47_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s554_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s554_47_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s554_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,058 reads  @     63.8 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.24 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,058\n",
      "  Read 1 with adapter:                  35,049 (100.0%)\n",
      "  Read 2 with adapter:                  34,873 (99.5%)\n",
      "Pairs written (passing filters):        35,058 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,683,280 bp\n",
      "  Read 1:    10,341,831 bp\n",
      "  Read 2:    10,341,449 bp\n",
      "Total written (filtered):     19,319,967 bp (93.4%)\n",
      "  Read 1:     9,640,971 bp\n",
      "  Read 2:     9,678,996 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35049 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t217\t0.0\t1\t0 214 3\n",
      "20\t34733\t0.0\t2\t34120 606 7\n",
      "21\t65\t0.0\t2\t0 64 1\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34873 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t226\t0.0\t1\t0 226\n",
      "19\t34555\t0.0\t1\t33741 814\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s556_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s556_184_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s556_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s556_184_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s556_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s556_184_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s556_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s556_184_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        45,701 reads  @     60.4 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.76 s (60 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,701\n",
      "  Read 1 with adapter:                  45,673 (99.9%)\n",
      "  Read 2 with adapter:                  45,597 (99.8%)\n",
      "Pairs written (passing filters):        45,701 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,958,495 bp\n",
      "  Read 1:    13,481,733 bp\n",
      "  Read 2:    13,476,762 bp\n",
      "Total written (filtered):     25,178,977 bp (93.4%)\n",
      "  Read 1:    12,568,443 bp\n",
      "  Read 2:    12,610,534 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45673 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t294\t0.0\t1\t0 289 5\n",
      "20\t45256\t0.0\t2\t44554 691 11\n",
      "21\t82\t0.0\t2\t0 81 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45597 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t208\t0.0\t1\t0 208\n",
      "19\t45296\t0.0\t1\t44553 743\n",
      "20\t93\t0.0\t1\t0 93\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s557_49_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s557_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s557_49_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s557_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s557_49_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s557_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s557_49_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s557_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        41,555 reads  @     60.2 µs/read;   1.00 M reads/minute\n",
      "Finished in 2.51 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,555\n",
      "  Read 1 with adapter:                  41,531 (99.9%)\n",
      "  Read 2 with adapter:                  41,449 (99.7%)\n",
      "Pairs written (passing filters):        41,555 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,513,148 bp\n",
      "  Read 1:    12,258,718 bp\n",
      "  Read 2:    12,254,430 bp\n",
      "Total written (filtered):     22,895,266 bp (93.4%)\n",
      "  Read 1:    11,428,273 bp\n",
      "  Read 2:    11,466,993 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41531 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t278\t0.0\t1\t0 275 3\n",
      "20\t41132\t0.0\t2\t40456 671 5\n",
      "21\t95\t0.0\t2\t0 95\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41449 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t189\t0.0\t1\t0 189\n",
      "19\t41165\t0.0\t1\t40549 616\n",
      "20\t95\t0.0\t1\t0 95\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s558_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s558_186_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s558_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s558_186_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s558_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s558_186_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s558_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s558_186_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        46,898 reads  @     61.2 µs/read;   0.98 M reads/minute\n",
      "Finished in 2.87 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,898\n",
      "  Read 1 with adapter:                  46,872 (99.9%)\n",
      "  Read 2 with adapter:                  46,315 (98.8%)\n",
      "Pairs written (passing filters):        46,898 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,638,601 bp\n",
      "  Read 1:    13,834,900 bp\n",
      "  Read 2:    13,803,701 bp\n",
      "Total written (filtered):     25,821,493 bp (93.4%)\n",
      "  Read 1:    12,897,624 bp\n",
      "  Read 2:    12,923,869 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46872 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t290\t0.0\t1\t0 287 3\n",
      "20\t46448\t0.0\t2\t45618 817 13\n",
      "21\t94\t0.0\t2\t0 93 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46315 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t230\t0.0\t1\t0 230\n",
      "19\t46008\t0.0\t1\t45336 672\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s559_51_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s559_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s559_51_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s559_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s559_51_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s559_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s559_51_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s559_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,635 reads  @     64.1 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.35 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,635\n",
      "  Read 1 with adapter:                  36,626 (100.0%)\n",
      "  Read 2 with adapter:                  36,582 (99.9%)\n",
      "Pairs written (passing filters):        36,635 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,614,373 bp\n",
      "  Read 1:    10,807,311 bp\n",
      "  Read 2:    10,807,062 bp\n",
      "Total written (filtered):     20,187,149 bp (93.4%)\n",
      "  Read 1:    10,075,050 bp\n",
      "  Read 2:    10,112,099 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t344\t0.0\t1\t0 339 5\n",
      "20\t36177\t0.0\t2\t35568 600 9\n",
      "21\t69\t0.0\t2\t0 69\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36582 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t161\t0.0\t1\t0 161\n",
      "19\t36355\t0.0\t1\t35762 593\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s560_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s560_188_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s560_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s560_188_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s560_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s560_188_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s560_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s560_188_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,699 reads  @     64.5 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.05 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,699\n",
      "  Read 1 with adapter:                  31,690 (100.0%)\n",
      "  Read 2 with adapter:                  31,623 (99.8%)\n",
      "Pairs written (passing filters):        31,699 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,702,188 bp\n",
      "  Read 1:     9,351,095 bp\n",
      "  Read 2:     9,351,093 bp\n",
      "Total written (filtered):     17,467,844 bp (93.4%)\n",
      "  Read 1:     8,717,525 bp\n",
      "  Read 2:     8,750,319 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31690 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t295\t0.0\t1\t0 289 6\n",
      "20\t31316\t0.0\t2\t30802 504 10\n",
      "21\t57\t0.0\t2\t0 57\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31623 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t134\t0.0\t1\t0 134\n",
      "19\t31418\t0.0\t1\t30770 648\n",
      "20\t71\t0.0\t1\t0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s561_53_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s561_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s561_53_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s561_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s561_53_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s561_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s561_53_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s561_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        30,333 reads  @     61.4 µs/read;   0.98 M reads/minute\n",
      "Finished in 1.86 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,333\n",
      "  Read 1 with adapter:                  30,324 (100.0%)\n",
      "  Read 2 with adapter:                  30,221 (99.6%)\n",
      "Pairs written (passing filters):        30,333 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,896,028 bp\n",
      "  Read 1:     8,948,226 bp\n",
      "  Read 2:     8,947,802 bp\n",
      "Total written (filtered):     16,715,616 bp (93.4%)\n",
      "  Read 1:     8,341,942 bp\n",
      "  Read 2:     8,373,674 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30324 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t290\t0.0\t1\t0 287 3\n",
      "20\t29945\t0.0\t2\t29475 467 3\n",
      "21\t56\t0.0\t2\t0 56\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30221 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t126\t0.0\t1\t0 126\n",
      "19\t30040\t0.0\t1\t29456 584\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s562_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s562_190_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s562_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s562_190_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s562_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s562_190_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s562_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s562_190_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        31,275 reads  @     63.7 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.00 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,275\n",
      "  Read 1 with adapter:                  31,263 (100.0%)\n",
      "  Read 2 with adapter:                  31,069 (99.3%)\n",
      "Pairs written (passing filters):        31,275 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,452,081 bp\n",
      "  Read 1:     9,226,115 bp\n",
      "  Read 2:     9,225,966 bp\n",
      "Total written (filtered):     17,236,802 bp (93.4%)\n",
      "  Read 1:     8,601,029 bp\n",
      "  Read 2:     8,635,773 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31263 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t278\t0.0\t1\t0 275 3\n",
      "20\t30872\t0.0\t2\t30392 470 10\n",
      "21\t78\t0.0\t2\t0 77 1\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31069 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t181\t0.0\t1\t0 181\n",
      "19\t30825\t0.0\t1\t29804 1021\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s564_55_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s564_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s564_55_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s564_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s564_55_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s564_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s564_55_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s564_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        41,220 reads  @     60.1 µs/read;   1.00 M reads/minute\n",
      "Finished in 2.48 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,220\n",
      "  Read 1 with adapter:                  41,209 (100.0%)\n",
      "  Read 2 with adapter:                  41,079 (99.7%)\n",
      "Pairs written (passing filters):        41,220 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,316,163 bp\n",
      "  Read 1:    12,159,887 bp\n",
      "  Read 2:    12,156,276 bp\n",
      "Total written (filtered):     22,711,851 bp (93.4%)\n",
      "  Read 1:    11,335,970 bp\n",
      "  Read 2:    11,375,881 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41209 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t383\t0.0\t1\t0 379 4\n",
      "20\t40709\t0.0\t2\t40038 669 2\n",
      "21\t74\t0.0\t2\t0 73 1\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41079 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t186\t0.0\t1\t0 186\n",
      "19\t40813\t0.0\t1\t39980 833\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s565_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s565_192_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s565_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s565_192_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s565_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s565_192_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s565_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s565_192_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,514 reads  @     59.7 µs/read;   1.00 M reads/minute\n",
      "Finished in 2.30 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,514\n",
      "  Read 1 with adapter:                  38,504 (100.0%)\n",
      "  Read 2 with adapter:                  38,397 (99.7%)\n",
      "Pairs written (passing filters):        38,514 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,719,596 bp\n",
      "  Read 1:    11,361,619 bp\n",
      "  Read 2:    11,357,977 bp\n",
      "Total written (filtered):     21,220,350 bp (93.4%)\n",
      "  Read 1:    10,591,784 bp\n",
      "  Read 2:    10,628,566 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38504 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t345\t0.0\t1\t0 341 4\n",
      "20\t38053\t0.0\t2\t37462 587 4\n",
      "21\t76\t0.0\t2\t0 72 4\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38397 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t193\t0.0\t1\t0 193\n",
      "19\t38143\t0.0\t1\t37450 693\n",
      "20\t61\t0.0\t1\t0 61\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s566_57_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s566_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s566_57_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s566_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s566_57_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s566_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s566_57_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s566_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        45,470 reads  @     60.4 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.75 s (60 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,470\n",
      "  Read 1 with adapter:                  45,458 (100.0%)\n",
      "  Read 2 with adapter:                  44,777 (98.5%)\n",
      "Pairs written (passing filters):        45,470 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,789,040 bp\n",
      "  Read 1:    13,413,642 bp\n",
      "  Read 2:    13,375,398 bp\n",
      "Total written (filtered):     25,029,624 bp (93.4%)\n",
      "  Read 1:    12,504,845 bp\n",
      "  Read 2:    12,524,779 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45458 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t446\t0.0\t1\t0 432 14\n",
      "20\t44892\t0.0\t2\t44147 733 12\n",
      "21\t77\t0.0\t2\t0 77\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44777 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t212\t0.0\t1\t0 212\n",
      "19\t44497\t0.0\t1\t43743 754\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s567_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s567_194_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s567_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s567_194_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s567_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s567_194_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s567_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s567_194_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,304 reads  @     62.2 µs/read;   0.96 M reads/minute\n",
      "Finished in 2.39 s (62 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,304\n",
      "  Read 1 with adapter:                  38,283 (99.9%)\n",
      "  Read 2 with adapter:                  38,232 (99.8%)\n",
      "Pairs written (passing filters):        38,304 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,597,973 bp\n",
      "  Read 1:    11,298,319 bp\n",
      "  Read 2:    11,299,654 bp\n",
      "Total written (filtered):     21,106,213 bp (93.4%)\n",
      "  Read 1:    10,532,840 bp\n",
      "  Read 2:    10,573,373 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38283 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t242\t0.0\t1\t0 235 7\n",
      "20\t37948\t0.0\t2\t37309 634 5\n",
      "21\t61\t0.0\t2\t0 61\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38232 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t205\t0.0\t1\t0 205\n",
      "19\t37949\t0.0\t1\t37383 566\n",
      "20\t78\t0.0\t1\t0 78\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s568_59_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s568_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s568_59_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s568_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s568_59_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s568_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s568_59_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s568_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,842 reads  @     59.5 µs/read;   1.01 M reads/minute\n",
      "Finished in 2.38 s (60 us/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,842\n",
      "  Read 1 with adapter:                  39,825 (100.0%)\n",
      "  Read 2 with adapter:                  39,766 (99.8%)\n",
      "Pairs written (passing filters):        39,842 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,505,772 bp\n",
      "  Read 1:    11,752,646 bp\n",
      "  Read 2:    11,753,126 bp\n",
      "Total written (filtered):     21,953,988 bp (93.4%)\n",
      "  Read 1:    10,956,288 bp\n",
      "  Read 2:    10,997,700 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39825 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t252\t0.0\t1\t0 248 4\n",
      "20\t39458\t0.0\t2\t38833 620 5\n",
      "21\t84\t0.0\t2\t0 84\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39766 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t179\t0.0\t1\t0 179\n",
      "19\t39536\t0.0\t1\t38946 590\n",
      "20\t51\t0.0\t1\t0 51\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s569_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s569_196_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s569_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s569_196_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s569_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s569_196_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s569_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s569_196_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        32,092 reads  @     58.9 µs/read;   1.02 M reads/minute\n",
      "Finished in 1.89 s (59 us/read; 1.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,092\n",
      "  Read 1 with adapter:                  32,075 (99.9%)\n",
      "  Read 2 with adapter:                  32,041 (99.8%)\n",
      "Pairs written (passing filters):        32,092 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,933,789 bp\n",
      "  Read 1:     9,466,807 bp\n",
      "  Read 2:     9,466,982 bp\n",
      "Total written (filtered):     17,683,750 bp (93.4%)\n",
      "  Read 1:     8,825,452 bp\n",
      "  Read 2:     8,858,298 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32075 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t209\t0.0\t1\t0 206 3\n",
      "20\t31783\t0.0\t2\t31264 517 2\n",
      "21\t50\t0.0\t2\t0 50\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32041 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t163\t0.0\t1\t0 163\n",
      "19\t31810\t0.0\t1\t31311 499\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s570_61_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s570_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s570_61_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s570_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s570_61_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s570_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s570_61_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s570_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        28,679 reads  @     65.5 µs/read;   0.92 M reads/minute\n",
      "Finished in 1.88 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,679\n",
      "  Read 1 with adapter:                  28,664 (99.9%)\n",
      "  Read 2 with adapter:                  28,529 (99.5%)\n",
      "Pairs written (passing filters):        28,679 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,919,348 bp\n",
      "  Read 1:     8,460,015 bp\n",
      "  Read 2:     8,459,333 bp\n",
      "Total written (filtered):     15,804,259 bp (93.4%)\n",
      "  Read 1:     7,886,844 bp\n",
      "  Read 2:     7,917,415 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 28664 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t172\t0.0\t1\t0 170 2\n",
      "20\t28412\t0.0\t2\t27937 467 8\n",
      "21\t53\t0.0\t2\t0 53\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 28529 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t198\t0.0\t1\t0 198\n",
      "19\t28266\t0.0\t1\t27637 629\n",
      "20\t65\t0.0\t1\t0 65\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s572_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s572_198_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s572_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s572_198_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s572_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s572_198_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s572_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s572_198_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        44,865 reads  @     60.2 µs/read;   1.00 M reads/minute\n",
      "Finished in 2.70 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,865\n",
      "  Read 1 with adapter:                  44,837 (99.9%)\n",
      "  Read 2 with adapter:                  44,767 (99.8%)\n",
      "Pairs written (passing filters):        44,865 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,466,156 bp\n",
      "  Read 1:    13,233,847 bp\n",
      "  Read 2:    13,232,309 bp\n",
      "Total written (filtered):     24,719,137 bp (93.4%)\n",
      "  Read 1:    12,337,289 bp\n",
      "  Read 2:    12,381,848 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44837 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t314\t0.0\t1\t0 304 10\n",
      "20\t44386\t0.0\t2\t43544 831 11\n",
      "21\t94\t0.0\t2\t0 93 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44767 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t208\t0.0\t1\t0 208\n",
      "19\t44463\t0.0\t1\t43765 698\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s573_63_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s573_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s573_63_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s573_199_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s573_63_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s573_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s573_63_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s573_199_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        41,389 reads  @     64.5 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.67 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,389\n",
      "  Read 1 with adapter:                  41,371 (100.0%)\n",
      "  Read 2 with adapter:                  41,287 (99.8%)\n",
      "Pairs written (passing filters):        41,389 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,416,377 bp\n",
      "  Read 1:    12,209,396 bp\n",
      "  Read 2:    12,206,981 bp\n",
      "Total written (filtered):     22,804,838 bp (93.4%)\n",
      "  Read 1:    11,382,194 bp\n",
      "  Read 2:    11,422,644 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41371 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t278\t0.0\t1\t0 274 4\n",
      "20\t41006\t0.0\t2\t40306 693 7\n",
      "21\t58\t0.0\t2\t0 57 1\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41287 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t203\t0.0\t1\t0 203\n",
      "19\t40997\t0.0\t1\t40348 649\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s574_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s574_200_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s574_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s574_200_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s574_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s574_200_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s574_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s574_200_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,224 reads  @     65.8 µs/read;   0.91 M reads/minute\n",
      "Finished in 3.18 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,224\n",
      "  Read 1 with adapter:                  48,201 (100.0%)\n",
      "  Read 2 with adapter:                  47,622 (98.8%)\n",
      "Pairs written (passing filters):        48,224 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,418,381 bp\n",
      "  Read 1:    14,225,403 bp\n",
      "  Read 2:    14,192,978 bp\n",
      "Total written (filtered):     26,549,924 bp (93.4%)\n",
      "  Read 1:    13,261,591 bp\n",
      "  Read 2:    13,288,333 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48201 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t307\t0.0\t1\t0 298 9\n",
      "20\t47746\t0.0\t2\t46903 831 12\n",
      "21\t97\t0.0\t2\t0 94 3\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47622 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t257\t0.0\t1\t0 257\n",
      "19\t47281\t0.0\t1\t46578 703\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s575_65_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s575_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s575_65_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s575_201_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s575_65_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s575_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s575_65_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s575_201_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,393 reads  @     70.5 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.36 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,393\n",
      "  Read 1 with adapter:                  33,377 (100.0%)\n",
      "  Read 2 with adapter:                  33,318 (99.8%)\n",
      "Pairs written (passing filters):        33,393 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,698,594 bp\n",
      "  Read 1:     9,849,845 bp\n",
      "  Read 2:     9,848,749 bp\n",
      "Total written (filtered):     18,398,217 bp (93.4%)\n",
      "  Read 1:     9,182,402 bp\n",
      "  Read 2:     9,215,815 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33377 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t184\t0.0\t1\t0 179 5\n",
      "20\t33106\t0.0\t2\t32585 521\n",
      "21\t59\t0.0\t2\t0 58 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33318 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t173\t0.0\t1\t0 173\n",
      "19\t33080\t0.0\t1\t32620 460\n",
      "20\t65\t0.0\t1\t0 65\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s576_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s576_202_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s576_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s576_202_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s576_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s576_202_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s576_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s576_202_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,840 reads  @     64.8 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.07 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,840\n",
      "  Read 1 with adapter:                  31,828 (100.0%)\n",
      "  Read 2 with adapter:                  31,781 (99.8%)\n",
      "Pairs written (passing filters):        31,840 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,785,253 bp\n",
      "  Read 1:     9,392,549 bp\n",
      "  Read 2:     9,392,704 bp\n",
      "Total written (filtered):     17,545,030 bp (93.4%)\n",
      "  Read 1:     8,756,066 bp\n",
      "  Read 2:     8,788,964 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31828 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t182\t0.0\t1\t0 179 3\n",
      "20\t31548\t0.0\t2\t31041 504 3\n",
      "21\t67\t0.0\t2\t0 66 1\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31781 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t144\t0.0\t1\t0 144\n",
      "19\t31592\t0.0\t1\t31101 491\n",
      "20\t45\t0.0\t1\t0 45\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s577_67_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s577_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s577_67_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s577_203_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s577_67_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s577_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s577_67_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s577_203_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        26,042 reads  @     67.4 µs/read;   0.89 M reads/minute\n",
      "Finished in 1.76 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,042\n",
      "  Read 1 with adapter:                  26,034 (100.0%)\n",
      "  Read 2 with adapter:                  25,958 (99.7%)\n",
      "Pairs written (passing filters):        26,042 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,361,036 bp\n",
      "  Read 1:     7,682,385 bp\n",
      "  Read 2:     7,678,651 bp\n",
      "Total written (filtered):     14,347,300 bp (93.4%)\n",
      "  Read 1:     7,161,782 bp\n",
      "  Read 2:     7,185,518 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 26034 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t157\t0.0\t1\t0 154 3\n",
      "20\t25798\t0.0\t2\t25382 408 8\n",
      "21\t54\t0.0\t2\t0 54\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 25958 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t118\t0.0\t1\t0 118\n",
      "19\t25791\t0.0\t1\t25379 412\n",
      "20\t49\t0.0\t1\t0 49\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s578_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s578_204_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s578_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s578_204_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s578_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s578_204_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s578_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s578_204_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,448 reads  @     73.3 µs/read;   0.82 M reads/minute\n",
      "Finished in 2.24 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,448\n",
      "  Read 1 with adapter:                  30,436 (100.0%)\n",
      "  Read 2 with adapter:                  30,255 (99.4%)\n",
      "Pairs written (passing filters):        30,448 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,963,598 bp\n",
      "  Read 1:     8,981,962 bp\n",
      "  Read 2:     8,981,636 bp\n",
      "Total written (filtered):     16,780,232 bp (93.4%)\n",
      "  Read 1:     8,373,334 bp\n",
      "  Read 2:     8,406,898 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30436 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t178\t0.0\t1\t0 175 3\n",
      "20\t30161\t0.0\t2\t29664 492 5\n",
      "21\t48\t0.0\t2\t0 48\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30255 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t171\t0.0\t1\t0 171\n",
      "19\t30020\t0.0\t1\t29198 822\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s580_69_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s580_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s580_69_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s580_205_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s580_69_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s580_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s580_69_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s580_205_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,113 reads  @     67.7 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.65 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,113\n",
      "  Read 1 with adapter:                  39,102 (100.0%)\n",
      "  Read 2 with adapter:                  39,000 (99.7%)\n",
      "Pairs written (passing filters):        39,113 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,070,856 bp\n",
      "  Read 1:    11,538,205 bp\n",
      "  Read 2:    11,532,651 bp\n",
      "Total written (filtered):     21,548,042 bp (93.4%)\n",
      "  Read 1:    10,756,285 bp\n",
      "  Read 2:    10,791,757 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39102 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t231\t0.0\t1\t0 224 7\n",
      "20\t38753\t0.0\t2\t38109 637 7\n",
      "21\t69\t0.0\t2\t0 68 1\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39000 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t177\t0.0\t1\t0 177\n",
      "19\t38752\t0.0\t1\t38047 705\n",
      "20\t71\t0.0\t1\t0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s581_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s581_206_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s581_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s581_206_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s581_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s581_206_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s581_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s581_206_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,849 reads  @     73.2 µs/read;   0.82 M reads/minute\n",
      "Finished in 2.63 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,849\n",
      "  Read 1 with adapter:                  35,834 (100.0%)\n",
      "  Read 2 with adapter:                  35,732 (99.7%)\n",
      "Pairs written (passing filters):        35,849 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,145,033 bp\n",
      "  Read 1:    10,574,867 bp\n",
      "  Read 2:    10,570,166 bp\n",
      "Total written (filtered):     19,749,674 bp (93.4%)\n",
      "  Read 1:     9,858,315 bp\n",
      "  Read 2:     9,891,359 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35834 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t207\t0.0\t1\t0 204 3\n",
      "20\t35545\t0.0\t2\t35020 521 4\n",
      "21\t57\t0.0\t2\t0 55 2\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35732 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t185\t0.0\t1\t0 185\n",
      "19\t35463\t0.0\t1\t34928 535\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s582_71_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s582_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s582_71_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s582_207_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s582_71_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s582_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s582_71_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s582_207_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,024 reads  @     68.7 µs/read;   0.87 M reads/minute\n",
      "Finished in 2.62 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,024\n",
      "  Read 1 with adapter:                  38,007 (100.0%)\n",
      "  Read 2 with adapter:                  37,509 (98.6%)\n",
      "Pairs written (passing filters):        38,024 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,406,016 bp\n",
      "  Read 1:    11,216,749 bp\n",
      "  Read 2:    11,189,267 bp\n",
      "Total written (filtered):     20,933,420 bp (93.4%)\n",
      "  Read 1:    10,456,736 bp\n",
      "  Read 2:    10,476,684 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38007 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t231\t0.0\t1\t0 225 6\n",
      "20\t37671\t0.0\t2\t37101 563 7\n",
      "21\t74\t0.0\t2\t0 74\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37509 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t163\t0.0\t1\t0 163\n",
      "19\t37271\t0.0\t1\t36739 532\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s583_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s583_208_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s583_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s583_208_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s583_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s583_208_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s583_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s583_208_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,428 reads  @     71.1 µs/read;   0.84 M reads/minute\n",
      "Finished in 2.66 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,428\n",
      "  Read 1 with adapter:                  37,408 (99.9%)\n",
      "  Read 2 with adapter:                  37,357 (99.8%)\n",
      "Pairs written (passing filters):        37,428 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,080,568 bp\n",
      "  Read 1:    11,039,615 bp\n",
      "  Read 2:    11,040,953 bp\n",
      "Total written (filtered):     20,622,863 bp (93.4%)\n",
      "  Read 1:    10,291,598 bp\n",
      "  Read 2:    10,331,265 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37408 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t249\t0.0\t1\t0 244 5\n",
      "20\t37052\t0.0\t2\t36465 577 10\n",
      "21\t72\t0.0\t2\t0 69 3\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37357 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t176\t0.0\t1\t0 176\n",
      "19\t37100\t0.0\t1\t36574 526\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s584_73_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s584_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s584_73_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s584_209_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s584_73_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s584_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s584_73_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s584_209_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        40,583 reads  @     70.4 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.86 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,583\n",
      "  Read 1 with adapter:                  40,557 (99.9%)\n",
      "  Read 2 with adapter:                  40,478 (99.7%)\n",
      "Pairs written (passing filters):        40,583 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,941,757 bp\n",
      "  Read 1:    11,970,333 bp\n",
      "  Read 2:    11,971,424 bp\n",
      "Total written (filtered):     22,361,758 bp (93.4%)\n",
      "  Read 1:    11,159,311 bp\n",
      "  Read 2:    11,202,447 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40557 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t229\t0.0\t1\t0 227 2\n",
      "20\t40209\t0.0\t2\t39550 651 8\n",
      "21\t75\t0.0\t2\t0 74 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40478 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t168\t0.0\t1\t0 168\n",
      "19\t40247\t0.0\t1\t39595 652\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s585_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s585_210_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s585_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s585_210_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s585_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s585_210_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s585_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s585_210_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,268 reads  @     70.6 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.28 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,268\n",
      "  Read 1 with adapter:                  32,250 (99.9%)\n",
      "  Read 2 with adapter:                  32,115 (99.5%)\n",
      "Pairs written (passing filters):        32,268 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,032,154 bp\n",
      "  Read 1:     9,517,451 bp\n",
      "  Read 2:     9,514,703 bp\n",
      "Total written (filtered):     17,777,145 bp (93.4%)\n",
      "  Read 1:     8,872,539 bp\n",
      "  Read 2:     8,904,606 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32250 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t201\t0.0\t1\t0 201\n",
      "20\t31949\t0.0\t2\t31360 579 10\n",
      "21\t59\t0.0\t2\t0 58 1\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32115 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t162\t0.0\t1\t0 162\n",
      "19\t31879\t0.0\t1\t31256 623\n",
      "20\t74\t0.0\t1\t0 74\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s586_75_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s586_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s586_75_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s586_211_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s586_75_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s586_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s586_75_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s586_211_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,415 reads  @     70.7 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.23 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,415\n",
      "  Read 1 with adapter:                  31,399 (99.9%)\n",
      "  Read 2 with adapter:                  31,193 (99.3%)\n",
      "Pairs written (passing filters):        31,415 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,532,161 bp\n",
      "  Read 1:     9,265,487 bp\n",
      "  Read 2:     9,266,674 bp\n",
      "Total written (filtered):     17,311,727 bp (93.4%)\n",
      "  Read 1:     8,637,590 bp\n",
      "  Read 2:     8,674,137 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31399 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t174\t0.0\t1\t0 170 4\n",
      "20\t31131\t0.0\t2\t30635 492 4\n",
      "21\t61\t0.0\t2\t0 60 1\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31193 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t204\t0.0\t1\t0 204\n",
      "19\t30915\t0.0\t1\t29934 981\n",
      "20\t74\t0.0\t1\t0 74\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s588_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s588_212_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s588_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s588_212_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s588_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s588_212_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s588_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s588_212_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,475 reads  @     70.6 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.14 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,475\n",
      "  Read 1 with adapter:                  44,450 (99.9%)\n",
      "  Read 2 with adapter:                  44,313 (99.6%)\n",
      "Pairs written (passing filters):        44,475 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,231,823 bp\n",
      "  Read 1:    13,118,022 bp\n",
      "  Read 2:    13,113,801 bp\n",
      "Total written (filtered):     24,501,165 bp (93.4%)\n",
      "  Read 1:    12,229,174 bp\n",
      "  Read 2:    12,271,991 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44450 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t281\t0.0\t1\t0 272 9\n",
      "20\t44046\t0.0\t2\t43270 769 7\n",
      "21\t77\t0.0\t2\t0 77\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44313 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t220\t0.0\t1\t0 220\n",
      "19\t44010\t0.0\t1\t43176 834\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s589_77_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s589_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s589_77_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s589_213_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s589_77_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s589_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s589_77_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s589_213_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         8,721 reads  @    105.4 µs/read;   0.57 M reads/minute\n",
      "Finished in 0.92 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              8,721\n",
      "  Read 1 with adapter:                   8,715 (99.9%)\n",
      "  Read 2 with adapter:                   8,637 (99.0%)\n",
      "Pairs written (passing filters):         8,721 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,139,005 bp\n",
      "  Read 1:     2,571,935 bp\n",
      "  Read 2:     2,567,070 bp\n",
      "Total written (filtered):      4,800,651 bp (93.4%)\n",
      "  Read 1:     2,397,647 bp\n",
      "  Read 2:     2,403,004 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 8715 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t50\t0.0\t1\t0 49 1\n",
      "20\t8622\t0.0\t2\t8416 203 3\n",
      "21\t28\t0.0\t2\t0 26 2\n",
      "22\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 8637 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t65\t0.0\t1\t0 65\n",
      "19\t8544\t0.0\t1\t8353 191\n",
      "20\t28\t0.0\t1\t0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s590_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s590_214_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s590_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s590_214_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s590_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s590_214_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s590_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s590_214_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        45,082 reads  @     61.4 µs/read;   0.98 M reads/minute\n",
      "Finished in 2.77 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,082\n",
      "  Read 1 with adapter:                  45,064 (100.0%)\n",
      "  Read 2 with adapter:                  44,458 (98.6%)\n",
      "Pairs written (passing filters):        45,082 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,564,663 bp\n",
      "  Read 1:    13,298,921 bp\n",
      "  Read 2:    13,265,742 bp\n",
      "Total written (filtered):     24,818,914 bp (93.4%)\n",
      "  Read 1:    12,397,770 bp\n",
      "  Read 2:    12,421,144 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45064 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t268\t0.0\t1\t0 260 8\n",
      "20\t44646\t0.0\t2\t43939 699 8\n",
      "21\t93\t0.0\t2\t0 91 2\n",
      "22\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44458 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t193\t0.0\t1\t0 193\n",
      "19\t44176\t0.0\t1\t43472 704\n",
      "20\t89\t0.0\t1\t0 89\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s591_79_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s591_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s591_79_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s591_215_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s591_79_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s591_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s591_79_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s591_215_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        30,785 reads  @     70.4 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.17 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,785\n",
      "  Read 1 with adapter:                  30,775 (100.0%)\n",
      "  Read 2 with adapter:                  30,733 (99.8%)\n",
      "Pairs written (passing filters):        30,785 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,162,243 bp\n",
      "  Read 1:     9,080,679 bp\n",
      "  Read 2:     9,081,564 bp\n",
      "Total written (filtered):     16,963,006 bp (93.4%)\n",
      "  Read 1:     8,465,269 bp\n",
      "  Read 2:     8,497,737 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30775 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t162\t0.0\t1\t0 159 3\n",
      "20\t30542\t0.0\t2\t29937 592 13\n",
      "21\t46\t0.0\t2\t0 44 2\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30733 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t144\t0.0\t1\t0 144\n",
      "19\t30545\t0.0\t1\t30177 368\n",
      "20\t44\t0.0\t1\t0 44\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s592_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s592_216_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s592_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s592_216_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s592_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s592_216_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s592_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s592_216_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,568 reads  @     63.0 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.75 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,568\n",
      "  Read 1 with adapter:                  43,548 (100.0%)\n",
      "  Read 2 with adapter:                  43,452 (99.7%)\n",
      "Pairs written (passing filters):        43,568 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,703,982 bp\n",
      "  Read 1:    12,851,687 bp\n",
      "  Read 2:    12,852,295 bp\n",
      "Total written (filtered):     24,007,741 bp (93.4%)\n",
      "  Read 1:    11,980,923 bp\n",
      "  Read 2:    12,026,818 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43548 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t277\t0.0\t1\t0 271 6\n",
      "20\t43161\t0.0\t2\t42501 657 3\n",
      "21\t67\t0.0\t2\t0 67\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43452 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t191\t0.0\t1\t0 191\n",
      "19\t43181\t0.0\t1\t42352 829\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s593_81_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s593_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s593_81_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s593_217_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s593_81_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s593_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s593_81_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s593_217_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,656 reads  @     64.7 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.05 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,656\n",
      "  Read 1 with adapter:                  31,639 (99.9%)\n",
      "  Read 2 with adapter:                  31,540 (99.6%)\n",
      "Pairs written (passing filters):        31,656 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,675,117 bp\n",
      "  Read 1:     9,338,082 bp\n",
      "  Read 2:     9,337,035 bp\n",
      "Total written (filtered):     17,443,261 bp (93.4%)\n",
      "  Read 1:     8,705,415 bp\n",
      "  Read 2:     8,737,846 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31639 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t177\t0.0\t1\t0 172 5\n",
      "20\t31385\t0.0\t2\t30879 503 3\n",
      "21\t62\t0.0\t2\t0 61 1\n",
      "22\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31540 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t139\t0.0\t1\t0 139\n",
      "19\t31333\t0.0\t1\t30710 623\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s594_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s594_218_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s594_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s594_218_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s594_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s594_218_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s594_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s594_218_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,777 reads  @     65.6 µs/read;   0.91 M reads/minute\n",
      "Finished in 2.28 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,777\n",
      "  Read 1 with adapter:                  34,758 (99.9%)\n",
      "  Read 2 with adapter:                  34,576 (99.4%)\n",
      "Pairs written (passing filters):        34,777 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,517,520 bp\n",
      "  Read 1:    10,258,843 bp\n",
      "  Read 2:    10,258,677 bp\n",
      "Total written (filtered):     19,165,637 bp (93.4%)\n",
      "  Read 1:     9,563,790 bp\n",
      "  Read 2:     9,601,847 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34758 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t217\t0.0\t1\t0 215 2\n",
      "20\t34431\t0.0\t2\t33831 588 12\n",
      "21\t70\t0.0\t2\t0 67 3\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34576 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t204\t0.0\t1\t0 204\n",
      "19\t34282\t0.0\t1\t33315 967\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s596_83_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s596_219_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s596_83_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s596_219_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s596_83_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s596_219_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s596_83_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s596_219_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,541 reads  @     60.9 µs/read;   0.99 M reads/minute\n",
      "Finished in 3.02 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,541\n",
      "  Read 1 with adapter:                  49,522 (100.0%)\n",
      "  Read 2 with adapter:                  49,413 (99.7%)\n",
      "Pairs written (passing filters):        49,541 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,224,228 bp\n",
      "  Read 1:    14,613,753 bp\n",
      "  Read 2:    14,610,475 bp\n",
      "Total written (filtered):     27,295,226 bp (93.4%)\n",
      "  Read 1:    13,623,453 bp\n",
      "  Read 2:    13,671,773 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49522 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t290\t0.0\t1\t0 288 2\n",
      "20\t49084\t0.0\t2\t48277 798 9\n",
      "21\t94\t0.0\t2\t0 93 1\n",
      "22\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49413 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t205\t0.0\t1\t0 205\n",
      "19\t49148\t0.0\t1\t48329 819\n",
      "20\t60\t0.0\t1\t0 60\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s597_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s597_220_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s597_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s597_220_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s597_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s597_220_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s597_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s597_220_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,152 reads  @     62.0 µs/read;   0.97 M reads/minute\n",
      "Finished in 2.68 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,152\n",
      "  Read 1 with adapter:                  43,142 (100.0%)\n",
      "  Read 2 with adapter:                  43,032 (99.7%)\n",
      "Pairs written (passing filters):        43,152 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,453,990 bp\n",
      "  Read 1:    12,729,742 bp\n",
      "  Read 2:    12,724,248 bp\n",
      "Total written (filtered):     23,773,806 bp (93.4%)\n",
      "  Read 1:    11,867,049 bp\n",
      "  Read 2:    11,906,757 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43142 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t259\t0.0\t1\t0 257 2\n",
      "20\t42763\t0.0\t2\t42114 643 6\n",
      "21\t80\t0.0\t2\t0 78 2\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43032 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t205\t0.0\t1\t0 205\n",
      "19\t42739\t0.0\t1\t42064 675\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s598_85_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s598_221_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s598_85_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s598_221_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s598_85_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s598_221_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s598_85_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s598_221_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        47,654 reads  @     62.5 µs/read;   0.96 M reads/minute\n",
      "Finished in 2.99 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,654\n",
      "  Read 1 with adapter:                  47,635 (100.0%)\n",
      "  Read 2 with adapter:                  46,974 (98.6%)\n",
      "Pairs written (passing filters):        47,654 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,077,500 bp\n",
      "  Read 1:    14,057,569 bp\n",
      "  Read 2:    14,019,931 bp\n",
      "Total written (filtered):     26,232,597 bp (93.4%)\n",
      "  Read 1:    13,105,046 bp\n",
      "  Read 2:    13,127,551 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47635 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t299\t0.0\t1\t0 290 9\n",
      "20\t47197\t0.0\t2\t46394 795 8\n",
      "21\t92\t0.0\t2\t0 90 2\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46974 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t210\t0.0\t1\t0 210\n",
      "19\t46680\t0.0\t1\t45911 769\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s599_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s599_222_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s599_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s599_222_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s599_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s599_222_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s599_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s599_222_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        41,852 reads  @     60.7 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.54 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,852\n",
      "  Read 1 with adapter:                  41,832 (100.0%)\n",
      "  Read 2 with adapter:                  41,773 (99.8%)\n",
      "Pairs written (passing filters):        41,852 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,691,606 bp\n",
      "  Read 1:    12,345,519 bp\n",
      "  Read 2:    12,346,087 bp\n",
      "Total written (filtered):     23,061,542 bp (93.4%)\n",
      "  Read 1:    11,508,987 bp\n",
      "  Read 2:    11,552,555 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41832 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t214\t0.0\t1\t0 212 2\n",
      "20\t41496\t0.0\t2\t40719 774 3\n",
      "21\t82\t0.0\t2\t0 82\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41773 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t234\t0.0\t1\t0 234\n",
      "19\t41460\t0.0\t1\t40721 739\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s601_87_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s601_223_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s601_87_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s601_223_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s601_87_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s601_223_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s601_87_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s601_223_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        45,050 reads  @     64.5 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.91 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,050\n",
      "  Read 1 with adapter:                  45,040 (100.0%)\n",
      "  Read 2 with adapter:                  44,899 (99.7%)\n",
      "Pairs written (passing filters):        45,050 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,573,317 bp\n",
      "  Read 1:    13,289,470 bp\n",
      "  Read 2:    13,283,847 bp\n",
      "Total written (filtered):     24,819,696 bp (93.4%)\n",
      "  Read 1:    12,388,810 bp\n",
      "  Read 2:    12,430,886 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45040 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t240\t0.0\t1\t0 237 3\n",
      "20\t44709\t0.0\t2\t43976 727 6\n",
      "21\t62\t0.0\t2\t0 60 2\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44899 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t232\t0.0\t1\t0 232\n",
      "19\t44555\t0.0\t1\t43673 882\n",
      "20\t112\t0.0\t1\t0 112\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s602_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s602_224_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s602_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s602_224_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s602_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s602_224_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s602_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s602_224_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,874 reads  @     65.5 µs/read;   0.92 M reads/minute\n",
      "Finished in 3.07 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,874\n",
      "  Read 1 with adapter:                  46,859 (100.0%)\n",
      "  Read 2 with adapter:                  46,661 (99.5%)\n",
      "Pairs written (passing filters):        46,874 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,648,178 bp\n",
      "  Read 1:    13,827,280 bp\n",
      "  Read 2:    13,820,898 bp\n",
      "Total written (filtered):     25,824,781 bp (93.4%)\n",
      "  Read 1:    12,890,300 bp\n",
      "  Read 2:    12,934,481 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46859 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t281\t0.0\t1\t0 271 10\n",
      "20\t46460\t0.0\t2\t45561 895 4\n",
      "21\t75\t0.0\t2\t0 74 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46661 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t233\t0.0\t1\t0 233\n",
      "19\t46337\t0.0\t1\t45330 1007\n",
      "20\t91\t0.0\t1\t0 91\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s603_89_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s603_225_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s603_89_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s603_225_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s603_89_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s603_225_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s603_89_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s603_225_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,697 reads  @     61.7 µs/read;   0.97 M reads/minute\n",
      "Finished in 3.07 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,697\n",
      "  Read 1 with adapter:                  49,682 (100.0%)\n",
      "  Read 2 with adapter:                  49,507 (99.6%)\n",
      "Pairs written (passing filters):        49,697 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,321,113 bp\n",
      "  Read 1:    14,660,562 bp\n",
      "  Read 2:    14,660,551 bp\n",
      "Total written (filtered):     27,387,134 bp (93.4%)\n",
      "  Read 1:    13,667,082 bp\n",
      "  Read 2:    13,720,052 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49682 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t287\t0.0\t1\t0 284 3\n",
      "20\t49238\t0.0\t2\t48501 725 12\n",
      "21\t99\t0.0\t2\t0 98 1\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49507 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t230\t0.0\t1\t0 230\n",
      "19\t49181\t0.0\t1\t47980 1201\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s604_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s604_226_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s604_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s604_226_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s604_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s604_226_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s604_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s604_226_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        48,767 reads  @     60.9 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.97 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,767\n",
      "  Read 1 with adapter:                  48,742 (99.9%)\n",
      "  Read 2 with adapter:                  48,659 (99.8%)\n",
      "Pairs written (passing filters):        48,767 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,771,335 bp\n",
      "  Read 1:    14,385,707 bp\n",
      "  Read 2:    14,385,628 bp\n",
      "Total written (filtered):     26,872,261 bp (93.4%)\n",
      "  Read 1:    13,411,020 bp\n",
      "  Read 2:    13,461,241 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48742 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t295\t0.0\t1\t0 292 3\n",
      "20\t48301\t0.0\t2\t47495 794 12\n",
      "21\t94\t0.0\t2\t0 92 2\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48659 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t265\t0.0\t1\t0 265\n",
      "19\t48263\t0.0\t1\t47423 840\n",
      "20\t131\t0.0\t1\t0 131\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s605_91_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s605_227_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s605_91_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s605_227_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s605_91_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s605_227_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s605_91_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s605_227_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        48,588 reads  @     60.2 µs/read;   1.00 M reads/minute\n",
      "Finished in 2.93 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,588\n",
      "  Read 1 with adapter:                  48,572 (100.0%)\n",
      "  Read 2 with adapter:                  48,443 (99.7%)\n",
      "Pairs written (passing filters):        48,588 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,657,312 bp\n",
      "  Read 1:    14,333,445 bp\n",
      "  Read 2:    14,323,867 bp\n",
      "Total written (filtered):     26,765,801 bp (93.4%)\n",
      "  Read 1:    13,362,184 bp\n",
      "  Read 2:    13,403,617 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48572 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t298\t0.0\t1\t0 290 8\n",
      "20\t48148\t0.0\t2\t47380 763 5\n",
      "21\t85\t0.0\t2\t0 85\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48443 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t260\t0.0\t1\t0 260\n",
      "19\t48090\t0.0\t1\t47146 944\n",
      "20\t93\t0.0\t1\t0 93\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s606_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s606_228_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s606_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s606_228_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s606_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s606_228_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s606_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s606_228_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        48,945 reads  @     60.0 µs/read;   1.00 M reads/minute\n",
      "Finished in 2.94 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,945\n",
      "  Read 1 with adapter:                  48,931 (100.0%)\n",
      "  Read 2 with adapter:                  48,791 (99.7%)\n",
      "Pairs written (passing filters):        48,945 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,869,199 bp\n",
      "  Read 1:    14,438,756 bp\n",
      "  Read 2:    14,430,443 bp\n",
      "Total written (filtered):     26,963,910 bp (93.4%)\n",
      "  Read 1:    13,460,340 bp\n",
      "  Read 2:    13,503,570 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48931 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t311\t0.0\t1\t0 310 1\n",
      "20\t48491\t0.0\t2\t47786 698 7\n",
      "21\t91\t0.0\t2\t0 90 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48791 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t250\t0.0\t1\t0 250\n",
      "19\t48447\t0.0\t1\t47594 853\n",
      "20\t94\t0.0\t1\t0 94\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s607_93_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s607_229_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s607_93_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s607_229_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s607_93_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s607_229_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s607_93_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s607_229_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        49,878 reads  @     59.2 µs/read;   1.01 M reads/minute\n",
      "Finished in 2.96 s (59 us/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,878\n",
      "  Read 1 with adapter:                  49,858 (100.0%)\n",
      "  Read 2 with adapter:                  49,766 (99.8%)\n",
      "Pairs written (passing filters):        49,878 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,427,766 bp\n",
      "  Read 1:    14,713,824 bp\n",
      "  Read 2:    14,713,942 bp\n",
      "Total written (filtered):     27,485,364 bp (93.4%)\n",
      "  Read 1:    13,716,847 bp\n",
      "  Read 2:    13,768,517 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49858 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t293\t0.0\t1\t0 289 4\n",
      "20\t49429\t0.0\t2\t48691 730 8\n",
      "21\t86\t0.0\t2\t0 85 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49766 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t218\t0.0\t1\t0 218\n",
      "19\t49459\t0.0\t1\t48627 832\n",
      "20\t89\t0.0\t1\t0 89\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s608_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s608_230_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s608_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s608_230_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s608_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s608_230_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s608_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s608_230_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,795 reads  @     62.9 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.76 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,795\n",
      "  Read 1 with adapter:                  43,783 (100.0%)\n",
      "  Read 2 with adapter:                  43,691 (99.8%)\n",
      "Pairs written (passing filters):        43,795 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,838,808 bp\n",
      "  Read 1:    12,919,516 bp\n",
      "  Read 2:    12,919,292 bp\n",
      "Total written (filtered):     24,133,292 bp (93.4%)\n",
      "  Read 1:    12,044,046 bp\n",
      "  Read 2:    12,089,246 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43783 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t291\t0.0\t1\t0 285 6\n",
      "20\t43365\t0.0\t2\t42680 680 5\n",
      "21\t69\t0.0\t2\t0 69\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43691 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t215\t0.0\t1\t0 215\n",
      "19\t43344\t0.0\t1\t42558 786\n",
      "20\t132\t0.0\t1\t0 132\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s609_95_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s609_231_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s609_95_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s609_231_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s609_95_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s609_231_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s609_95_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s609_231_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,069 reads  @     61.1 µs/read;   0.98 M reads/minute\n",
      "Finished in 2.21 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,069\n",
      "  Read 1 with adapter:                  36,055 (100.0%)\n",
      "  Read 2 with adapter:                  35,987 (99.8%)\n",
      "Pairs written (passing filters):        36,069 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,274,816 bp\n",
      "  Read 1:    10,640,344 bp\n",
      "  Read 2:    10,634,472 bp\n",
      "Total written (filtered):     19,870,175 bp (93.4%)\n",
      "  Read 1:     9,919,343 bp\n",
      "  Read 2:     9,950,832 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36055 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t183\t0.0\t1\t0 183\n",
      "20\t35795\t0.0\t2\t35218 571 6\n",
      "21\t50\t0.0\t2\t0 50\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35987 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t203\t0.0\t1\t0 203\n",
      "19\t35694\t0.0\t1\t35125 569\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s610_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s610_232_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s610_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s610_232_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s610_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s610_232_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s610_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s610_232_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,808 reads  @     60.9 µs/read;   0.98 M reads/minute\n",
      "Finished in 2.13 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,808\n",
      "  Read 1 with adapter:                  34,798 (100.0%)\n",
      "  Read 2 with adapter:                  34,715 (99.7%)\n",
      "Pairs written (passing filters):        34,808 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,532,948 bp\n",
      "  Read 1:    10,268,352 bp\n",
      "  Read 2:    10,264,596 bp\n",
      "Total written (filtered):     19,177,630 bp (93.4%)\n",
      "  Read 1:     9,572,496 bp\n",
      "  Read 2:     9,605,134 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34798 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t184\t0.0\t1\t0 182 2\n",
      "20\t34523\t0.0\t2\t33901 617 5\n",
      "21\t54\t0.0\t2\t0 54\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34715 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t193\t0.0\t1\t0 193\n",
      "19\t34452\t0.0\t1\t33836 616\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s611_97_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s611_233_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s611_97_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s611_233_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s611_97_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s611_233_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s611_97_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s611_233_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        39,852 reads  @     59.7 µs/read;   1.01 M reads/minute\n",
      "Finished in 2.38 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,852\n",
      "  Read 1 with adapter:                  39,838 (100.0%)\n",
      "  Read 2 with adapter:                  39,776 (99.8%)\n",
      "Pairs written (passing filters):        39,852 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,512,641 bp\n",
      "  Read 1:    11,756,318 bp\n",
      "  Read 2:    11,756,323 bp\n",
      "Total written (filtered):     21,960,412 bp (93.4%)\n",
      "  Read 1:    10,959,701 bp\n",
      "  Read 2:    11,000,711 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39838 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t234\t0.0\t1\t0 232 2\n",
      "20\t39499\t0.0\t2\t38780 713 6\n",
      "21\t63\t0.0\t2\t0 62 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39776 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t191\t0.0\t1\t0 191\n",
      "19\t39526\t0.0\t1\t38828 698\n",
      "20\t59\t0.0\t1\t0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s612_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s612_234_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s612_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s612_234_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s612_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s612_234_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s612_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s612_234_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,166 reads  @     59.7 µs/read;   1.01 M reads/minute\n",
      "Finished in 2.22 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,166\n",
      "  Read 1 with adapter:                  37,149 (100.0%)\n",
      "  Read 2 with adapter:                  37,096 (99.8%)\n",
      "Pairs written (passing filters):        37,166 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,927,741 bp\n",
      "  Read 1:    10,963,960 bp\n",
      "  Read 2:    10,963,781 bp\n",
      "Total written (filtered):     20,480,096 bp (93.4%)\n",
      "  Read 1:    10,221,044 bp\n",
      "  Read 2:    10,259,052 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37149 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t198\t0.0\t1\t0 191 7\n",
      "20\t36823\t0.0\t2\t36231 582 10\n",
      "21\t86\t0.0\t2\t0 85 1\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37096 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t184\t0.0\t1\t0 184\n",
      "19\t36823\t0.0\t1\t36299 524\n",
      "20\t89\t0.0\t1\t0 89\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s613_99_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s613_235_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s613_99_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s613_235_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s613_99_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s613_235_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s613_99_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s613_235_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,828 reads  @     59.5 µs/read;   1.01 M reads/minute\n",
      "Finished in 2.19 s (60 us/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,828\n",
      "  Read 1 with adapter:                  36,821 (100.0%)\n",
      "  Read 2 with adapter:                  36,710 (99.7%)\n",
      "Pairs written (passing filters):        36,828 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,722,399 bp\n",
      "  Read 1:    10,864,246 bp\n",
      "  Read 2:    10,858,153 bp\n",
      "Total written (filtered):     20,288,772 bp (93.4%)\n",
      "  Read 1:    10,127,961 bp\n",
      "  Read 2:    10,160,811 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36821 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t236\t0.0\t1\t0 232 4\n",
      "20\t36475\t0.0\t2\t35896 571 8\n",
      "21\t75\t0.0\t2\t0 75\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36710 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t219\t0.0\t1\t0 219\n",
      "19\t36420\t0.0\t1\t35770 650\n",
      "20\t71\t0.0\t1\t0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s614_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s614_236_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s614_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s614_236_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s614_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s614_236_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s614_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s614_236_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,946 reads  @     59.0 µs/read;   1.02 M reads/minute\n",
      "Finished in 2.36 s (59 us/read; 1.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,946\n",
      "  Read 1 with adapter:                  39,930 (100.0%)\n",
      "  Read 2 with adapter:                  39,824 (99.7%)\n",
      "Pairs written (passing filters):        39,946 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,562,491 bp\n",
      "  Read 1:    11,784,062 bp\n",
      "  Read 2:    11,778,429 bp\n",
      "Total written (filtered):     22,007,510 bp (93.4%)\n",
      "  Read 1:    10,985,645 bp\n",
      "  Read 2:    11,021,865 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39930 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t270\t0.0\t1\t0 267 3\n",
      "20\t39556\t0.0\t2\t38967 587 2\n",
      "21\t69\t0.0\t2\t0 69\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39824 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t191\t0.0\t1\t0 191\n",
      "19\t39534\t0.0\t1\t38901 633\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s615_101_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s615_237_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s615_101_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s615_237_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s615_101_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s615_237_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s615_101_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s615_237_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        40,613 reads  @     59.4 µs/read;   1.01 M reads/minute\n",
      "Finished in 2.41 s (59 us/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,613\n",
      "  Read 1 with adapter:                  40,596 (100.0%)\n",
      "  Read 2 with adapter:                  40,501 (99.7%)\n",
      "Pairs written (passing filters):        40,613 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,961,642 bp\n",
      "  Read 1:    11,980,826 bp\n",
      "  Read 2:    11,980,816 bp\n",
      "Total written (filtered):     22,380,457 bp (93.4%)\n",
      "  Read 1:    11,169,058 bp\n",
      "  Read 2:    11,211,399 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40596 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t250\t0.0\t1\t0 246 4\n",
      "20\t40240\t0.0\t2\t39639 588 13\n",
      "21\t66\t0.0\t2\t0 64 2\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40501 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t187\t0.0\t1\t0 187\n",
      "19\t40229\t0.0\t1\t39544 685\n",
      "20\t85\t0.0\t1\t0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s616_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s616_238_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s616_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s616_238_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s616_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s616_238_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s616_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s616_238_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,665 reads  @     59.7 µs/read;   1.01 M reads/minute\n",
      "Finished in 2.07 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,665\n",
      "  Read 1 with adapter:                  34,656 (100.0%)\n",
      "  Read 2 with adapter:                  34,572 (99.7%)\n",
      "Pairs written (passing filters):        34,665 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,452,321 bp\n",
      "  Read 1:    10,226,167 bp\n",
      "  Read 2:    10,226,154 bp\n",
      "Total written (filtered):     19,102,546 bp (93.4%)\n",
      "  Read 1:     9,533,174 bp\n",
      "  Read 2:     9,569,372 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34656 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t185\t0.0\t1\t0 183 2\n",
      "20\t34381\t0.0\t2\t33821 554 6\n",
      "21\t50\t0.0\t2\t0 50\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34572 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t189\t0.0\t1\t0 189\n",
      "19\t34280\t0.0\t1\t33680 600\n",
      "20\t103\t0.0\t1\t0 103\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s617_103_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s617_239_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s617_103_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s617_239_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s617_103_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s617_239_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s617_103_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s617_239_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        43,773 reads  @     60.4 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.65 s (60 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,773\n",
      "  Read 1 with adapter:                  43,760 (100.0%)\n",
      "  Read 2 with adapter:                  43,658 (99.7%)\n",
      "Pairs written (passing filters):        43,773 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,818,843 bp\n",
      "  Read 1:    12,912,802 bp\n",
      "  Read 2:    12,906,041 bp\n",
      "Total written (filtered):     24,114,367 bp (93.4%)\n",
      "  Read 1:    12,037,709 bp\n",
      "  Read 2:    12,076,658 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43760 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t208\t0.0\t1\t0 200 8\n",
      "20\t43431\t0.0\t2\t42701 725 5\n",
      "21\t77\t0.0\t2\t0 76 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43658 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t235\t0.0\t1\t0 235\n",
      "19\t43307\t0.0\t1\t42529 778\n",
      "20\t116\t0.0\t1\t0 116\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s618_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s618_240_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s618_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s618_240_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s618_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s618_240_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s618_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s618_240_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        44,187 reads  @     61.5 µs/read;   0.98 M reads/minute\n",
      "Finished in 2.72 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,187\n",
      "  Read 1 with adapter:                  44,171 (100.0%)\n",
      "  Read 2 with adapter:                  44,030 (99.6%)\n",
      "Pairs written (passing filters):        44,187 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,064,335 bp\n",
      "  Read 1:    13,035,152 bp\n",
      "  Read 2:    13,029,183 bp\n",
      "Total written (filtered):     24,344,704 bp (93.4%)\n",
      "  Read 1:    12,151,938 bp\n",
      "  Read 2:    12,192,766 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44171 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t280\t0.0\t1\t0 274 6\n",
      "20\t43784\t0.0\t2\t43032 745 7\n",
      "21\t72\t0.0\t2\t0 68 4\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44030 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t223\t0.0\t1\t0 223\n",
      "19\t43737\t0.0\t1\t42927 810\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s619_105_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s619_241_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s619_105_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s619_241_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s619_105_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s619_241_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s619_105_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s619_241_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,679 reads  @     60.1 µs/read;   1.00 M reads/minute\n",
      "Finished in 3.11 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,679\n",
      "  Read 1 with adapter:                  51,662 (100.0%)\n",
      "  Read 2 with adapter:                  51,547 (99.7%)\n",
      "Pairs written (passing filters):        51,679 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,490,564 bp\n",
      "  Read 1:    15,245,283 bp\n",
      "  Read 2:    15,245,281 bp\n",
      "Total written (filtered):     28,478,236 bp (93.4%)\n",
      "  Read 1:    14,212,219 bp\n",
      "  Read 2:    14,266,017 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51662 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t300\t0.0\t1\t0 297 3\n",
      "20\t51198\t0.0\t2\t50359 831 8\n",
      "21\t112\t0.0\t2\t0 110 2\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51547 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t219\t0.0\t1\t0 219\n",
      "19\t51238\t0.0\t1\t50312 926\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s620_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s620_242_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s620_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s620_242_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s620_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s620_242_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s620_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s620_242_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        49,082 reads  @     60.3 µs/read;   1.00 M reads/minute\n",
      "Finished in 2.96 s (60 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,082\n",
      "  Read 1 with adapter:                  49,058 (100.0%)\n",
      "  Read 2 with adapter:                  48,943 (99.7%)\n",
      "Pairs written (passing filters):        49,082 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,957,872 bp\n",
      "  Read 1:    14,478,997 bp\n",
      "  Read 2:    14,478,875 bp\n",
      "Total written (filtered):     27,047,104 bp (93.4%)\n",
      "  Read 1:    13,498,003 bp\n",
      "  Read 2:    13,549,101 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49058 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t304\t0.0\t1\t0 295 9\n",
      "20\t48591\t0.0\t2\t47700 884 7\n",
      "21\t96\t0.0\t2\t0 96\n",
      "22\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48943 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t284\t0.0\t1\t0 284\n",
      "19\t48518\t0.0\t1\t47687 831\n",
      "20\t141\t0.0\t1\t0 141\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s621_107_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s621_243_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s621_107_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s621_243_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s621_107_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s621_243_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s621_107_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s621_243_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        47,801 reads  @     59.5 µs/read;   1.01 M reads/minute\n",
      "Finished in 2.85 s (60 us/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,801\n",
      "  Read 1 with adapter:                  47,782 (100.0%)\n",
      "  Read 2 with adapter:                  47,627 (99.6%)\n",
      "Pairs written (passing filters):        47,801 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,193,511 bp\n",
      "  Read 1:    14,101,265 bp\n",
      "  Read 2:    14,092,246 bp\n",
      "Total written (filtered):     26,333,233 bp (93.4%)\n",
      "  Read 1:    13,145,765 bp\n",
      "  Read 2:    13,187,468 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47782 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t259\t0.0\t1\t0 252 7\n",
      "20\t47378\t0.0\t2\t46608 761 9\n",
      "21\t111\t0.0\t2\t0 111\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47627 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t234\t0.0\t1\t0 234\n",
      "19\t47294\t0.0\t1\t46395 899\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s622_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s622_244_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s622_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s622_244_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s622_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s622_244_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s622_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s622_244_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        45,295 reads  @     58.6 µs/read;   1.02 M reads/minute\n",
      "Finished in 2.66 s (59 us/read; 1.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,295\n",
      "  Read 1 with adapter:                  45,280 (100.0%)\n",
      "  Read 2 with adapter:                  45,158 (99.7%)\n",
      "Pairs written (passing filters):        45,295 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,717,841 bp\n",
      "  Read 1:    13,362,012 bp\n",
      "  Read 2:    13,355,829 bp\n",
      "Total written (filtered):     24,954,516 bp (93.4%)\n",
      "  Read 1:    12,456,525 bp\n",
      "  Read 2:    12,497,991 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45280 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t249\t0.0\t1\t0 244 5\n",
      "20\t44900\t0.0\t2\t44202 690 8\n",
      "21\t102\t0.0\t2\t0 99 3\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45158 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t251\t0.0\t1\t0 251\n",
      "19\t44820\t0.0\t1\t44084 736\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s623_109_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s623_245_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s623_109_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s623_245_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s623_109_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s623_245_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s623_109_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s623_245_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        52,474 reads  @     64.6 µs/read;   0.93 M reads/minute\n",
      "Finished in 3.40 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,474\n",
      "  Read 1 with adapter:                  52,445 (99.9%)\n",
      "  Read 2 with adapter:                  52,337 (99.7%)\n",
      "Pairs written (passing filters):        52,474 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,957,643 bp\n",
      "  Read 1:    15,477,958 bp\n",
      "  Read 2:    15,479,685 bp\n",
      "Total written (filtered):     28,914,589 bp (93.4%)\n",
      "  Read 1:    14,429,180 bp\n",
      "  Read 2:    14,485,409 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52445 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t297\t0.0\t1\t0 291 6\n",
      "20\t51968\t0.0\t2\t51134 818 16\n",
      "21\t129\t0.0\t2\t0 129\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52337 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t228\t0.0\t1\t0 228\n",
      "19\t52008\t0.0\t1\t51068 940\n",
      "20\t101\t0.0\t1\t0 101\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s624_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s624_246_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s624_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s624_246_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s624_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s624_246_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s624_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s624_246_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,165 reads  @     67.4 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.25 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,165\n",
      "  Read 1 with adapter:                  48,135 (99.9%)\n",
      "  Read 2 with adapter:                  48,033 (99.7%)\n",
      "Pairs written (passing filters):        48,165 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,416,128 bp\n",
      "  Read 1:    14,207,790 bp\n",
      "  Read 2:    14,208,338 bp\n",
      "Total written (filtered):     26,541,042 bp (93.4%)\n",
      "  Read 1:    13,245,184 bp\n",
      "  Read 2:    13,295,858 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48135 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t272\t0.0\t1\t0 271 1\n",
      "20\t47679\t0.0\t2\t46884 790 5\n",
      "21\t134\t0.0\t2\t0 134\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48033 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t274\t0.0\t1\t0 274\n",
      "19\t47632\t0.0\t1\t46803 829\n",
      "20\t127\t0.0\t1\t0 127\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s625_111_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s625_247_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s625_111_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s625_247_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s625_111_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s625_247_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s625_111_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s625_247_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,329 reads  @     66.3 µs/read;   0.91 M reads/minute\n",
      "Finished in 3.07 s (66 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,329\n",
      "  Read 1 with adapter:                  46,308 (100.0%)\n",
      "  Read 2 with adapter:                  46,197 (99.7%)\n",
      "Pairs written (passing filters):        46,329 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,329,125 bp\n",
      "  Read 1:    13,667,048 bp\n",
      "  Read 2:    13,662,077 bp\n",
      "Total written (filtered):     25,525,489 bp (93.4%)\n",
      "  Read 1:    12,741,028 bp\n",
      "  Read 2:    12,784,461 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46308 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t264\t0.0\t1\t0 259 5\n",
      "20\t45914\t0.0\t2\t45123 785 6\n",
      "21\t80\t0.0\t2\t0 78 2\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46197 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t243\t0.0\t1\t0 243\n",
      "19\t45838\t0.0\t1\t45104 734\n",
      "20\t116\t0.0\t1\t0 116\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s626_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s626_248_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s626_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s626_248_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s626_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s626_248_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s626_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s626_248_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,010 reads  @     63.3 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.73 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,010\n",
      "  Read 1 with adapter:                  42,990 (100.0%)\n",
      "  Read 2 with adapter:                  42,896 (99.7%)\n",
      "Pairs written (passing filters):        43,010 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,372,686 bp\n",
      "  Read 1:    12,687,929 bp\n",
      "  Read 2:    12,684,757 bp\n",
      "Total written (filtered):     23,698,122 bp (93.4%)\n",
      "  Read 1:    11,828,237 bp\n",
      "  Read 2:    11,869,885 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42990 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t250\t0.0\t1\t0 246 4\n",
      "20\t42582\t0.0\t2\t41839 734 9\n",
      "21\t110\t0.0\t2\t0 109 1\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42896 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t234\t0.0\t1\t0 234\n",
      "19\t42580\t0.0\t1\t41845 735\n",
      "20\t82\t0.0\t1\t0 82\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s627_113_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s627_249_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s627_113_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s627_249_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s627_113_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s627_249_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s627_113_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s627_249_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,210 reads  @     65.8 µs/read;   0.91 M reads/minute\n",
      "Finished in 3.11 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,210\n",
      "  Read 1 with adapter:                  47,187 (100.0%)\n",
      "  Read 2 with adapter:                  47,124 (99.8%)\n",
      "Pairs written (passing filters):        47,210 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,853,860 bp\n",
      "  Read 1:    13,926,935 bp\n",
      "  Read 2:    13,926,925 bp\n",
      "Total written (filtered):     26,015,031 bp (93.4%)\n",
      "  Read 1:    12,983,360 bp\n",
      "  Read 2:    13,031,671 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47187 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t301\t0.0\t1\t0 296 5\n",
      "20\t46733\t0.0\t2\t45939 784 10\n",
      "21\t98\t0.0\t2\t0 96 2\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47124 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t186\t0.0\t1\t0 186\n",
      "19\t46854\t0.0\t1\t46085 769\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s628_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s628_250_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s628_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s628_250_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s628_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s628_250_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s628_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s628_250_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,384 reads  @     68.4 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.45 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,384\n",
      "  Read 1 with adapter:                  50,346 (99.9%)\n",
      "  Read 2 with adapter:                  50,278 (99.8%)\n",
      "Pairs written (passing filters):        50,384 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,725,694 bp\n",
      "  Read 1:    14,862,844 bp\n",
      "  Read 2:    14,862,850 bp\n",
      "Total written (filtered):     27,763,857 bp (93.4%)\n",
      "  Read 1:    13,856,140 bp\n",
      "  Read 2:    13,907,717 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50346 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t359\t0.0\t1\t0 351 8\n",
      "20\t49820\t0.0\t2\t48882 923 15\n",
      "21\t103\t0.0\t2\t0 102 1\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50278 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t269\t0.0\t1\t0 269\n",
      "19\t49889\t0.0\t1\t49066 823\n",
      "20\t120\t0.0\t1\t0 120\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s629_115_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s629_251_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s629_115_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s629_251_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s629_115_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s629_251_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s629_115_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s629_251_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        43,063 reads  @     69.1 µs/read;   0.87 M reads/minute\n",
      "Finished in 2.98 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,063\n",
      "  Read 1 with adapter:                  43,049 (100.0%)\n",
      "  Read 2 with adapter:                  42,958 (99.8%)\n",
      "Pairs written (passing filters):        43,063 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,404,310 bp\n",
      "  Read 1:    12,703,340 bp\n",
      "  Read 2:    12,700,970 bp\n",
      "Total written (filtered):     23,727,578 bp (93.4%)\n",
      "  Read 1:    11,842,677 bp\n",
      "  Read 2:    11,884,901 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43049 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t402\t0.0\t1\t0 394 8\n",
      "20\t42532\t0.0\t2\t41810 712 10\n",
      "21\t77\t0.0\t2\t0 76 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42958 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t238\t0.0\t1\t0 238\n",
      "19\t42615\t0.0\t1\t41933 682\n",
      "20\t105\t0.0\t1\t0 105\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s630_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s630_252_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s630_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s630_252_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s630_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s630_252_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s630_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s630_252_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,590 reads  @     66.2 µs/read;   0.91 M reads/minute\n",
      "Finished in 3.22 s (66 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,590\n",
      "  Read 1 with adapter:                  48,571 (100.0%)\n",
      "  Read 2 with adapter:                  48,464 (99.7%)\n",
      "Pairs written (passing filters):        48,590 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,664,407 bp\n",
      "  Read 1:    14,333,793 bp\n",
      "  Read 2:    14,330,614 bp\n",
      "Total written (filtered):     26,772,638 bp (93.4%)\n",
      "  Read 1:    13,362,704 bp\n",
      "  Read 2:    13,409,934 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48571 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t491\t0.0\t1\t0 484 7\n",
      "20\t47912\t0.0\t2\t47113 791 8\n",
      "21\t108\t0.0\t2\t0 106 2\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48464 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t232\t0.0\t1\t0 232\n",
      "19\t48136\t0.0\t1\t47356 780\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s631_117_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s631_253_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s631_117_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s631_253_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s631_117_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s631_253_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s631_117_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s631_253_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,528 reads  @     70.8 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.37 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,528\n",
      "  Read 1 with adapter:                  47,506 (100.0%)\n",
      "  Read 2 with adapter:                  47,448 (99.8%)\n",
      "Pairs written (passing filters):        47,528 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,041,198 bp\n",
      "  Read 1:    14,020,505 bp\n",
      "  Read 2:    14,020,693 bp\n",
      "Total written (filtered):     26,190,014 bp (93.4%)\n",
      "  Read 1:    13,070,692 bp\n",
      "  Read 2:    13,119,322 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47506 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t431\t0.0\t1\t0 426 5\n",
      "20\t46930\t0.0\t2\t46149 773 8\n",
      "21\t102\t0.0\t2\t0 102\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47448 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t229\t0.0\t1\t0 229\n",
      "19\t47131\t0.0\t1\t46397 734\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s632_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s632_254_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s632_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s632_254_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s632_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s632_254_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s632_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s632_254_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,692 reads  @     67.4 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.61 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,692\n",
      "  Read 1 with adapter:                  38,680 (100.0%)\n",
      "  Read 2 with adapter:                  38,612 (99.8%)\n",
      "Pairs written (passing filters):        38,692 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,828,025 bp\n",
      "  Read 1:    11,414,125 bp\n",
      "  Read 2:    11,413,900 bp\n",
      "Total written (filtered):     21,321,197 bp (93.4%)\n",
      "  Read 1:    10,640,821 bp\n",
      "  Read 2:    10,680,376 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38680 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t372\t0.0\t1\t0 362 10\n",
      "20\t38213\t0.0\t2\t37602 603 8\n",
      "21\t62\t0.0\t2\t0 62\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38612 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t203\t0.0\t1\t0 203\n",
      "19\t38310\t0.0\t1\t37734 576\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s633_119_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s633_255_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s633_119_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s633_255_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s633_119_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s633_255_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s633_119_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s633_255_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,505 reads  @     68.1 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.56 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,505\n",
      "  Read 1 with adapter:                  37,485 (99.9%)\n",
      "  Read 2 with adapter:                  37,400 (99.7%)\n",
      "Pairs written (passing filters):        37,505 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,123,730 bp\n",
      "  Read 1:    11,063,684 bp\n",
      "  Read 2:    11,060,046 bp\n",
      "Total written (filtered):     20,663,684 bp (93.4%)\n",
      "  Read 1:    10,314,137 bp\n",
      "  Read 2:    10,349,547 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37485 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t219\t0.0\t1\t0 213 6\n",
      "20\t37171\t0.0\t2\t36479 686 6\n",
      "21\t68\t0.0\t2\t0 68\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37400 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t191\t0.0\t1\t0 191\n",
      "19\t37119\t0.0\t1\t36522 597\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s634_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s634_256_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s634_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s634_256_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s634_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s634_256_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s634_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s634_256_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,058 reads  @     69.8 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.08 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,058\n",
      "  Read 1 with adapter:                  44,036 (100.0%)\n",
      "  Read 2 with adapter:                  43,903 (99.6%)\n",
      "Pairs written (passing filters):        44,058 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,986,020 bp\n",
      "  Read 1:    12,995,833 bp\n",
      "  Read 2:    12,990,187 bp\n",
      "Total written (filtered):     24,271,611 bp (93.4%)\n",
      "  Read 1:    12,115,449 bp\n",
      "  Read 2:    12,156,162 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44036 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t75\t0.0\t1\t0 0 75\n",
      "19\t311\t0.0\t1\t0 306 5\n",
      "20\t43544\t0.0\t2\t42751 786 7\n",
      "21\t87\t0.0\t2\t0 85 2\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43903 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t223\t0.0\t1\t0 223\n",
      "19\t43589\t0.0\t1\t42758 831\n",
      "20\t91\t0.0\t1\t0 91\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s635_121_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s635_257_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s635_121_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s635_257_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s635_121_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s635_257_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s635_121_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s635_257_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        46,428 reads  @     67.6 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.15 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,428\n",
      "  Read 1 with adapter:                  46,401 (99.9%)\n",
      "  Read 2 with adapter:                  46,334 (99.8%)\n",
      "Pairs written (passing filters):        46,428 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,392,022 bp\n",
      "  Read 1:    13,695,787 bp\n",
      "  Read 2:    13,696,235 bp\n",
      "Total written (filtered):     25,584,034 bp (93.4%)\n",
      "  Read 1:    12,768,013 bp\n",
      "  Read 2:    12,816,021 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46401 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t350\t0.0\t1\t0 343 7\n",
      "20\t45913\t0.0\t2\t45095 808 10\n",
      "21\t84\t0.0\t2\t0 83 1\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46334 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t229\t0.0\t1\t0 229\n",
      "19\t46008\t0.0\t1\t45239 769\n",
      "20\t97\t0.0\t1\t0 97\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s636_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s636_258_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s636_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s636_258_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s636_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s636_258_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s636_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s636_258_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,154 reads  @     65.3 µs/read;   0.92 M reads/minute\n",
      "Finished in 3.54 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,154\n",
      "  Read 1 with adapter:                  54,121 (99.9%)\n",
      "  Read 2 with adapter:                  54,007 (99.7%)\n",
      "Pairs written (passing filters):        54,154 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,949,675 bp\n",
      "  Read 1:    15,975,320 bp\n",
      "  Read 2:    15,974,355 bp\n",
      "Total written (filtered):     29,841,508 bp (93.4%)\n",
      "  Read 1:    14,893,120 bp\n",
      "  Read 2:    14,948,388 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54121 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t344\t0.0\t1\t0 339 5\n",
      "20\t53619\t0.0\t2\t52659 948 12\n",
      "21\t112\t0.0\t2\t0 109 3\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54007 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t304\t0.0\t1\t0 304\n",
      "19\t53565\t0.0\t1\t52689 876\n",
      "20\t138\t0.0\t1\t0 138\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s637_123_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s637_259_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s637_123_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s637_259_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s637_123_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s637_259_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s637_123_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s637_259_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        44,547 reads  @     66.8 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.98 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,547\n",
      "  Read 1 with adapter:                  44,535 (100.0%)\n",
      "  Read 2 with adapter:                  44,358 (99.6%)\n",
      "Pairs written (passing filters):        44,547 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,276,144 bp\n",
      "  Read 1:    13,140,465 bp\n",
      "  Read 2:    13,135,679 bp\n",
      "Total written (filtered):     24,542,965 bp (93.4%)\n",
      "  Read 1:    12,249,943 bp\n",
      "  Read 2:    12,293,022 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44535 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t282\t0.0\t1\t0 278 4\n",
      "20\t44141\t0.0\t2\t43429 700 12\n",
      "21\t64\t0.0\t2\t0 63 1\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44358 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t255\t0.0\t1\t0 255\n",
      "19\t43993\t0.0\t1\t43165 828\n",
      "20\t110\t0.0\t1\t0 110\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s638_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s638_260_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s638_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s638_260_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s638_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s638_260_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s638_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s638_260_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,206 reads  @     69.0 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.33 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,206\n",
      "  Read 1 with adapter:                  48,191 (100.0%)\n",
      "  Read 2 with adapter:                  48,075 (99.7%)\n",
      "Pairs written (passing filters):        48,206 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,436,376 bp\n",
      "  Read 1:    14,220,445 bp\n",
      "  Read 2:    14,215,931 bp\n",
      "Total written (filtered):     26,559,458 bp (93.4%)\n",
      "  Read 1:    13,256,808 bp\n",
      "  Read 2:    13,302,650 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48191 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t301\t0.0\t1\t0 295 6\n",
      "20\t47747\t0.0\t2\t46929 814 4\n",
      "21\t84\t0.0\t2\t0 79 5\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48075 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t262\t0.0\t1\t0 262\n",
      "19\t47695\t0.0\t1\t46925 770\n",
      "20\t118\t0.0\t1\t0 118\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s639_125_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s639_261_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s639_125_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s639_261_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s639_125_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s639_261_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s639_125_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s639_261_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         9,323 reads  @     71.8 µs/read;   0.84 M reads/minute\n",
      "Finished in 0.67 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              9,323\n",
      "  Read 1 with adapter:                   9,318 (99.9%)\n",
      "  Read 2 with adapter:                   9,302 (99.8%)\n",
      "Pairs written (passing filters):         9,323 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,500,138 bp\n",
      "  Read 1:     2,749,855 bp\n",
      "  Read 2:     2,750,283 bp\n",
      "Total written (filtered):      5,137,076 bp (93.4%)\n",
      "  Read 1:     2,563,492 bp\n",
      "  Read 2:     2,573,584 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 9318 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t45\t0.0\t1\t0 43 2\n",
      "20\t9233\t0.0\t2\t9006 225 2\n",
      "21\t20\t0.0\t2\t0 18 2\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 9302 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t63\t0.0\t1\t0 63\n",
      "19\t9215\t0.0\t1\t9013 202\n",
      "20\t24\t0.0\t1\t0 24\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s640_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s640_262_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s640_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s640_262_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s640_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s640_262_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s640_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s640_262_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        41,793 reads  @     61.1 µs/read;   0.98 M reads/minute\n",
      "Finished in 2.56 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,793\n",
      "  Read 1 with adapter:                  41,771 (99.9%)\n",
      "  Read 2 with adapter:                  41,707 (99.8%)\n",
      "Pairs written (passing filters):        41,793 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,657,644 bp\n",
      "  Read 1:    12,328,736 bp\n",
      "  Read 2:    12,328,908 bp\n",
      "Total written (filtered):     23,030,038 bp (93.4%)\n",
      "  Read 1:    11,493,461 bp\n",
      "  Read 2:    11,536,577 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41771 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t270\t0.0\t1\t0 264 6\n",
      "20\t41376\t0.0\t2\t40654 716 6\n",
      "21\t65\t0.0\t2\t0 63 2\n",
      "22\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41707 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t221\t0.0\t1\t0 221\n",
      "19\t41367\t0.0\t1\t40720 647\n",
      "20\t119\t0.0\t1\t0 119\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s641_127_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s641_263_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s641_127_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s641_263_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s641_127_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s641_263_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s641_127_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s641_263_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        36,706 reads  @     60.9 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.24 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,706\n",
      "  Read 1 with adapter:                  36,693 (100.0%)\n",
      "  Read 2 with adapter:                  36,605 (99.7%)\n",
      "Pairs written (passing filters):        36,706 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,650,521 bp\n",
      "  Read 1:    10,828,124 bp\n",
      "  Read 2:    10,822,397 bp\n",
      "Total written (filtered):     20,221,395 bp (93.4%)\n",
      "  Read 1:    10,094,401 bp\n",
      "  Read 2:    10,126,994 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36693 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t199\t0.0\t1\t0 196 3\n",
      "20\t36403\t0.0\t2\t35829 564 10\n",
      "21\t64\t0.0\t2\t0 63 1\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36605 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t172\t0.0\t1\t0 172\n",
      "19\t36353\t0.0\t1\t35620 733\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s642_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s642_264_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s642_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s642_264_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s642_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s642_264_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s642_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s642_264_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,912 reads  @     61.0 µs/read;   0.98 M reads/minute\n",
      "Finished in 2.38 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,912\n",
      "  Read 1 with adapter:                  38,898 (100.0%)\n",
      "  Read 2 with adapter:                  38,791 (99.7%)\n",
      "Pairs written (passing filters):        38,912 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,953,164 bp\n",
      "  Read 1:    11,479,027 bp\n",
      "  Read 2:    11,474,137 bp\n",
      "Total written (filtered):     21,438,475 bp (93.4%)\n",
      "  Read 1:    10,701,232 bp\n",
      "  Read 2:    10,737,243 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38898 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t247\t0.0\t1\t0 245 2\n",
      "20\t38538\t0.0\t2\t37888 648 2\n",
      "21\t72\t0.0\t2\t0 70 2\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38791 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t197\t0.0\t1\t0 197\n",
      "19\t38532\t0.0\t1\t37777 755\n",
      "20\t62\t0.0\t1\t0 62\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s643_129_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s643_265_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s643_129_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s643_265_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s643_129_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s643_265_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s643_129_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s643_265_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        41,509 reads  @     63.7 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.65 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,509\n",
      "  Read 1 with adapter:                  41,489 (100.0%)\n",
      "  Read 2 with adapter:                  41,322 (99.5%)\n",
      "Pairs written (passing filters):        41,509 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,489,761 bp\n",
      "  Read 1:    12,244,902 bp\n",
      "  Read 2:    12,244,859 bp\n",
      "Total written (filtered):     22,875,141 bp (93.4%)\n",
      "  Read 1:    11,415,288 bp\n",
      "  Read 2:    11,459,853 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41489 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t279\t0.0\t1\t0 272 7\n",
      "20\t41088\t0.0\t2\t40349 727 12\n",
      "21\t83\t0.0\t2\t0 81 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41322 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t184\t0.0\t1\t0 184\n",
      "19\t41066\t0.0\t1\t40083 983\n",
      "20\t72\t0.0\t1\t0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s644_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s644_266_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s644_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s644_266_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s644_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s644_266_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s644_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s644_266_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,542 reads  @     61.7 µs/read;   0.97 M reads/minute\n",
      "Finished in 2.44 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,542\n",
      "  Read 1 with adapter:                  39,528 (100.0%)\n",
      "  Read 2 with adapter:                  39,457 (99.8%)\n",
      "Pairs written (passing filters):        39,542 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,329,141 bp\n",
      "  Read 1:    11,664,643 bp\n",
      "  Read 2:    11,664,498 bp\n",
      "Total written (filtered):     21,789,087 bp (93.4%)\n",
      "  Read 1:    10,874,186 bp\n",
      "  Read 2:    10,914,901 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39528 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t226\t0.0\t1\t0 224 2\n",
      "20\t39178\t0.0\t2\t38531 642 5\n",
      "21\t81\t0.0\t2\t0 81\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39457 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t194\t0.0\t1\t0 194\n",
      "19\t39155\t0.0\t1\t38462 693\n",
      "20\t108\t0.0\t1\t0 108\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s645_131_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s645_267_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s645_131_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s645_267_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s645_131_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s645_267_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s645_131_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s645_267_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,214 reads  @     61.4 µs/read;   0.98 M reads/minute\n",
      "Finished in 2.29 s (61 us/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,214\n",
      "  Read 1 with adapter:                  37,205 (100.0%)\n",
      "  Read 2 with adapter:                  37,098 (99.7%)\n",
      "Pairs written (passing filters):        37,214 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,950,787 bp\n",
      "  Read 1:    10,977,891 bp\n",
      "  Read 2:    10,972,896 bp\n",
      "Total written (filtered):     20,502,052 bp (93.4%)\n",
      "  Read 1:    10,233,938 bp\n",
      "  Read 2:    10,268,114 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37205 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t225\t0.0\t1\t0 222 3\n",
      "20\t36888\t0.0\t2\t36297 588 3\n",
      "21\t78\t0.0\t2\t0 75 3\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37098 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t177\t0.0\t1\t0 177\n",
      "19\t36824\t0.0\t1\t36076 748\n",
      "20\t97\t0.0\t1\t0 97\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s646_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s646_268_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s646_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s646_268_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s646_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s646_268_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s646_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s646_268_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        40,033 reads  @     59.9 µs/read;   1.00 M reads/minute\n",
      "Finished in 2.40 s (60 us/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,033\n",
      "  Read 1 with adapter:                  40,019 (100.0%)\n",
      "  Read 2 with adapter:                  39,926 (99.7%)\n",
      "Pairs written (passing filters):        40,033 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,614,345 bp\n",
      "  Read 1:    11,809,053 bp\n",
      "  Read 2:    11,805,292 bp\n",
      "Total written (filtered):     22,055,607 bp (93.4%)\n",
      "  Read 1:    11,008,786 bp\n",
      "  Read 2:    11,046,821 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40019 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t218\t0.0\t1\t0 213 5\n",
      "20\t39706\t0.0\t2\t39058 643 5\n",
      "21\t65\t0.0\t2\t0 64 1\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39926 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t210\t0.0\t1\t0 210\n",
      "19\t39629\t0.0\t1\t38918 711\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s647_133_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s647_269_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s647_133_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s647_269_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s647_133_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s647_269_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s647_133_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s647_269_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        42,040 reads  @     60.5 µs/read;   0.99 M reads/minute\n",
      "Finished in 2.55 s (61 us/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,040\n",
      "  Read 1 with adapter:                  42,020 (100.0%)\n",
      "  Read 2 with adapter:                  41,953 (99.8%)\n",
      "Pairs written (passing filters):        42,040 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,802,963 bp\n",
      "  Read 1:    12,401,229 bp\n",
      "  Read 2:    12,401,734 bp\n",
      "Total written (filtered):     23,165,720 bp (93.4%)\n",
      "  Read 1:    11,560,981 bp\n",
      "  Read 2:    11,604,739 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42020 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t253\t0.0\t1\t0 250 3\n",
      "20\t41647\t0.0\t2\t40927 714 6\n",
      "21\t87\t0.0\t2\t0 86 1\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41953 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t204\t0.0\t1\t0 204\n",
      "19\t41657\t0.0\t1\t40877 780\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s648_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s648_270_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s648_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s648_270_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s648_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s648_270_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s648_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s648_270_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,871 reads  @     62.1 µs/read;   0.97 M reads/minute\n",
      "Finished in 2.36 s (62 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,871\n",
      "  Read 1 with adapter:                  37,859 (100.0%)\n",
      "  Read 2 with adapter:                  37,792 (99.8%)\n",
      "Pairs written (passing filters):        37,871 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,343,525 bp\n",
      "  Read 1:    11,171,632 bp\n",
      "  Read 2:    11,171,893 bp\n",
      "Total written (filtered):     20,868,523 bp (93.4%)\n",
      "  Read 1:    10,414,604 bp\n",
      "  Read 2:    10,453,919 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37859 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t239\t0.0\t1\t0 237 2\n",
      "20\t37523\t0.0\t2\t36863 648 12\n",
      "21\t63\t0.0\t2\t0 63\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37792 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t174\t0.0\t1\t0 174\n",
      "19\t37518\t0.0\t1\t36816 702\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s649_135_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s649_271_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s649_135_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s649_271_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s649_135_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l8jn7zis/s649_271_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s649_135_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-p3piyvk_/24eb81f5-6474-4a9a-a1d7-ce096d051b54/data/s649_271_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        27,986 reads  @     62.7 µs/read;   0.96 M reads/minute\n",
      "Finished in 1.76 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,986\n",
      "  Read 1 with adapter:                  27,970 (99.9%)\n",
      "  Read 2 with adapter:                  27,930 (99.8%)\n",
      "Pairs written (passing filters):        27,986 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,511,482 bp\n",
      "  Read 1:     8,255,665 bp\n",
      "  Read 2:     8,255,817 bp\n",
      "Total written (filtered):     15,421,652 bp (93.4%)\n",
      "  Read 1:     7,696,359 bp\n",
      "  Read 2:     7,725,293 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 27970 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t170\t0.0\t1\t0 166 4\n",
      "20\t27713\t0.0\t2\t27179 529 5\n",
      "21\t66\t0.0\t2\t0 66\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 27930 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t198\t0.0\t1\t0 198\n",
      "19\t27680\t0.0\t1\t27169 511\n",
      "20\t52\t0.0\t1\t0 52\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: 18S-demux-pairedend-trimmed3.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences 18S-paired-end-demux_r1.qza \\\n",
    "    --p-front-f '^AGGGCAAKYCTGGTGCCAGC' \\\n",
    "    --p-front-r '^GRCGGTATCTRATCGYCTT' \\\n",
    "    --o-trimmed-sequences 18S-demux-pairedend-trimmed1.qza \\\n",
    "    --verbose\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences 18S-paired-end-demux_r2.qza \\\n",
    "    --p-front-f '^AGGGCAAKYCTGGTGCCAGC' \\\n",
    "    --p-front-r '^GRCGGTATCTRATCGYCTT' \\\n",
    "    --o-trimmed-sequences 18S-demux-pairedend-trimmed2.qza \\\n",
    "    --verbose\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences 18S-paired-end-demux_r3.qza \\\n",
    "    --p-front-f '^AGGGCAAKYCTGGTGCCAGC' \\\n",
    "    --p-front-r '^GRCGGTATCTRATCGYCTT' \\\n",
    "    --o-trimmed-sequences 18S-demux-pairedend-trimmed3.qza \\\n",
    "    --verbose"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: 18S-demux-pairedend-trimmed1.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: 18S-demux-pairedend-trimmed2.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: 18S-demux-pairedend-trimmed3.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime demux summarize \\\n",
    "  --i-data 18S-demux-pairedend-trimmed1.qza\\\n",
    "  --o-visualization 18S-demux-pairedend-trimmed1.qzv\n",
    "!qiime demux summarize \\\n",
    "  --i-data 18S-demux-pairedend-trimmed2.qza\\\n",
    "  --o-visualization 18S-demux-pairedend-trimmed2.qzv\n",
    "!qiime demux summarize \\\n",
    "  --i-data 18S-demux-pairedend-trimmed3.qza\\\n",
    "  --o-visualization 18S-demux-pairedend-trimmed3.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-cib2ail_')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 57ccfd29-98cb-4623-a517-245affde68bb>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q2.Visualization.load('18S-demux-pairedend-trimmed1.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-guabmhcr')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 0eb45950-45cb-4416-907a-19c818feb1c2>"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q2.Visualization.load('18S-demux-pairedend-trimmed2.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-k1hr1_52')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 4753125e-1abb-4898-b0f2-cf46575b5750>"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q2.Visualization.load('18S-demux-pairedend-trimmed3.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: 18S-paired-end-demux_table-dada2_r1.qza\u001b[0m\r\n",
      "\u001b[32mSaved FeatureData[Sequence] to: 18S-paired-end-demux_rep-seqs-dada2_r1.qza\u001b[0m\r\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: 18S-paired-end-demux_stats-dada2_r1.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#Generate ESVs (Exact Sequence Variants). QIIME2 has two packages available for this. We are using dada2 for our single end reads.\n",
    "# NOTE: if you are wanting to do paired end reads, you'll use qiime dada2 denoise-paired (https://docs.qiime2.org/2019.7/plugins/available/dada2/denoise-paired/)\n",
    "# NOTE: if you are wanting to join reads you'll have to use Deblur\n",
    "\n",
    "\n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs 18S-demux-pairedend-trimmed1.qza \\\n",
    "  --p-trunc-len-f 273 \\\n",
    "  --p-trunc-len-r 216 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method consensus \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences 18S-paired-end-demux_rep-seqs-dada2_r1.qza \\\n",
    "  --o-table 18S-paired-end-demux_table-dada2_r1.qza \\\n",
    "  --o-denoising-stats 18S-paired-end-demux_stats-dada2_r1.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: 18S-paired-end-demux_table-dada2_r2.qza\u001b[0m\r\n",
      "\u001b[32mSaved FeatureData[Sequence] to: 18S-paired-end-demux_rep-seqs-dada2_r2.qza\u001b[0m\r\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: 18S-paired-end-demux_stats-dada2_r2.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#Generate ESVs (Exact Sequence Variants). QIIME2 has two packages available for this. We are using dada2 for our single end reads.\n",
    "# NOTE: if you are wanting to do paired end reads, you'll use qiime dada2 denoise-paired (https://docs.qiime2.org/2019.7/plugins/available/dada2/denoise-paired/)\n",
    "# NOTE: if you are wanting to join reads you'll have to use Deblur\n",
    "\n",
    "\n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs 18S-demux-pairedend-trimmed2.qza \\\n",
    "  --p-trunc-len-f 273 \\\n",
    "  --p-trunc-len-r 213 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method consensus \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences 18S-paired-end-demux_rep-seqs-dada2_r2.qza \\\n",
    "  --o-table 18S-paired-end-demux_table-dada2_r2.qza \\\n",
    "  --o-denoising-stats 18S-paired-end-demux_stats-dada2_r2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: 18S-paired-end-demux_table-dada2_r3.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: 18S-paired-end-demux_rep-seqs-dada2_r3.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: 18S-paired-end-demux_stats-dada2_r3.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#Generate ESVs (Exact Sequence Variants). QIIME2 has two packages available for this. We are using dada2 for our single end reads.\n",
    "# NOTE: if you are wanting to do paired end reads, you'll use qiime dada2 denoise-paired (https://docs.qiime2.org/2019.7/plugins/available/dada2/denoise-paired/)\n",
    "# NOTE: if you are wanting to join reads you'll have to use Deblur\n",
    "\n",
    "\n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs 18S-demux-pairedend-trimmed3.qza \\\n",
    "  --p-trunc-len-f 240 \\\n",
    "  --p-trunc-len-r 199 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method consensus \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences 18S-paired-end-demux_rep-seqs-dada2_r3.qza \\\n",
    "  --o-table 18S-paired-end-demux_table-dada2_r3.qza \\\n",
    "  --o-denoising-stats 18S-paired-end-demux_stats-dada2_r3.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: 18S-table_r1.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: 18S-table_r2.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: 18S-table_r3.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "# The feature-table summarize command will give you information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table 18S-paired-end-demux_table-dada2_r1.qza \\\n",
    "  --o-visualization 18S-table_r1.qzv \\\n",
    "  #--m-sample-metadata-file metadata_18S_v3.txt\n",
    "# The feature-table summarize command will give you information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table 18S-paired-end-demux_table-dada2_r2.qza \\\n",
    "  --o-visualization 18S-table_r2.qzv \\\n",
    "  #--m-sample-metadata-file metadata_18S_v3.txt\n",
    "!qiime feature-table summarize \\\n",
    "  --i-table 18S-paired-end-demux_table-dada2_r3.qza \\\n",
    "  --o-visualization 18S-table_r3.qzv \\\n",
    "  #--m-sample-metadata-file metadata_18S_v3.txt\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-13f3pyof')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: f9e8e6cc-f01d-4f58-9a29-ccedabe64cdb>"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('18S-table_r1.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-gw371eql')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: b00ea57b-98dc-45f2-bbe7-9a8550e2e2dc>"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('18S-table_r2.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-zkvdfb_i')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: d232a199-c99c-483b-b644-ed7e70c57ca7>"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('18S-table_r3.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: 18S-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: 18S-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "\n",
    "#now merge the three runs\n",
    "!qiime feature-table merge \\\n",
    "  --i-tables 18S-paired-end-demux_table-dada2_r1.qza \\\n",
    "  --i-tables 18S-paired-end-demux_table-dada2_r2.qza \\\n",
    "  --i-tables 18S-paired-end-demux_table-dada2_r3.qza \\\n",
    "  --o-merged-table 18S-paired-end-demux_table-dada2.qza\n",
    "!qiime feature-table merge-seqs \\\n",
    "  --i-data 18S-paired-end-demux_rep-seqs-dada2_r1.qza  \\\n",
    "  --i-data 18S-paired-end-demux_rep-seqs-dada2_r2.qza  \\\n",
    "  --i-data 18S-paired-end-demux_rep-seqs-dada2_r3.qza  \\\n",
    "  --o-merged-data 18S-paired-end-demux_rep-seqs-dada2.qza "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: 18S-table.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "# The feature-table summarize command will give you information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table 18S-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization 18S-table.qzv \\\n",
    " # --m-sample-metadata-file metadata_18S.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-nvisifm9')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: c5c69970-ae2a-4c14-b6b2-f509784a364c>"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('18S-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: 18S-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-fk51iqxs')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 2e42f9db-2248-4ef0-bbb1-523a19cedd88>"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  18S-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization 18S-paired-end-demux_rep-seqs-dada2.qzv\n",
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('18S-paired-end-demux_rep-seqs-dada2.qzv')\n",
    "#download as fasta from visualized data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mExported 18S-paired-end-demux_table-dada2.qza as BIOMV210DirFmt to directory exported-feature-table\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime tools export \\\n",
    "  --input-path 18S-paired-end-demux_table-dada2.qza \\\n",
    "  --output-path exported-feature-table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "!biom convert -i exported-feature-table/feature-table.biom -o 18S_ASVtable.tsv --to-tsv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "!biom convert -i exported-feature-table/feature-table.biom -o exported-feature-table/feature-table_hdf5.biom --to-hdf5 --table-type=\"OTU table\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "!biom convert -i exported-feature-table/feature-table.biom -o exported-feature-table/feature-table_json.biom --to-json --table-type=\"OTU table\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[31m\u001b[1mPlugin error from feature-classifier:\n",
      "\n",
      "  The scikit-learn version (0.19.1) used to generate this artifact does not match the current version of scikit-learn installed (0.23.1). Please retrain your classifier for your current deployment to prevent data-corruption errors.\n",
      "\n",
      "Debug info has been saved to /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-q2cli-err-74lsabo7.log\u001b[0m\n",
      "\u001b[31m\u001b[1mThere was an issue with loading the file 18SASV-rep-seqs-dada2_sklearnSILVA132_taxonomy.qza as metadata:\n",
      "\n",
      "  Metadata file path doesn't exist, or the path points to something other than a file. Please check that the path exists, has read permissions, and points to a regular file (not a directory): 18SASV-rep-seqs-dada2_sklearnSILVA132_taxonomy.qza\n",
      "\n",
      "  There may be more errors present in the metadata file. To get a full report, sample/feature metadata files can be validated with Keemei: https://keemei.qiime2.org\n",
      "\n",
      "  Find details on QIIME 2 metadata requirements here: https://docs.qiime2.org/2020.6/tutorials/metadata/\u001b[0m\n",
      "\n"
     ]
    }
   ],
   "source": [
    "## classify with sklearn SILVA classifier\n",
    "\n",
    "#!qiime feature-classifier classify-sklearn \\\n",
    "#  --i-classifier SILVA_132_99_nb_classifier.qza \\\n",
    "#  --i-reads 18S-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "#  --o-classification 18SASV-rep-seqs-dada2_sklearnSILVA132_taxonomy.qza\n",
    "#\n",
    "#!qiime metadata tabulate \\\n",
    "#  --m-input-file 18SASV-rep-seqs-dada2_sklearnSILVA132_taxonomy.qza \\\n",
    "#  --o-visualization 18SASV-rep-seqs-dada2_sklearnSILVA132_taxonomy.qzv\n",
    "\n",
    "\n",
    "## this doesn't work b/c the clasifier was trained using scikit-learn v 0.19.1 on the jcvi grid, and the local version of sklearn\n",
    "#is 0.23.1. Transferring this fasta to the grid, importing to qiime, and classifying \n",
    "#the seqs using the version of sklearn that's installed there"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: 18SSILVA_skl_taxonomy.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-3ldjf9n3')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: af2d8afe-c231-4b7c-a80d-cbb095a3261a>"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "  --m-input-file  18SSILVA_skl_taxonomy.qza \\\n",
    "  --o-visualization 18SSILVA_skl_taxonomy.qzv\n",
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('18SSILVA_skl_taxonomy.qzv')\n",
    "#download as fasta from visualized data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Taxonomy] to: 18S-MZGtrimmed-sklearn-taxonomy.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: 18S-MZGtrimmed-sklearn-taxonomy.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-classifier classify-sklearn \\\n",
    "  --i-classifier /Users/stephaniematthews/Documents/databases/metazoogene_steph_edits/MZG-zhan-classifier_r_trimmed.qza \\\n",
    "  --i-reads 18S-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-classification 18S-MZGtrimmed-sklearn-taxonomy.qza\n",
    "\n",
    "!qiime metadata tabulate \\\n",
    "  --m-input-file 18S-MZGtrimmed-sklearn-taxonomy.qza \\\n",
    "  --o-visualization 18S-MZGtrimmed-sklearn-taxonomy.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-oa6q8n7u')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: f8cbed61-f56d-4d5d-8bb5-46fab2fcb1d3>"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import qiime2 as q2\n",
    "q2.Visualization.load('18S-MZGtrimmed-sklearn-taxonomy.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
